---
title: "Template"
author: "Lachlan Baer"
date: "`r format(Sys.time(), '%d %B, %Y')`"
output:
  html_document:
    toc: true
    toc_float: true
    toc_depth: 3
    fig_width: 8
    fig_height: 6
    fig_align: "center"
    code_folding: hide
editor_options: 
  chunk_output_type: console
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(
  echo = TRUE,
  message = FALSE,
  warning = FALSE,
  error = FALSE, 
  fig.align = "center"
)
knitr::opts_knit$set(root.dir = rprojroot::find_rstudio_root_file())
```

# Setup 

```{r packages}
library(tidyverse)
library(magrittr)
library(parallel)
library(here)
library(SeqArray)
library(AnnotationHub)
```

```{r options}
if (interactive()) setwd(here::here())
theme_set(theme_bw())
cores <- detectCores() - 2
```

## Annotation

```{r}
ah <- AnnotationHub() %>%
  subset(species == "Danio rerio") %>%
  subset(rdataclass == "EnsDb")
ensDb <- ah[["AH83189"]]
genes <- genes(ensDb)
mcols(genes) <- mcols(genes)[
  c("gene_id", "gene_name", "gene_biotype", "entrezid")
]
```

# GDS file

```{r}
vcfPath <- "/hpcfs/users/a1647910/210305_chromosomeProject/1_sorl1_4way/12_mergeVcfs/vcf/mergedVcf.vcf.gz"
gdsPath <- "/hpcfs/users/a1647910/210305_chromosomeProject/1_sorl1_4way/12_mergeVcfs/gds/mergedGds.gds"
makeGds <-!file.exists(gdsPath)
if (makeGds) {
  seqVCF2GDS(vcfPath, gdsPath)
}
```

```{r}
gds <- seqOpen(gdsPath, readonly = FALSE)
print(gds, all=TRUE, attribute=TRUE)
```

# Metadata

```{r}
meta <- read_csv(here("files/SraRunTable.txt")) %>%
  dplyr::select(Run, gender, Genotype) %>%
  mutate(
    bname = case_when(
      Genotype == "wild-type" ~ "WT",
      Genotype == "sorl1V1482Afs/+" ~ "V1482Afs",
      Genotype == "sorl1R122Pfs/+" ~ "R122Pfs",
      Genotype == "sorl1V1482Afs/R122Pfs" ~ "Trans"
    )
  ) %>%
  dplyr::arrange(Genotype, Run) %>%
  mutate(
    sample =  paste0(bname, "_", c(1:4, 1:6, 1:6, 1:8))
  ) %>%
  dplyr::select(-bname) %>%
  dplyr::arrange(Run)
```

## Variants

```{r}
seqResetFilter(gds)
chrs <- paste(c(1:25))
allSnps <- tibble(
  variant.id = seqGetData(gds, "variant.id"),
  chromosome = seqGetData(gds, "chromosome"),
  position = seqGetData(gds, "position"),
  allele = seqGetData(gds, "allele")
) %>%
  dplyr::filter(chromosome %in% chrs) %>%
  as_tibble()
```

## Gene-specific variants

```{r}
goi <- subset(genes, gene_name == "sorl1")
geneSnps <- allSnps %>%
  dplyr::filter(
    chromosome == as.character(seqnames(goi)), 
    position >= start(goi),
    position <= end(goi)
  )
# geneSnps$allele %>% lapply(function(x){str_split(x, ",") %>% unlist() %>% length() == 2}) %>% unlist()
```

```{r}
## Samples per genotype
wt <- str_detect(meta$sample, "WT")
v1482 <- str_detect(meta$sample, "V1482")
r122 <- str_detect(meta$sample, "R122")
trans <- str_detect(meta$sample, "Trans")
```

```{r}
seqResetFilter(gds)
seqSetFilter(gds, variant.sel = geneSnps$variant.id, sample.sel = wt)
```

```{r}
lapply(geneSnps$variant.id, function(x){
  seqResetFilter(gds)
  seqSetFilter(gds, variant.sel = x, sample.sel = wt)
  als <- seqGetData(gds, "allele") %>%
    str_split(",") %>% 
    unlist()
  seqGetData(gds, "annotation/format/AD")$data %>%
    t() %>%
    rowSums(na.rm = TRUE) %>%
    set_names(als) %>%
    bind_rows() %>%
    mutate(variant.id = x)
}) %>%
  purrr::reduce(full_join) %>%
  right_join(geneSnps) %>%
  dplyr::select(variant.id, chromosome, position, allele, A, C, G, T, everything())
```

---

```{r}
lapply(geneSnps$variant.id, function(x){
  seqResetFilter(gds)
  seqSetFilter(gds, variant.sel = x, sample.sel = wt)
  als <- seqGetData(gds, "allele") %>%
    str_split(",") %>% 
    unlist()
  lapply(als, function(x){
    seqAlleleCount(gds, ref.allele = x)
  }) %>%
    unlist() %>%
    set_names(als) %>%
    bind_rows() %>%
    mutate(variant.id = x)
}) %>%
  purrr::reduce(full_join) %>%
  right_join(geneSnps) %>%
  dplyr::select(variant.id, chromosome, position, allele, A, C, G, T, everything())
```

---

```{r}
geneGenos <- lapply(geneSnps$variant.id, function(x){
  seqResetFilter(gds)
  vId <- x
  seqSetFilter(gds, vId)
  chr <- geneSnps$chromosome[geneSnps$variant.id == vId]
  pos <- geneSnps$position[geneSnps$variant.id == vId]
  allele <- geneSnps$allele[geneSnps$variant.id == vId]
  alleles <- geneSnps$allele[geneSnps$variant.id == vId] %>%
    str_split(",") %>%
    unlist()
  genotype <- seqGetData(gds, "genotype") %>%
    set_colnames(meta$sample) %>%
    .[,,1] %>%
    as_tibble()
  genotype[] <- lapply(genotype, function(x){x+1}) %>%
    lapply(function(x){alleles[x]})
  sampleGeno <- genotype %>%
    t() %>% 
    map2(
      .x = .[,1],
      .y = .[,2],
      .f = ~ ifelse(!is.na(.x) & !is.na(.y), paste0(.x, ",", .y), NA)
    )
  return(
    tibble(
      variant.id = vId,
      chromosome = chr,
      position = pos,
      variants = allele
    ) %>%
      cbind(sampleGeno)
  )
  seqResetFilter(gds)
}) %>%
  purrr::reduce(rbind) %>% 
  as_tibble()
```

```{r}
geneAD <- lapply(geneSnps$variant.id, function(x){
  seqResetFilter(gds)
  vId <- x
  seqSetFilter(gds, vId)
  chr <- geneSnps$chromosome[geneSnps$variant.id == vId]
  pos <- geneSnps$position[geneSnps$variant.id == vId]
  allele <- geneSnps$allele[geneSnps$variant.id == vId]
  alleleSplit <- geneSnps$allele[geneSnps$variant.id == vId] %>%
    str_split(",") %>%
    unlist()
  adMat <- seqGetData(gds, "annotation/format/AD")$data %>% 
    t() %>% 
    set_colnames(meta$sample) %>%
    set_rownames(alleleSplit)
  return(
    tibble(
      variant.id = vId,
      chromosome = chr,
      position = pos,
      variants = allele,
      AD = adMat
    )
  )
  seqResetFilter(gds)
}) %>%
  purrr::reduce(rbind) %>% 
  as_tibble()

adMat %>% 
  as_tibble() %>% 
  mutate(
    variant.id = vId,
    chromosome = chr,
    position = pos,
    allele = alleleSplit
  ) %>% 
  pivot_longer(cols = !c("variant.id", "chromosome", "position", "allele")) %>%
  group_by()
```

---

```{r}
a <- lapply(geneSnps$variant.id, function(x){
  seqResetFilter(gds)
  seqSetFilter(gds, x)
  als <- seqGetData(gds, "allele") %>%
    str_split(",") %>% unlist()
  lapply()
  seqAlleleCount(gds, ref.allele = "A")
}) %>%
  unlist()
c <- lapply(geneSnps$variant.id, function(x){
  seqResetFilter(gds)
  seqSetFilter(gds, x)
  seqAlleleCount(gds, ref.allele = "C")
}) %>%
  unlist()
g <- lapply(geneSnps$variant.id, function(x){
  seqResetFilter(gds)
  seqSetFilter(gds, x)
  seqAlleleCount(gds, ref.allele = "G")
}) %>%
  unlist()
t <- lapply(geneSnps$variant.id, function(x){
  seqResetFilter(gds)
  seqSetFilter(gds, x)
  seqAlleleCount(gds, ref.allele = "T")
}) %>%
  unlist()
geneSnps %>%
  mutate(
    A.count = a,
    C.count = c,
    G.count = g,
    T.count = t
  )
```

---

```{r}
homozygous <- seqGetData(gds, "genotype") %>%
  set_colnames(meta$sample) %>%
  .[,,1] %>%
  t() %>%
  map2(.x = .[,1], .y = .[,2], .f = ~ .x == .y) %>% 
  unlist()
# genoList <- lapply(seq(dim(genoArray)[3]), function(x){genoArray[,,x]})
```

```{r}
vId <- 368358
alleles <- snps$allele[snps$variant.id == vId] %>%
  str_split(",") %>%
  unlist()
```

```{r}
vId <- 368355
seqResetFilter(gds)
seqSetFilter(gds, vId)
seqGetData(gds, "genotype") %>% 
  set_colnames(meta$sample)
seqGetData(gds, "annotation/format/AD")$data %>% 
  t() %>% 
  set_colnames(meta$sample) %>%
  set_rownames(alleles)
```
