<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Lachlan Baer" />


<title>Allele Specific Expression Quality Control</title>

<script src="site_libs/header-attrs-2.9/header-attrs.js"></script>
<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<script src="site_libs/plotly-binding-4.9.4.1/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-1.57.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-1.57.1/plotly-latest.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">210408_psen1_fADfAI_snv</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="ase_qc.html">ASE QC</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/baerlachlan/210216_sorl1_snv">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Allele Specific Expression Quality Control</h1>
<h3 class="subtitle">210216_sorl1 dataset</h3>
<h4 class="author">Lachlan Baer</h4>
<h4 class="date">18 September, 2021</h4>

</div>


<div id="setup" class="section level1">
<h1>Setup</h1>
<pre class="r"><code>suppressPackageStartupMessages({
  ## Common
  library(tidyverse)
  library(magrittr)
  library(future.apply)
  library(here)
  library(AnnotationHub)
  library(purrr)
  library(scales)
  library(kableExtra)
  library(tictoc)
  library(ggrepel)
  library(RColorBrewer)
  library(ggpubr)
  library(pander)
  library(rmarkdown)
  ## Project specific
  library(UpSetR)
  library(SeqArray)
  library(ngsReports)
})</code></pre>
<pre class="r"><code>if (interactive()) setwd(here::here())
theme_set(theme_bw())
cores &lt;- availableCores() - 1</code></pre>
<pre class="r"><code>source(&quot;~/bioinformatics/bioToolkit/lbFuncs.R&quot;)</code></pre>
<div id="ensdb" class="section level2">
<h2>EnsDb</h2>
<pre class="r"><code>ah &lt;- AnnotationHub() %&gt;%
  subset(species == &quot;Danio rerio&quot;) %&gt;%
  subset(rdataclass == &quot;EnsDb&quot;)
ensDb &lt;- ah[[&quot;AH83189&quot;]] ## Ens101
genes &lt;- genes(ensDb)
mcols(genes) &lt;- mcols(genes)[
  c(&quot;gene_id&quot;, &quot;gene_name&quot;, &quot;gene_biotype&quot;, &quot;entrezid&quot;)
]
exons &lt;- exonsBy(ensDb, by = &quot;gene&quot;)</code></pre>
<p>An <code>EnsDb</code> object for Ensembl release 101 was setup for extracting gene and exon annotation information.</p>
</div>
<div id="metadata" class="section level2">
<h2>Metadata</h2>
<pre class="r"><code>metadata &lt;- read_csv(here(&quot;files/SraRunTable.txt&quot;)) %&gt;%
  as.data.frame() %&gt;%
  dplyr::arrange(Run) %&gt;%
  mutate(
    Genotype = factor(Genotype, levels = unique(Genotype)),
    Run = factor(Run, levels = Run), 
    alias = c(
      paste0(rep(&quot;WT&quot;, 8), seq(1, 8)),
      paste0(rep(&quot;V1482Afs&quot;, 6), seq(1, 6)),
      paste0(rep(&quot;R122Pfs&quot;, 4), seq(1, 4)),
      paste0(rep(&quot;Trans&quot;, 6), seq(1, 6))
    )
  ) %&gt;%
  dplyr::select(
    sample = Run, genotype = Genotype, alias, gender, tank = Tank
  )
metadata %&gt;%
  dplyr::rename(
    Sample = sample, Genotype = genotype, Alias = alias, Gender = gender,
    Tank = tank
  ) %&gt;%
  kable(
    align = &quot;l&quot;,
    caption = &quot;Sample metadata&quot;
  ) %&gt;%
  kable_styling(
    bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;, &quot;condensed&quot;, &quot;responsive&quot;)
  )</code></pre>
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<caption>
Sample metadata
</caption>
<thead>
<tr>
<th style="text-align:left;">
Sample
</th>
<th style="text-align:left;">
Genotype
</th>
<th style="text-align:left;">
Alias
</th>
<th style="text-align:left;">
Gender
</th>
<th style="text-align:left;">
Tank
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
SRR11951228
</td>
<td style="text-align:left;">
wild-type
</td>
<td style="text-align:left;">
WT1
</td>
<td style="text-align:left;">
female
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951229
</td>
<td style="text-align:left;">
wild-type
</td>
<td style="text-align:left;">
WT2
</td>
<td style="text-align:left;">
female
</td>
<td style="text-align:left;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951230
</td>
<td style="text-align:left;">
wild-type
</td>
<td style="text-align:left;">
WT3
</td>
<td style="text-align:left;">
male
</td>
<td style="text-align:left;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951231
</td>
<td style="text-align:left;">
wild-type
</td>
<td style="text-align:left;">
WT4
</td>
<td style="text-align:left;">
male
</td>
<td style="text-align:left;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951232
</td>
<td style="text-align:left;">
wild-type
</td>
<td style="text-align:left;">
WT5
</td>
<td style="text-align:left;">
male
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951233
</td>
<td style="text-align:left;">
wild-type
</td>
<td style="text-align:left;">
WT6
</td>
<td style="text-align:left;">
female
</td>
<td style="text-align:left;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951234
</td>
<td style="text-align:left;">
wild-type
</td>
<td style="text-align:left;">
WT7
</td>
<td style="text-align:left;">
male
</td>
<td style="text-align:left;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951235
</td>
<td style="text-align:left;">
wild-type
</td>
<td style="text-align:left;">
WT8
</td>
<td style="text-align:left;">
female
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951236
</td>
<td style="text-align:left;">
sorl1V1482Afs/+
</td>
<td style="text-align:left;">
V1482Afs1
</td>
<td style="text-align:left;">
male
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951237
</td>
<td style="text-align:left;">
sorl1V1482Afs/+
</td>
<td style="text-align:left;">
V1482Afs2
</td>
<td style="text-align:left;">
male
</td>
<td style="text-align:left;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951238
</td>
<td style="text-align:left;">
sorl1V1482Afs/+
</td>
<td style="text-align:left;">
V1482Afs3
</td>
<td style="text-align:left;">
male
</td>
<td style="text-align:left;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951239
</td>
<td style="text-align:left;">
sorl1V1482Afs/+
</td>
<td style="text-align:left;">
V1482Afs4
</td>
<td style="text-align:left;">
female
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951240
</td>
<td style="text-align:left;">
sorl1V1482Afs/+
</td>
<td style="text-align:left;">
V1482Afs5
</td>
<td style="text-align:left;">
female
</td>
<td style="text-align:left;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951241
</td>
<td style="text-align:left;">
sorl1V1482Afs/+
</td>
<td style="text-align:left;">
V1482Afs6
</td>
<td style="text-align:left;">
female
</td>
<td style="text-align:left;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951242
</td>
<td style="text-align:left;">
sorl1R122Pfs/+
</td>
<td style="text-align:left;">
R122Pfs1
</td>
<td style="text-align:left;">
male
</td>
<td style="text-align:left;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951243
</td>
<td style="text-align:left;">
sorl1R122Pfs/+
</td>
<td style="text-align:left;">
R122Pfs2
</td>
<td style="text-align:left;">
male
</td>
<td style="text-align:left;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951244
</td>
<td style="text-align:left;">
sorl1R122Pfs/+
</td>
<td style="text-align:left;">
R122Pfs3
</td>
<td style="text-align:left;">
female
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951245
</td>
<td style="text-align:left;">
sorl1R122Pfs/+
</td>
<td style="text-align:left;">
R122Pfs4
</td>
<td style="text-align:left;">
female
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951246
</td>
<td style="text-align:left;">
sorl1V1482Afs/R122Pfs
</td>
<td style="text-align:left;">
Trans1
</td>
<td style="text-align:left;">
female
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951247
</td>
<td style="text-align:left;">
sorl1V1482Afs/R122Pfs
</td>
<td style="text-align:left;">
Trans2
</td>
<td style="text-align:left;">
female
</td>
<td style="text-align:left;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951248
</td>
<td style="text-align:left;">
sorl1V1482Afs/R122Pfs
</td>
<td style="text-align:left;">
Trans3
</td>
<td style="text-align:left;">
female
</td>
<td style="text-align:left;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951249
</td>
<td style="text-align:left;">
sorl1V1482Afs/R122Pfs
</td>
<td style="text-align:left;">
Trans4
</td>
<td style="text-align:left;">
male
</td>
<td style="text-align:left;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951250
</td>
<td style="text-align:left;">
sorl1V1482Afs/R122Pfs
</td>
<td style="text-align:left;">
Trans5
</td>
<td style="text-align:left;">
male
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR11951251
</td>
<td style="text-align:left;">
sorl1V1482Afs/R122Pfs
</td>
<td style="text-align:left;">
Trans6
</td>
<td style="text-align:left;">
male
</td>
<td style="text-align:left;">
3
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>genoCols &lt;- metadata$genotype %&gt;%
  levels() %&gt;%
  length() %&gt;%
  brewer.pal(&quot;Set1&quot;) %&gt;%
  setNames(levels(metadata$genotype))</code></pre>
<pre class="r"><code>drChrs &lt;- seq(1:25)</code></pre>
</div>
</div>
<div id="pre-processing" class="section level1">
<h1>Pre-processing</h1>
<div id="raw-data" class="section level2">
<h2>Raw data</h2>
<pre class="r"><code>rawFqc &lt;- list.files(
  path = here(&quot;00_rawData/FastQC/&quot;),
  pattern = &quot;zip&quot;,
  full.names = TRUE
) %&gt;%
  FastqcDataList()</code></pre>
<div id="library-sizes" class="section level3">
<h3>Library sizes</h3>
<p>Library Sizes for the raw, unprocessed dataset ranged between <em>30,648,602</em> and <em>37,751,699</em> reads.</p>
<pre class="r"><code>plotReadTotals(rawFqc)</code></pre>
<p><img src="ase_qc_files/figure-html/unnamed-chunk-6-1.png" width="768" style="display: block; margin: auto;" /></p>
</div>
<div id="gc-content" class="section level3">
<h3>GC content</h3>
<pre class="r"><code>plotly::ggplotly({
  plotGcContent(
    x = rawFqc, 
    plotType = &quot;line&quot;,
    gcType = &quot;Transcriptome&quot;,
    species = &quot;Drerio&quot;
  ) +
    theme(legend.position = &quot;none&quot;)
})</code></pre>
<div class="figure" style="text-align: center">
<div id="htmlwidget-e90a3d7484106363866c" style="width:768px;height:576px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-e90a3d7484106363866c">{"x":{"data":[{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0,0,0,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.01,0.02,0.03,0.05,0.07,0.11,0.15,0.21,0.28,0.38,0.48,0.59,0.72,0.85,0.99,1.11,1.25,1.36,1.48,1.58,1.67,1.77,1.88,2.04,2.24,2.47,2.75,3.07,3.44,3.79,4.16,4.47,4.69,4.86,4.91,4.87,4.7,4.43,4.11,3.72,3.3,2.86,2.44,2.03,1.67,1.35,1.08,0.85,0.66,0.51,0.39,0.29,0.22,0.16,0.12,0.09,0.06,0.04,0.03,0.02,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"text":["Filename: Theoretical Distribution<br />GC_Content:   0<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:   1<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:   2<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:   3<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:   4<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:   5<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:   6<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:   7<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:   8<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:   9<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  10<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  11<br />Percent: 0.01","Filename: Theoretical Distribution<br />GC_Content:  12<br />Percent: 0.01","Filename: Theoretical Distribution<br />GC_Content:  13<br />Percent: 0.01","Filename: Theoretical Distribution<br />GC_Content:  14<br />Percent: 0.01","Filename: Theoretical Distribution<br />GC_Content:  15<br />Percent: 0.01","Filename: Theoretical Distribution<br />GC_Content:  16<br />Percent: 0.02","Filename: Theoretical Distribution<br />GC_Content:  17<br />Percent: 0.03","Filename: Theoretical Distribution<br />GC_Content:  18<br />Percent: 0.05","Filename: Theoretical Distribution<br />GC_Content:  19<br />Percent: 0.07","Filename: Theoretical Distribution<br />GC_Content:  20<br />Percent: 0.11","Filename: Theoretical Distribution<br />GC_Content:  21<br />Percent: 0.15","Filename: Theoretical Distribution<br />GC_Content:  22<br />Percent: 0.21","Filename: Theoretical Distribution<br />GC_Content:  23<br />Percent: 0.28","Filename: Theoretical Distribution<br />GC_Content:  24<br />Percent: 0.38","Filename: Theoretical Distribution<br />GC_Content:  25<br />Percent: 0.48","Filename: Theoretical Distribution<br />GC_Content:  26<br />Percent: 0.59","Filename: Theoretical Distribution<br />GC_Content:  27<br />Percent: 0.72","Filename: Theoretical Distribution<br />GC_Content:  28<br />Percent: 0.85","Filename: Theoretical Distribution<br />GC_Content:  29<br />Percent: 0.99","Filename: Theoretical Distribution<br />GC_Content:  30<br />Percent: 1.11","Filename: Theoretical Distribution<br />GC_Content:  31<br />Percent: 1.25","Filename: Theoretical Distribution<br />GC_Content:  32<br />Percent: 1.36","Filename: Theoretical Distribution<br />GC_Content:  33<br />Percent: 1.48","Filename: Theoretical Distribution<br />GC_Content:  34<br />Percent: 1.58","Filename: Theoretical Distribution<br />GC_Content:  35<br />Percent: 1.67","Filename: Theoretical Distribution<br />GC_Content:  36<br />Percent: 1.77","Filename: Theoretical Distribution<br />GC_Content:  37<br />Percent: 1.88","Filename: Theoretical Distribution<br />GC_Content:  38<br />Percent: 2.04","Filename: Theoretical Distribution<br />GC_Content:  39<br />Percent: 2.24","Filename: Theoretical Distribution<br />GC_Content:  40<br />Percent: 2.47","Filename: Theoretical Distribution<br />GC_Content:  41<br />Percent: 2.75","Filename: Theoretical Distribution<br />GC_Content:  42<br />Percent: 3.07","Filename: Theoretical Distribution<br />GC_Content:  43<br />Percent: 3.44","Filename: Theoretical Distribution<br />GC_Content:  44<br />Percent: 3.79","Filename: Theoretical Distribution<br />GC_Content:  45<br />Percent: 4.16","Filename: Theoretical Distribution<br />GC_Content:  46<br />Percent: 4.47","Filename: Theoretical Distribution<br />GC_Content:  47<br />Percent: 4.69","Filename: Theoretical Distribution<br />GC_Content:  48<br />Percent: 4.86","Filename: Theoretical Distribution<br />GC_Content:  49<br />Percent: 4.91","Filename: Theoretical Distribution<br />GC_Content:  50<br />Percent: 4.87","Filename: Theoretical Distribution<br />GC_Content:  51<br />Percent: 4.70","Filename: Theoretical Distribution<br />GC_Content:  52<br />Percent: 4.43","Filename: Theoretical Distribution<br />GC_Content:  53<br />Percent: 4.11","Filename: Theoretical Distribution<br />GC_Content:  54<br />Percent: 3.72","Filename: Theoretical Distribution<br />GC_Content:  55<br />Percent: 3.30","Filename: Theoretical Distribution<br />GC_Content:  56<br />Percent: 2.86","Filename: Theoretical Distribution<br />GC_Content:  57<br />Percent: 2.44","Filename: Theoretical Distribution<br />GC_Content:  58<br />Percent: 2.03","Filename: Theoretical Distribution<br />GC_Content:  59<br />Percent: 1.67","Filename: Theoretical Distribution<br />GC_Content:  60<br />Percent: 1.35","Filename: Theoretical Distribution<br />GC_Content:  61<br />Percent: 1.08","Filename: Theoretical Distribution<br />GC_Content:  62<br />Percent: 0.85","Filename: Theoretical Distribution<br />GC_Content:  63<br />Percent: 0.66","Filename: Theoretical Distribution<br />GC_Content:  64<br />Percent: 0.51","Filename: Theoretical Distribution<br />GC_Content:  65<br />Percent: 0.39","Filename: Theoretical Distribution<br />GC_Content:  66<br />Percent: 0.29","Filename: Theoretical Distribution<br />GC_Content:  67<br />Percent: 0.22","Filename: Theoretical Distribution<br />GC_Content:  68<br />Percent: 0.16","Filename: Theoretical Distribution<br />GC_Content:  69<br />Percent: 0.12","Filename: Theoretical Distribution<br />GC_Content:  70<br />Percent: 0.09","Filename: Theoretical Distribution<br />GC_Content:  71<br />Percent: 0.06","Filename: Theoretical Distribution<br />GC_Content:  72<br />Percent: 0.04","Filename: Theoretical Distribution<br />GC_Content:  73<br />Percent: 0.03","Filename: Theoretical Distribution<br />GC_Content:  74<br />Percent: 0.02","Filename: Theoretical Distribution<br />GC_Content:  75<br />Percent: 0.01","Filename: Theoretical Distribution<br />GC_Content:  76<br />Percent: 0.01","Filename: Theoretical Distribution<br />GC_Content:  77<br />Percent: 0.01","Filename: Theoretical Distribution<br />GC_Content:  78<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  79<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  80<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  81<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  82<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  83<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  84<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  85<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  86<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  87<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  88<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  89<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  90<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  91<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  92<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  93<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  94<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  95<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  96<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  97<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  98<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content:  99<br />Percent: 0.00","Filename: Theoretical Distribution<br />GC_Content: 100<br />Percent: 0.00"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)","dash":"solid"},"hoveron":"points","name":"Theoretical Distribution","legendgroup":"Theoretical Distribution","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.01,0.01,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.06,0.09,0.13,0.15,0.2,0.28,0.39,0.47,0.61,0.88,1.16,1.34,1.55,1.9,2.24,2.43,2.68,3.06,3.37,3.56,3.75,3.86,3.84,3.84,3.8,3.71,3.58,3.55,3.6,3.5,3.35,3.24,3.14,3.04,2.86,2.67,2.54,2.35,2.07,1.8,1.6,1.41,1.17,0.95,0.79,0.67,0.54,0.42,0.3,0.24,0.21,0.16,0.12,0.1,0.08,0.07,0.05,0.04,0.04,0.03,0.02,0.02,0.02,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951228<br />GC_Content:   0<br />Percent: 0.01","Filename: SRR11951228<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951228<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951228<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951228<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951228<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951228<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951228<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951228<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951228<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951228<br />GC_Content:  18<br />Percent: 0.06","Filename: SRR11951228<br />GC_Content:  19<br />Percent: 0.09","Filename: SRR11951228<br />GC_Content:  20<br />Percent: 0.13","Filename: SRR11951228<br />GC_Content:  21<br />Percent: 0.15","Filename: SRR11951228<br />GC_Content:  22<br />Percent: 0.20","Filename: SRR11951228<br />GC_Content:  23<br />Percent: 0.28","Filename: SRR11951228<br />GC_Content:  24<br />Percent: 0.39","Filename: SRR11951228<br />GC_Content:  25<br />Percent: 0.47","Filename: SRR11951228<br />GC_Content:  26<br />Percent: 0.61","Filename: SRR11951228<br />GC_Content:  27<br />Percent: 0.88","Filename: SRR11951228<br />GC_Content:  28<br />Percent: 1.16","Filename: SRR11951228<br />GC_Content:  29<br />Percent: 1.34","Filename: SRR11951228<br />GC_Content:  30<br />Percent: 1.55","Filename: SRR11951228<br />GC_Content:  31<br />Percent: 1.90","Filename: SRR11951228<br />GC_Content:  32<br />Percent: 2.24","Filename: SRR11951228<br />GC_Content:  33<br />Percent: 2.43","Filename: SRR11951228<br />GC_Content:  34<br />Percent: 2.68","Filename: SRR11951228<br />GC_Content:  35<br />Percent: 3.06","Filename: SRR11951228<br />GC_Content:  36<br />Percent: 3.37","Filename: SRR11951228<br />GC_Content:  37<br />Percent: 3.56","Filename: SRR11951228<br />GC_Content:  38<br />Percent: 3.75","Filename: SRR11951228<br />GC_Content:  39<br />Percent: 3.86","Filename: SRR11951228<br />GC_Content:  40<br />Percent: 3.84","Filename: SRR11951228<br />GC_Content:  41<br />Percent: 3.84","Filename: SRR11951228<br />GC_Content:  42<br />Percent: 3.80","Filename: SRR11951228<br />GC_Content:  43<br />Percent: 3.71","Filename: SRR11951228<br />GC_Content:  44<br />Percent: 3.58","Filename: SRR11951228<br />GC_Content:  45<br />Percent: 3.55","Filename: SRR11951228<br />GC_Content:  46<br />Percent: 3.60","Filename: SRR11951228<br />GC_Content:  47<br />Percent: 3.50","Filename: SRR11951228<br />GC_Content:  48<br />Percent: 3.35","Filename: SRR11951228<br />GC_Content:  49<br />Percent: 3.24","Filename: SRR11951228<br />GC_Content:  50<br />Percent: 3.14","Filename: SRR11951228<br />GC_Content:  51<br />Percent: 3.04","Filename: SRR11951228<br />GC_Content:  52<br />Percent: 2.86","Filename: SRR11951228<br />GC_Content:  53<br />Percent: 2.67","Filename: SRR11951228<br />GC_Content:  54<br />Percent: 2.54","Filename: SRR11951228<br />GC_Content:  55<br />Percent: 2.35","Filename: SRR11951228<br />GC_Content:  56<br />Percent: 2.07","Filename: SRR11951228<br />GC_Content:  57<br />Percent: 1.80","Filename: SRR11951228<br />GC_Content:  58<br />Percent: 1.60","Filename: SRR11951228<br />GC_Content:  59<br />Percent: 1.41","Filename: SRR11951228<br />GC_Content:  60<br />Percent: 1.17","Filename: SRR11951228<br />GC_Content:  61<br />Percent: 0.95","Filename: SRR11951228<br />GC_Content:  62<br />Percent: 0.79","Filename: SRR11951228<br />GC_Content:  63<br />Percent: 0.67","Filename: SRR11951228<br />GC_Content:  64<br />Percent: 0.54","Filename: SRR11951228<br />GC_Content:  65<br />Percent: 0.42","Filename: SRR11951228<br />GC_Content:  66<br />Percent: 0.30","Filename: SRR11951228<br />GC_Content:  67<br />Percent: 0.24","Filename: SRR11951228<br />GC_Content:  68<br />Percent: 0.21","Filename: SRR11951228<br />GC_Content:  69<br />Percent: 0.16","Filename: SRR11951228<br />GC_Content:  70<br />Percent: 0.12","Filename: SRR11951228<br />GC_Content:  71<br />Percent: 0.10","Filename: SRR11951228<br />GC_Content:  72<br />Percent: 0.08","Filename: SRR11951228<br />GC_Content:  73<br />Percent: 0.07","Filename: SRR11951228<br />GC_Content:  74<br />Percent: 0.05","Filename: SRR11951228<br />GC_Content:  75<br />Percent: 0.04","Filename: SRR11951228<br />GC_Content:  76<br />Percent: 0.04","Filename: SRR11951228<br />GC_Content:  77<br />Percent: 0.03","Filename: SRR11951228<br />GC_Content:  78<br />Percent: 0.02","Filename: SRR11951228<br />GC_Content:  79<br />Percent: 0.02","Filename: SRR11951228<br />GC_Content:  80<br />Percent: 0.02","Filename: SRR11951228<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951228<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951228<br />GC_Content:  83<br />Percent: 0.01","Filename: SRR11951228<br />GC_Content:  84<br />Percent: 0.01","Filename: SRR11951228<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:  94<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951228<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951228<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(166,206,227,1)","dash":"solid"},"hoveron":"points","name":"SRR11951228","legendgroup":"SRR11951228","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.02,0.01,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.07,0.1,0.14,0.16,0.21,0.3,0.41,0.5,0.65,0.92,1.23,1.44,1.67,2.05,2.41,2.62,2.91,3.33,3.67,3.86,4.05,4.13,4.07,4.03,3.96,3.82,3.61,3.54,3.56,3.42,3.22,3.07,2.94,2.8,2.59,2.4,2.29,2.12,1.86,1.6,1.42,1.24,1.02,0.83,0.69,0.58,0.46,0.36,0.26,0.21,0.18,0.14,0.11,0.09,0.08,0.06,0.05,0.04,0.03,0.03,0.02,0.02,0.02,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0.01,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951229<br />GC_Content:   0<br />Percent: 0.02","Filename: SRR11951229<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951229<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951229<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951229<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951229<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951229<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951229<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951229<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951229<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951229<br />GC_Content:  18<br />Percent: 0.07","Filename: SRR11951229<br />GC_Content:  19<br />Percent: 0.10","Filename: SRR11951229<br />GC_Content:  20<br />Percent: 0.14","Filename: SRR11951229<br />GC_Content:  21<br />Percent: 0.16","Filename: SRR11951229<br />GC_Content:  22<br />Percent: 0.21","Filename: SRR11951229<br />GC_Content:  23<br />Percent: 0.30","Filename: SRR11951229<br />GC_Content:  24<br />Percent: 0.41","Filename: SRR11951229<br />GC_Content:  25<br />Percent: 0.50","Filename: SRR11951229<br />GC_Content:  26<br />Percent: 0.65","Filename: SRR11951229<br />GC_Content:  27<br />Percent: 0.92","Filename: SRR11951229<br />GC_Content:  28<br />Percent: 1.23","Filename: SRR11951229<br />GC_Content:  29<br />Percent: 1.44","Filename: SRR11951229<br />GC_Content:  30<br />Percent: 1.67","Filename: SRR11951229<br />GC_Content:  31<br />Percent: 2.05","Filename: SRR11951229<br />GC_Content:  32<br />Percent: 2.41","Filename: SRR11951229<br />GC_Content:  33<br />Percent: 2.62","Filename: SRR11951229<br />GC_Content:  34<br />Percent: 2.91","Filename: SRR11951229<br />GC_Content:  35<br />Percent: 3.33","Filename: SRR11951229<br />GC_Content:  36<br />Percent: 3.67","Filename: SRR11951229<br />GC_Content:  37<br />Percent: 3.86","Filename: SRR11951229<br />GC_Content:  38<br />Percent: 4.05","Filename: SRR11951229<br />GC_Content:  39<br />Percent: 4.13","Filename: SRR11951229<br />GC_Content:  40<br />Percent: 4.07","Filename: SRR11951229<br />GC_Content:  41<br />Percent: 4.03","Filename: SRR11951229<br />GC_Content:  42<br />Percent: 3.96","Filename: SRR11951229<br />GC_Content:  43<br />Percent: 3.82","Filename: SRR11951229<br />GC_Content:  44<br />Percent: 3.61","Filename: SRR11951229<br />GC_Content:  45<br />Percent: 3.54","Filename: SRR11951229<br />GC_Content:  46<br />Percent: 3.56","Filename: SRR11951229<br />GC_Content:  47<br />Percent: 3.42","Filename: SRR11951229<br />GC_Content:  48<br />Percent: 3.22","Filename: SRR11951229<br />GC_Content:  49<br />Percent: 3.07","Filename: SRR11951229<br />GC_Content:  50<br />Percent: 2.94","Filename: SRR11951229<br />GC_Content:  51<br />Percent: 2.80","Filename: SRR11951229<br />GC_Content:  52<br />Percent: 2.59","Filename: SRR11951229<br />GC_Content:  53<br />Percent: 2.40","Filename: SRR11951229<br />GC_Content:  54<br />Percent: 2.29","Filename: SRR11951229<br />GC_Content:  55<br />Percent: 2.12","Filename: SRR11951229<br />GC_Content:  56<br />Percent: 1.86","Filename: SRR11951229<br />GC_Content:  57<br />Percent: 1.60","Filename: SRR11951229<br />GC_Content:  58<br />Percent: 1.42","Filename: SRR11951229<br />GC_Content:  59<br />Percent: 1.24","Filename: SRR11951229<br />GC_Content:  60<br />Percent: 1.02","Filename: SRR11951229<br />GC_Content:  61<br />Percent: 0.83","Filename: SRR11951229<br />GC_Content:  62<br />Percent: 0.69","Filename: SRR11951229<br />GC_Content:  63<br />Percent: 0.58","Filename: SRR11951229<br />GC_Content:  64<br />Percent: 0.46","Filename: SRR11951229<br />GC_Content:  65<br />Percent: 0.36","Filename: SRR11951229<br />GC_Content:  66<br />Percent: 0.26","Filename: SRR11951229<br />GC_Content:  67<br />Percent: 0.21","Filename: SRR11951229<br />GC_Content:  68<br />Percent: 0.18","Filename: SRR11951229<br />GC_Content:  69<br />Percent: 0.14","Filename: SRR11951229<br />GC_Content:  70<br />Percent: 0.11","Filename: SRR11951229<br />GC_Content:  71<br />Percent: 0.09","Filename: SRR11951229<br />GC_Content:  72<br />Percent: 0.08","Filename: SRR11951229<br />GC_Content:  73<br />Percent: 0.06","Filename: SRR11951229<br />GC_Content:  74<br />Percent: 0.05","Filename: SRR11951229<br />GC_Content:  75<br />Percent: 0.04","Filename: SRR11951229<br />GC_Content:  76<br />Percent: 0.03","Filename: SRR11951229<br />GC_Content:  77<br />Percent: 0.03","Filename: SRR11951229<br />GC_Content:  78<br />Percent: 0.02","Filename: SRR11951229<br />GC_Content:  79<br />Percent: 0.02","Filename: SRR11951229<br />GC_Content:  80<br />Percent: 0.02","Filename: SRR11951229<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951229<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951229<br />GC_Content:  83<br />Percent: 0.01","Filename: SRR11951229<br />GC_Content:  84<br />Percent: 0.01","Filename: SRR11951229<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:  94<br />Percent: 0.01","Filename: SRR11951229<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951229<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951229<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(101,164,204,1)","dash":"solid"},"hoveron":"points","name":"SRR11951229","legendgroup":"SRR11951229","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.01,0.01,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.06,0.09,0.14,0.17,0.21,0.31,0.43,0.52,0.66,0.92,1.19,1.35,1.53,1.83,2.09,2.24,2.42,2.69,2.91,3.06,3.22,3.33,3.37,3.42,3.44,3.43,3.41,3.48,3.59,3.57,3.52,3.48,3.44,3.39,3.25,3.05,2.91,2.71,2.4,2.11,1.89,1.67,1.38,1.12,0.93,0.79,0.62,0.48,0.35,0.27,0.23,0.17,0.12,0.09,0.08,0.06,0.04,0.03,0.03,0.02,0.01,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951230<br />GC_Content:   0<br />Percent: 0.01","Filename: SRR11951230<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951230<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951230<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951230<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951230<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951230<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951230<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951230<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951230<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951230<br />GC_Content:  18<br />Percent: 0.06","Filename: SRR11951230<br />GC_Content:  19<br />Percent: 0.09","Filename: SRR11951230<br />GC_Content:  20<br />Percent: 0.14","Filename: SRR11951230<br />GC_Content:  21<br />Percent: 0.17","Filename: SRR11951230<br />GC_Content:  22<br />Percent: 0.21","Filename: SRR11951230<br />GC_Content:  23<br />Percent: 0.31","Filename: SRR11951230<br />GC_Content:  24<br />Percent: 0.43","Filename: SRR11951230<br />GC_Content:  25<br />Percent: 0.52","Filename: SRR11951230<br />GC_Content:  26<br />Percent: 0.66","Filename: SRR11951230<br />GC_Content:  27<br />Percent: 0.92","Filename: SRR11951230<br />GC_Content:  28<br />Percent: 1.19","Filename: SRR11951230<br />GC_Content:  29<br />Percent: 1.35","Filename: SRR11951230<br />GC_Content:  30<br />Percent: 1.53","Filename: SRR11951230<br />GC_Content:  31<br />Percent: 1.83","Filename: SRR11951230<br />GC_Content:  32<br />Percent: 2.09","Filename: SRR11951230<br />GC_Content:  33<br />Percent: 2.24","Filename: SRR11951230<br />GC_Content:  34<br />Percent: 2.42","Filename: SRR11951230<br />GC_Content:  35<br />Percent: 2.69","Filename: SRR11951230<br />GC_Content:  36<br />Percent: 2.91","Filename: SRR11951230<br />GC_Content:  37<br />Percent: 3.06","Filename: SRR11951230<br />GC_Content:  38<br />Percent: 3.22","Filename: SRR11951230<br />GC_Content:  39<br />Percent: 3.33","Filename: SRR11951230<br />GC_Content:  40<br />Percent: 3.37","Filename: SRR11951230<br />GC_Content:  41<br />Percent: 3.42","Filename: SRR11951230<br />GC_Content:  42<br />Percent: 3.44","Filename: SRR11951230<br />GC_Content:  43<br />Percent: 3.43","Filename: SRR11951230<br />GC_Content:  44<br />Percent: 3.41","Filename: SRR11951230<br />GC_Content:  45<br />Percent: 3.48","Filename: SRR11951230<br />GC_Content:  46<br />Percent: 3.59","Filename: SRR11951230<br />GC_Content:  47<br />Percent: 3.57","Filename: SRR11951230<br />GC_Content:  48<br />Percent: 3.52","Filename: SRR11951230<br />GC_Content:  49<br />Percent: 3.48","Filename: SRR11951230<br />GC_Content:  50<br />Percent: 3.44","Filename: SRR11951230<br />GC_Content:  51<br />Percent: 3.39","Filename: SRR11951230<br />GC_Content:  52<br />Percent: 3.25","Filename: SRR11951230<br />GC_Content:  53<br />Percent: 3.05","Filename: SRR11951230<br />GC_Content:  54<br />Percent: 2.91","Filename: SRR11951230<br />GC_Content:  55<br />Percent: 2.71","Filename: SRR11951230<br />GC_Content:  56<br />Percent: 2.40","Filename: SRR11951230<br />GC_Content:  57<br />Percent: 2.11","Filename: SRR11951230<br />GC_Content:  58<br />Percent: 1.89","Filename: SRR11951230<br />GC_Content:  59<br />Percent: 1.67","Filename: SRR11951230<br />GC_Content:  60<br />Percent: 1.38","Filename: SRR11951230<br />GC_Content:  61<br />Percent: 1.12","Filename: SRR11951230<br />GC_Content:  62<br />Percent: 0.93","Filename: SRR11951230<br />GC_Content:  63<br />Percent: 0.79","Filename: SRR11951230<br />GC_Content:  64<br />Percent: 0.62","Filename: SRR11951230<br />GC_Content:  65<br />Percent: 0.48","Filename: SRR11951230<br />GC_Content:  66<br />Percent: 0.35","Filename: SRR11951230<br />GC_Content:  67<br />Percent: 0.27","Filename: SRR11951230<br />GC_Content:  68<br />Percent: 0.23","Filename: SRR11951230<br />GC_Content:  69<br />Percent: 0.17","Filename: SRR11951230<br />GC_Content:  70<br />Percent: 0.12","Filename: SRR11951230<br />GC_Content:  71<br />Percent: 0.09","Filename: SRR11951230<br />GC_Content:  72<br />Percent: 0.08","Filename: SRR11951230<br />GC_Content:  73<br />Percent: 0.06","Filename: SRR11951230<br />GC_Content:  74<br />Percent: 0.04","Filename: SRR11951230<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951230<br />GC_Content:  76<br />Percent: 0.03","Filename: SRR11951230<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951230<br />GC_Content:  78<br />Percent: 0.01","Filename: SRR11951230<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951230<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951230<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951230<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951230<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  94<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951230<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951230<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(36,123,182,1)","dash":"solid"},"hoveron":"points","name":"SRR11951230","legendgroup":"SRR11951230","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.01,0.01,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.06,0.09,0.13,0.16,0.2,0.29,0.4,0.48,0.63,0.9,1.18,1.36,1.56,1.9,2.21,2.38,2.61,2.95,3.24,3.4,3.57,3.68,3.68,3.7,3.68,3.61,3.52,3.54,3.62,3.55,3.43,3.34,3.26,3.15,2.98,2.77,2.64,2.45,2.17,1.89,1.69,1.49,1.23,0.99,0.83,0.7,0.55,0.43,0.31,0.25,0.21,0.16,0.11,0.09,0.07,0.06,0.04,0.03,0.03,0.02,0.02,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0.01,0,0,0,0,0.01,0.02],"text":["Filename: SRR11951231<br />GC_Content:   0<br />Percent: 0.01","Filename: SRR11951231<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951231<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951231<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951231<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951231<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951231<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951231<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951231<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951231<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951231<br />GC_Content:  18<br />Percent: 0.06","Filename: SRR11951231<br />GC_Content:  19<br />Percent: 0.09","Filename: SRR11951231<br />GC_Content:  20<br />Percent: 0.13","Filename: SRR11951231<br />GC_Content:  21<br />Percent: 0.16","Filename: SRR11951231<br />GC_Content:  22<br />Percent: 0.20","Filename: SRR11951231<br />GC_Content:  23<br />Percent: 0.29","Filename: SRR11951231<br />GC_Content:  24<br />Percent: 0.40","Filename: SRR11951231<br />GC_Content:  25<br />Percent: 0.48","Filename: SRR11951231<br />GC_Content:  26<br />Percent: 0.63","Filename: SRR11951231<br />GC_Content:  27<br />Percent: 0.90","Filename: SRR11951231<br />GC_Content:  28<br />Percent: 1.18","Filename: SRR11951231<br />GC_Content:  29<br />Percent: 1.36","Filename: SRR11951231<br />GC_Content:  30<br />Percent: 1.56","Filename: SRR11951231<br />GC_Content:  31<br />Percent: 1.90","Filename: SRR11951231<br />GC_Content:  32<br />Percent: 2.21","Filename: SRR11951231<br />GC_Content:  33<br />Percent: 2.38","Filename: SRR11951231<br />GC_Content:  34<br />Percent: 2.61","Filename: SRR11951231<br />GC_Content:  35<br />Percent: 2.95","Filename: SRR11951231<br />GC_Content:  36<br />Percent: 3.24","Filename: SRR11951231<br />GC_Content:  37<br />Percent: 3.40","Filename: SRR11951231<br />GC_Content:  38<br />Percent: 3.57","Filename: SRR11951231<br />GC_Content:  39<br />Percent: 3.68","Filename: SRR11951231<br />GC_Content:  40<br />Percent: 3.68","Filename: SRR11951231<br />GC_Content:  41<br />Percent: 3.70","Filename: SRR11951231<br />GC_Content:  42<br />Percent: 3.68","Filename: SRR11951231<br />GC_Content:  43<br />Percent: 3.61","Filename: SRR11951231<br />GC_Content:  44<br />Percent: 3.52","Filename: SRR11951231<br />GC_Content:  45<br />Percent: 3.54","Filename: SRR11951231<br />GC_Content:  46<br />Percent: 3.62","Filename: SRR11951231<br />GC_Content:  47<br />Percent: 3.55","Filename: SRR11951231<br />GC_Content:  48<br />Percent: 3.43","Filename: SRR11951231<br />GC_Content:  49<br />Percent: 3.34","Filename: SRR11951231<br />GC_Content:  50<br />Percent: 3.26","Filename: SRR11951231<br />GC_Content:  51<br />Percent: 3.15","Filename: SRR11951231<br />GC_Content:  52<br />Percent: 2.98","Filename: SRR11951231<br />GC_Content:  53<br />Percent: 2.77","Filename: SRR11951231<br />GC_Content:  54<br />Percent: 2.64","Filename: SRR11951231<br />GC_Content:  55<br />Percent: 2.45","Filename: SRR11951231<br />GC_Content:  56<br />Percent: 2.17","Filename: SRR11951231<br />GC_Content:  57<br />Percent: 1.89","Filename: SRR11951231<br />GC_Content:  58<br />Percent: 1.69","Filename: SRR11951231<br />GC_Content:  59<br />Percent: 1.49","Filename: SRR11951231<br />GC_Content:  60<br />Percent: 1.23","Filename: SRR11951231<br />GC_Content:  61<br />Percent: 0.99","Filename: SRR11951231<br />GC_Content:  62<br />Percent: 0.83","Filename: SRR11951231<br />GC_Content:  63<br />Percent: 0.70","Filename: SRR11951231<br />GC_Content:  64<br />Percent: 0.55","Filename: SRR11951231<br />GC_Content:  65<br />Percent: 0.43","Filename: SRR11951231<br />GC_Content:  66<br />Percent: 0.31","Filename: SRR11951231<br />GC_Content:  67<br />Percent: 0.25","Filename: SRR11951231<br />GC_Content:  68<br />Percent: 0.21","Filename: SRR11951231<br />GC_Content:  69<br />Percent: 0.16","Filename: SRR11951231<br />GC_Content:  70<br />Percent: 0.11","Filename: SRR11951231<br />GC_Content:  71<br />Percent: 0.09","Filename: SRR11951231<br />GC_Content:  72<br />Percent: 0.07","Filename: SRR11951231<br />GC_Content:  73<br />Percent: 0.06","Filename: SRR11951231<br />GC_Content:  74<br />Percent: 0.04","Filename: SRR11951231<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951231<br />GC_Content:  76<br />Percent: 0.03","Filename: SRR11951231<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951231<br />GC_Content:  78<br />Percent: 0.02","Filename: SRR11951231<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951231<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951231<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951231<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951231<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  94<br />Percent: 0.01","Filename: SRR11951231<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951231<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951231<br />GC_Content: 100<br />Percent: 0.02"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(94,164,161,1)","dash":"solid"},"hoveron":"points","name":"SRR11951231","legendgroup":"SRR11951231","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.01,0.01,0,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.06,0.08,0.12,0.15,0.2,0.28,0.39,0.47,0.61,0.86,1.11,1.26,1.43,1.71,1.96,2.1,2.27,2.53,2.78,2.95,3.13,3.26,3.31,3.4,3.44,3.45,3.45,3.54,3.67,3.67,3.64,3.63,3.59,3.53,3.37,3.17,3.02,2.81,2.49,2.19,1.96,1.72,1.42,1.15,0.96,0.81,0.63,0.49,0.36,0.28,0.23,0.17,0.12,0.09,0.08,0.06,0.04,0.03,0.02,0.02,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951232<br />GC_Content:   0<br />Percent: 0.01","Filename: SRR11951232<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951232<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  10<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951232<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951232<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951232<br />GC_Content:  14<br />Percent: 0.01","Filename: SRR11951232<br />GC_Content:  15<br />Percent: 0.02","Filename: SRR11951232<br />GC_Content:  16<br />Percent: 0.03","Filename: SRR11951232<br />GC_Content:  17<br />Percent: 0.04","Filename: SRR11951232<br />GC_Content:  18<br />Percent: 0.06","Filename: SRR11951232<br />GC_Content:  19<br />Percent: 0.08","Filename: SRR11951232<br />GC_Content:  20<br />Percent: 0.12","Filename: SRR11951232<br />GC_Content:  21<br />Percent: 0.15","Filename: SRR11951232<br />GC_Content:  22<br />Percent: 0.20","Filename: SRR11951232<br />GC_Content:  23<br />Percent: 0.28","Filename: SRR11951232<br />GC_Content:  24<br />Percent: 0.39","Filename: SRR11951232<br />GC_Content:  25<br />Percent: 0.47","Filename: SRR11951232<br />GC_Content:  26<br />Percent: 0.61","Filename: SRR11951232<br />GC_Content:  27<br />Percent: 0.86","Filename: SRR11951232<br />GC_Content:  28<br />Percent: 1.11","Filename: SRR11951232<br />GC_Content:  29<br />Percent: 1.26","Filename: SRR11951232<br />GC_Content:  30<br />Percent: 1.43","Filename: SRR11951232<br />GC_Content:  31<br />Percent: 1.71","Filename: SRR11951232<br />GC_Content:  32<br />Percent: 1.96","Filename: SRR11951232<br />GC_Content:  33<br />Percent: 2.10","Filename: SRR11951232<br />GC_Content:  34<br />Percent: 2.27","Filename: SRR11951232<br />GC_Content:  35<br />Percent: 2.53","Filename: SRR11951232<br />GC_Content:  36<br />Percent: 2.78","Filename: SRR11951232<br />GC_Content:  37<br />Percent: 2.95","Filename: SRR11951232<br />GC_Content:  38<br />Percent: 3.13","Filename: SRR11951232<br />GC_Content:  39<br />Percent: 3.26","Filename: SRR11951232<br />GC_Content:  40<br />Percent: 3.31","Filename: SRR11951232<br />GC_Content:  41<br />Percent: 3.40","Filename: SRR11951232<br />GC_Content:  42<br />Percent: 3.44","Filename: SRR11951232<br />GC_Content:  43<br />Percent: 3.45","Filename: SRR11951232<br />GC_Content:  44<br />Percent: 3.45","Filename: SRR11951232<br />GC_Content:  45<br />Percent: 3.54","Filename: SRR11951232<br />GC_Content:  46<br />Percent: 3.67","Filename: SRR11951232<br />GC_Content:  47<br />Percent: 3.67","Filename: SRR11951232<br />GC_Content:  48<br />Percent: 3.64","Filename: SRR11951232<br />GC_Content:  49<br />Percent: 3.63","Filename: SRR11951232<br />GC_Content:  50<br />Percent: 3.59","Filename: SRR11951232<br />GC_Content:  51<br />Percent: 3.53","Filename: SRR11951232<br />GC_Content:  52<br />Percent: 3.37","Filename: SRR11951232<br />GC_Content:  53<br />Percent: 3.17","Filename: SRR11951232<br />GC_Content:  54<br />Percent: 3.02","Filename: SRR11951232<br />GC_Content:  55<br />Percent: 2.81","Filename: SRR11951232<br />GC_Content:  56<br />Percent: 2.49","Filename: SRR11951232<br />GC_Content:  57<br />Percent: 2.19","Filename: SRR11951232<br />GC_Content:  58<br />Percent: 1.96","Filename: SRR11951232<br />GC_Content:  59<br />Percent: 1.72","Filename: SRR11951232<br />GC_Content:  60<br />Percent: 1.42","Filename: SRR11951232<br />GC_Content:  61<br />Percent: 1.15","Filename: SRR11951232<br />GC_Content:  62<br />Percent: 0.96","Filename: SRR11951232<br />GC_Content:  63<br />Percent: 0.81","Filename: SRR11951232<br />GC_Content:  64<br />Percent: 0.63","Filename: SRR11951232<br />GC_Content:  65<br />Percent: 0.49","Filename: SRR11951232<br />GC_Content:  66<br />Percent: 0.36","Filename: SRR11951232<br />GC_Content:  67<br />Percent: 0.28","Filename: SRR11951232<br />GC_Content:  68<br />Percent: 0.23","Filename: SRR11951232<br />GC_Content:  69<br />Percent: 0.17","Filename: SRR11951232<br />GC_Content:  70<br />Percent: 0.12","Filename: SRR11951232<br />GC_Content:  71<br />Percent: 0.09","Filename: SRR11951232<br />GC_Content:  72<br />Percent: 0.08","Filename: SRR11951232<br />GC_Content:  73<br />Percent: 0.06","Filename: SRR11951232<br />GC_Content:  74<br />Percent: 0.04","Filename: SRR11951232<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951232<br />GC_Content:  76<br />Percent: 0.02","Filename: SRR11951232<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951232<br />GC_Content:  78<br />Percent: 0.01","Filename: SRR11951232<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951232<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951232<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951232<br />GC_Content:  82<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  94<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951232<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951232<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(165,214,141,1)","dash":"solid"},"hoveron":"points","name":"SRR11951232","legendgroup":"SRR11951232","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.01,0.01,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.06,0.1,0.14,0.17,0.21,0.31,0.43,0.52,0.67,0.95,1.25,1.43,1.65,2.01,2.33,2.52,2.75,3.09,3.36,3.51,3.66,3.75,3.74,3.73,3.7,3.62,3.52,3.5,3.55,3.47,3.33,3.23,3.14,3.04,2.87,2.68,2.54,2.35,2.07,1.81,1.61,1.42,1.18,0.95,0.79,0.67,0.53,0.41,0.3,0.24,0.2,0.15,0.11,0.09,0.07,0.05,0.04,0.03,0.03,0.02,0.02,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0.01,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951233<br />GC_Content:   0<br />Percent: 0.01","Filename: SRR11951233<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951233<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951233<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951233<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951233<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951233<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951233<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951233<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951233<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951233<br />GC_Content:  18<br />Percent: 0.06","Filename: SRR11951233<br />GC_Content:  19<br />Percent: 0.10","Filename: SRR11951233<br />GC_Content:  20<br />Percent: 0.14","Filename: SRR11951233<br />GC_Content:  21<br />Percent: 0.17","Filename: SRR11951233<br />GC_Content:  22<br />Percent: 0.21","Filename: SRR11951233<br />GC_Content:  23<br />Percent: 0.31","Filename: SRR11951233<br />GC_Content:  24<br />Percent: 0.43","Filename: SRR11951233<br />GC_Content:  25<br />Percent: 0.52","Filename: SRR11951233<br />GC_Content:  26<br />Percent: 0.67","Filename: SRR11951233<br />GC_Content:  27<br />Percent: 0.95","Filename: SRR11951233<br />GC_Content:  28<br />Percent: 1.25","Filename: SRR11951233<br />GC_Content:  29<br />Percent: 1.43","Filename: SRR11951233<br />GC_Content:  30<br />Percent: 1.65","Filename: SRR11951233<br />GC_Content:  31<br />Percent: 2.01","Filename: SRR11951233<br />GC_Content:  32<br />Percent: 2.33","Filename: SRR11951233<br />GC_Content:  33<br />Percent: 2.52","Filename: SRR11951233<br />GC_Content:  34<br />Percent: 2.75","Filename: SRR11951233<br />GC_Content:  35<br />Percent: 3.09","Filename: SRR11951233<br />GC_Content:  36<br />Percent: 3.36","Filename: SRR11951233<br />GC_Content:  37<br />Percent: 3.51","Filename: SRR11951233<br />GC_Content:  38<br />Percent: 3.66","Filename: SRR11951233<br />GC_Content:  39<br />Percent: 3.75","Filename: SRR11951233<br />GC_Content:  40<br />Percent: 3.74","Filename: SRR11951233<br />GC_Content:  41<br />Percent: 3.73","Filename: SRR11951233<br />GC_Content:  42<br />Percent: 3.70","Filename: SRR11951233<br />GC_Content:  43<br />Percent: 3.62","Filename: SRR11951233<br />GC_Content:  44<br />Percent: 3.52","Filename: SRR11951233<br />GC_Content:  45<br />Percent: 3.50","Filename: SRR11951233<br />GC_Content:  46<br />Percent: 3.55","Filename: SRR11951233<br />GC_Content:  47<br />Percent: 3.47","Filename: SRR11951233<br />GC_Content:  48<br />Percent: 3.33","Filename: SRR11951233<br />GC_Content:  49<br />Percent: 3.23","Filename: SRR11951233<br />GC_Content:  50<br />Percent: 3.14","Filename: SRR11951233<br />GC_Content:  51<br />Percent: 3.04","Filename: SRR11951233<br />GC_Content:  52<br />Percent: 2.87","Filename: SRR11951233<br />GC_Content:  53<br />Percent: 2.68","Filename: SRR11951233<br />GC_Content:  54<br />Percent: 2.54","Filename: SRR11951233<br />GC_Content:  55<br />Percent: 2.35","Filename: SRR11951233<br />GC_Content:  56<br />Percent: 2.07","Filename: SRR11951233<br />GC_Content:  57<br />Percent: 1.81","Filename: SRR11951233<br />GC_Content:  58<br />Percent: 1.61","Filename: SRR11951233<br />GC_Content:  59<br />Percent: 1.42","Filename: SRR11951233<br />GC_Content:  60<br />Percent: 1.18","Filename: SRR11951233<br />GC_Content:  61<br />Percent: 0.95","Filename: SRR11951233<br />GC_Content:  62<br />Percent: 0.79","Filename: SRR11951233<br />GC_Content:  63<br />Percent: 0.67","Filename: SRR11951233<br />GC_Content:  64<br />Percent: 0.53","Filename: SRR11951233<br />GC_Content:  65<br />Percent: 0.41","Filename: SRR11951233<br />GC_Content:  66<br />Percent: 0.30","Filename: SRR11951233<br />GC_Content:  67<br />Percent: 0.24","Filename: SRR11951233<br />GC_Content:  68<br />Percent: 0.20","Filename: SRR11951233<br />GC_Content:  69<br />Percent: 0.15","Filename: SRR11951233<br />GC_Content:  70<br />Percent: 0.11","Filename: SRR11951233<br />GC_Content:  71<br />Percent: 0.09","Filename: SRR11951233<br />GC_Content:  72<br />Percent: 0.07","Filename: SRR11951233<br />GC_Content:  73<br />Percent: 0.05","Filename: SRR11951233<br />GC_Content:  74<br />Percent: 0.04","Filename: SRR11951233<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951233<br />GC_Content:  76<br />Percent: 0.03","Filename: SRR11951233<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951233<br />GC_Content:  78<br />Percent: 0.02","Filename: SRR11951233<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951233<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951233<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951233<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951233<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  94<br />Percent: 0.01","Filename: SRR11951233<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951233<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951233<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(128,198,101,1)","dash":"solid"},"hoveron":"points","name":"SRR11951233","legendgroup":"SRR11951233","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.01,0.01,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.06,0.1,0.14,0.17,0.21,0.31,0.43,0.51,0.66,0.95,1.25,1.45,1.67,2.03,2.37,2.56,2.81,3.18,3.47,3.64,3.81,3.9,3.86,3.85,3.8,3.7,3.56,3.53,3.57,3.46,3.29,3.17,3.06,2.95,2.75,2.55,2.43,2.25,1.98,1.72,1.52,1.34,1.11,0.9,0.75,0.64,0.5,0.39,0.29,0.23,0.19,0.15,0.11,0.09,0.08,0.06,0.05,0.04,0.03,0.03,0.02,0.02,0.02,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0.01,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951234<br />GC_Content:   0<br />Percent: 0.01","Filename: SRR11951234<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951234<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951234<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951234<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951234<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951234<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951234<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951234<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951234<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951234<br />GC_Content:  18<br />Percent: 0.06","Filename: SRR11951234<br />GC_Content:  19<br />Percent: 0.10","Filename: SRR11951234<br />GC_Content:  20<br />Percent: 0.14","Filename: SRR11951234<br />GC_Content:  21<br />Percent: 0.17","Filename: SRR11951234<br />GC_Content:  22<br />Percent: 0.21","Filename: SRR11951234<br />GC_Content:  23<br />Percent: 0.31","Filename: SRR11951234<br />GC_Content:  24<br />Percent: 0.43","Filename: SRR11951234<br />GC_Content:  25<br />Percent: 0.51","Filename: SRR11951234<br />GC_Content:  26<br />Percent: 0.66","Filename: SRR11951234<br />GC_Content:  27<br />Percent: 0.95","Filename: SRR11951234<br />GC_Content:  28<br />Percent: 1.25","Filename: SRR11951234<br />GC_Content:  29<br />Percent: 1.45","Filename: SRR11951234<br />GC_Content:  30<br />Percent: 1.67","Filename: SRR11951234<br />GC_Content:  31<br />Percent: 2.03","Filename: SRR11951234<br />GC_Content:  32<br />Percent: 2.37","Filename: SRR11951234<br />GC_Content:  33<br />Percent: 2.56","Filename: SRR11951234<br />GC_Content:  34<br />Percent: 2.81","Filename: SRR11951234<br />GC_Content:  35<br />Percent: 3.18","Filename: SRR11951234<br />GC_Content:  36<br />Percent: 3.47","Filename: SRR11951234<br />GC_Content:  37<br />Percent: 3.64","Filename: SRR11951234<br />GC_Content:  38<br />Percent: 3.81","Filename: SRR11951234<br />GC_Content:  39<br />Percent: 3.90","Filename: SRR11951234<br />GC_Content:  40<br />Percent: 3.86","Filename: SRR11951234<br />GC_Content:  41<br />Percent: 3.85","Filename: SRR11951234<br />GC_Content:  42<br />Percent: 3.80","Filename: SRR11951234<br />GC_Content:  43<br />Percent: 3.70","Filename: SRR11951234<br />GC_Content:  44<br />Percent: 3.56","Filename: SRR11951234<br />GC_Content:  45<br />Percent: 3.53","Filename: SRR11951234<br />GC_Content:  46<br />Percent: 3.57","Filename: SRR11951234<br />GC_Content:  47<br />Percent: 3.46","Filename: SRR11951234<br />GC_Content:  48<br />Percent: 3.29","Filename: SRR11951234<br />GC_Content:  49<br />Percent: 3.17","Filename: SRR11951234<br />GC_Content:  50<br />Percent: 3.06","Filename: SRR11951234<br />GC_Content:  51<br />Percent: 2.95","Filename: SRR11951234<br />GC_Content:  52<br />Percent: 2.75","Filename: SRR11951234<br />GC_Content:  53<br />Percent: 2.55","Filename: SRR11951234<br />GC_Content:  54<br />Percent: 2.43","Filename: SRR11951234<br />GC_Content:  55<br />Percent: 2.25","Filename: SRR11951234<br />GC_Content:  56<br />Percent: 1.98","Filename: SRR11951234<br />GC_Content:  57<br />Percent: 1.72","Filename: SRR11951234<br />GC_Content:  58<br />Percent: 1.52","Filename: SRR11951234<br />GC_Content:  59<br />Percent: 1.34","Filename: SRR11951234<br />GC_Content:  60<br />Percent: 1.11","Filename: SRR11951234<br />GC_Content:  61<br />Percent: 0.90","Filename: SRR11951234<br />GC_Content:  62<br />Percent: 0.75","Filename: SRR11951234<br />GC_Content:  63<br />Percent: 0.64","Filename: SRR11951234<br />GC_Content:  64<br />Percent: 0.50","Filename: SRR11951234<br />GC_Content:  65<br />Percent: 0.39","Filename: SRR11951234<br />GC_Content:  66<br />Percent: 0.29","Filename: SRR11951234<br />GC_Content:  67<br />Percent: 0.23","Filename: SRR11951234<br />GC_Content:  68<br />Percent: 0.19","Filename: SRR11951234<br />GC_Content:  69<br />Percent: 0.15","Filename: SRR11951234<br />GC_Content:  70<br />Percent: 0.11","Filename: SRR11951234<br />GC_Content:  71<br />Percent: 0.09","Filename: SRR11951234<br />GC_Content:  72<br />Percent: 0.08","Filename: SRR11951234<br />GC_Content:  73<br />Percent: 0.06","Filename: SRR11951234<br />GC_Content:  74<br />Percent: 0.05","Filename: SRR11951234<br />GC_Content:  75<br />Percent: 0.04","Filename: SRR11951234<br />GC_Content:  76<br />Percent: 0.03","Filename: SRR11951234<br />GC_Content:  77<br />Percent: 0.03","Filename: SRR11951234<br />GC_Content:  78<br />Percent: 0.02","Filename: SRR11951234<br />GC_Content:  79<br />Percent: 0.02","Filename: SRR11951234<br />GC_Content:  80<br />Percent: 0.02","Filename: SRR11951234<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951234<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951234<br />GC_Content:  83<br />Percent: 0.01","Filename: SRR11951234<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:  94<br />Percent: 0.01","Filename: SRR11951234<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951234<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951234<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(67,168,56,1)","dash":"solid"},"hoveron":"points","name":"SRR11951234","legendgroup":"SRR11951234","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.01,0.01,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.06,0.09,0.13,0.16,0.21,0.3,0.41,0.49,0.62,0.86,1.12,1.27,1.45,1.74,2.01,2.16,2.34,2.6,2.83,2.96,3.11,3.23,3.28,3.35,3.39,3.41,3.43,3.5,3.61,3.62,3.59,3.56,3.53,3.48,3.34,3.14,2.99,2.79,2.49,2.18,1.95,1.73,1.44,1.17,0.98,0.82,0.65,0.51,0.37,0.29,0.24,0.18,0.13,0.1,0.08,0.06,0.04,0.03,0.03,0.02,0.02,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951235<br />GC_Content:   0<br />Percent: 0.01","Filename: SRR11951235<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951235<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951235<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951235<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951235<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951235<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951235<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951235<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951235<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951235<br />GC_Content:  18<br />Percent: 0.06","Filename: SRR11951235<br />GC_Content:  19<br />Percent: 0.09","Filename: SRR11951235<br />GC_Content:  20<br />Percent: 0.13","Filename: SRR11951235<br />GC_Content:  21<br />Percent: 0.16","Filename: SRR11951235<br />GC_Content:  22<br />Percent: 0.21","Filename: SRR11951235<br />GC_Content:  23<br />Percent: 0.30","Filename: SRR11951235<br />GC_Content:  24<br />Percent: 0.41","Filename: SRR11951235<br />GC_Content:  25<br />Percent: 0.49","Filename: SRR11951235<br />GC_Content:  26<br />Percent: 0.62","Filename: SRR11951235<br />GC_Content:  27<br />Percent: 0.86","Filename: SRR11951235<br />GC_Content:  28<br />Percent: 1.12","Filename: SRR11951235<br />GC_Content:  29<br />Percent: 1.27","Filename: SRR11951235<br />GC_Content:  30<br />Percent: 1.45","Filename: SRR11951235<br />GC_Content:  31<br />Percent: 1.74","Filename: SRR11951235<br />GC_Content:  32<br />Percent: 2.01","Filename: SRR11951235<br />GC_Content:  33<br />Percent: 2.16","Filename: SRR11951235<br />GC_Content:  34<br />Percent: 2.34","Filename: SRR11951235<br />GC_Content:  35<br />Percent: 2.60","Filename: SRR11951235<br />GC_Content:  36<br />Percent: 2.83","Filename: SRR11951235<br />GC_Content:  37<br />Percent: 2.96","Filename: SRR11951235<br />GC_Content:  38<br />Percent: 3.11","Filename: SRR11951235<br />GC_Content:  39<br />Percent: 3.23","Filename: SRR11951235<br />GC_Content:  40<br />Percent: 3.28","Filename: SRR11951235<br />GC_Content:  41<br />Percent: 3.35","Filename: SRR11951235<br />GC_Content:  42<br />Percent: 3.39","Filename: SRR11951235<br />GC_Content:  43<br />Percent: 3.41","Filename: SRR11951235<br />GC_Content:  44<br />Percent: 3.43","Filename: SRR11951235<br />GC_Content:  45<br />Percent: 3.50","Filename: SRR11951235<br />GC_Content:  46<br />Percent: 3.61","Filename: SRR11951235<br />GC_Content:  47<br />Percent: 3.62","Filename: SRR11951235<br />GC_Content:  48<br />Percent: 3.59","Filename: SRR11951235<br />GC_Content:  49<br />Percent: 3.56","Filename: SRR11951235<br />GC_Content:  50<br />Percent: 3.53","Filename: SRR11951235<br />GC_Content:  51<br />Percent: 3.48","Filename: SRR11951235<br />GC_Content:  52<br />Percent: 3.34","Filename: SRR11951235<br />GC_Content:  53<br />Percent: 3.14","Filename: SRR11951235<br />GC_Content:  54<br />Percent: 2.99","Filename: SRR11951235<br />GC_Content:  55<br />Percent: 2.79","Filename: SRR11951235<br />GC_Content:  56<br />Percent: 2.49","Filename: SRR11951235<br />GC_Content:  57<br />Percent: 2.18","Filename: SRR11951235<br />GC_Content:  58<br />Percent: 1.95","Filename: SRR11951235<br />GC_Content:  59<br />Percent: 1.73","Filename: SRR11951235<br />GC_Content:  60<br />Percent: 1.44","Filename: SRR11951235<br />GC_Content:  61<br />Percent: 1.17","Filename: SRR11951235<br />GC_Content:  62<br />Percent: 0.98","Filename: SRR11951235<br />GC_Content:  63<br />Percent: 0.82","Filename: SRR11951235<br />GC_Content:  64<br />Percent: 0.65","Filename: SRR11951235<br />GC_Content:  65<br />Percent: 0.51","Filename: SRR11951235<br />GC_Content:  66<br />Percent: 0.37","Filename: SRR11951235<br />GC_Content:  67<br />Percent: 0.29","Filename: SRR11951235<br />GC_Content:  68<br />Percent: 0.24","Filename: SRR11951235<br />GC_Content:  69<br />Percent: 0.18","Filename: SRR11951235<br />GC_Content:  70<br />Percent: 0.13","Filename: SRR11951235<br />GC_Content:  71<br />Percent: 0.10","Filename: SRR11951235<br />GC_Content:  72<br />Percent: 0.08","Filename: SRR11951235<br />GC_Content:  73<br />Percent: 0.06","Filename: SRR11951235<br />GC_Content:  74<br />Percent: 0.04","Filename: SRR11951235<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951235<br />GC_Content:  76<br />Percent: 0.03","Filename: SRR11951235<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951235<br />GC_Content:  78<br />Percent: 0.02","Filename: SRR11951235<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951235<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951235<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951235<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951235<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  94<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951235<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951235<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(120,157,81,1)","dash":"solid"},"hoveron":"points","name":"SRR11951235","legendgroup":"SRR11951235","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.02,0.01,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.07,0.1,0.14,0.16,0.21,0.3,0.42,0.5,0.65,0.92,1.2,1.38,1.58,1.91,2.21,2.38,2.6,2.92,3.19,3.36,3.54,3.64,3.64,3.67,3.66,3.6,3.5,3.52,3.61,3.56,3.44,3.36,3.28,3.18,3.01,2.81,2.67,2.48,2.19,1.91,1.7,1.49,1.23,0.99,0.82,0.69,0.54,0.42,0.31,0.24,0.2,0.15,0.11,0.08,0.07,0.05,0.04,0.03,0.02,0.02,0.01,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0.01,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951236<br />GC_Content:   0<br />Percent: 0.02","Filename: SRR11951236<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951236<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951236<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951236<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951236<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951236<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951236<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951236<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951236<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951236<br />GC_Content:  18<br />Percent: 0.07","Filename: SRR11951236<br />GC_Content:  19<br />Percent: 0.10","Filename: SRR11951236<br />GC_Content:  20<br />Percent: 0.14","Filename: SRR11951236<br />GC_Content:  21<br />Percent: 0.16","Filename: SRR11951236<br />GC_Content:  22<br />Percent: 0.21","Filename: SRR11951236<br />GC_Content:  23<br />Percent: 0.30","Filename: SRR11951236<br />GC_Content:  24<br />Percent: 0.42","Filename: SRR11951236<br />GC_Content:  25<br />Percent: 0.50","Filename: SRR11951236<br />GC_Content:  26<br />Percent: 0.65","Filename: SRR11951236<br />GC_Content:  27<br />Percent: 0.92","Filename: SRR11951236<br />GC_Content:  28<br />Percent: 1.20","Filename: SRR11951236<br />GC_Content:  29<br />Percent: 1.38","Filename: SRR11951236<br />GC_Content:  30<br />Percent: 1.58","Filename: SRR11951236<br />GC_Content:  31<br />Percent: 1.91","Filename: SRR11951236<br />GC_Content:  32<br />Percent: 2.21","Filename: SRR11951236<br />GC_Content:  33<br />Percent: 2.38","Filename: SRR11951236<br />GC_Content:  34<br />Percent: 2.60","Filename: SRR11951236<br />GC_Content:  35<br />Percent: 2.92","Filename: SRR11951236<br />GC_Content:  36<br />Percent: 3.19","Filename: SRR11951236<br />GC_Content:  37<br />Percent: 3.36","Filename: SRR11951236<br />GC_Content:  38<br />Percent: 3.54","Filename: SRR11951236<br />GC_Content:  39<br />Percent: 3.64","Filename: SRR11951236<br />GC_Content:  40<br />Percent: 3.64","Filename: SRR11951236<br />GC_Content:  41<br />Percent: 3.67","Filename: SRR11951236<br />GC_Content:  42<br />Percent: 3.66","Filename: SRR11951236<br />GC_Content:  43<br />Percent: 3.60","Filename: SRR11951236<br />GC_Content:  44<br />Percent: 3.50","Filename: SRR11951236<br />GC_Content:  45<br />Percent: 3.52","Filename: SRR11951236<br />GC_Content:  46<br />Percent: 3.61","Filename: SRR11951236<br />GC_Content:  47<br />Percent: 3.56","Filename: SRR11951236<br />GC_Content:  48<br />Percent: 3.44","Filename: SRR11951236<br />GC_Content:  49<br />Percent: 3.36","Filename: SRR11951236<br />GC_Content:  50<br />Percent: 3.28","Filename: SRR11951236<br />GC_Content:  51<br />Percent: 3.18","Filename: SRR11951236<br />GC_Content:  52<br />Percent: 3.01","Filename: SRR11951236<br />GC_Content:  53<br />Percent: 2.81","Filename: SRR11951236<br />GC_Content:  54<br />Percent: 2.67","Filename: SRR11951236<br />GC_Content:  55<br />Percent: 2.48","Filename: SRR11951236<br />GC_Content:  56<br />Percent: 2.19","Filename: SRR11951236<br />GC_Content:  57<br />Percent: 1.91","Filename: SRR11951236<br />GC_Content:  58<br />Percent: 1.70","Filename: SRR11951236<br />GC_Content:  59<br />Percent: 1.49","Filename: SRR11951236<br />GC_Content:  60<br />Percent: 1.23","Filename: SRR11951236<br />GC_Content:  61<br />Percent: 0.99","Filename: SRR11951236<br />GC_Content:  62<br />Percent: 0.82","Filename: SRR11951236<br />GC_Content:  63<br />Percent: 0.69","Filename: SRR11951236<br />GC_Content:  64<br />Percent: 0.54","Filename: SRR11951236<br />GC_Content:  65<br />Percent: 0.42","Filename: SRR11951236<br />GC_Content:  66<br />Percent: 0.31","Filename: SRR11951236<br />GC_Content:  67<br />Percent: 0.24","Filename: SRR11951236<br />GC_Content:  68<br />Percent: 0.20","Filename: SRR11951236<br />GC_Content:  69<br />Percent: 0.15","Filename: SRR11951236<br />GC_Content:  70<br />Percent: 0.11","Filename: SRR11951236<br />GC_Content:  71<br />Percent: 0.08","Filename: SRR11951236<br />GC_Content:  72<br />Percent: 0.07","Filename: SRR11951236<br />GC_Content:  73<br />Percent: 0.05","Filename: SRR11951236<br />GC_Content:  74<br />Percent: 0.04","Filename: SRR11951236<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951236<br />GC_Content:  76<br />Percent: 0.02","Filename: SRR11951236<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951236<br />GC_Content:  78<br />Percent: 0.01","Filename: SRR11951236<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951236<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951236<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951236<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951236<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  94<br />Percent: 0.01","Filename: SRR11951236<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951236<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951236<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(216,155,134,1)","dash":"solid"},"hoveron":"points","name":"SRR11951236","legendgroup":"SRR11951236","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.01,0.01,0,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.02,0.02,0.03,0.04,0.06,0.08,0.12,0.14,0.18,0.26,0.36,0.43,0.56,0.78,1.01,1.16,1.33,1.62,1.88,2.03,2.21,2.49,2.72,2.87,3.02,3.16,3.23,3.32,3.37,3.4,3.43,3.54,3.66,3.69,3.68,3.67,3.65,3.6,3.46,3.27,3.11,2.9,2.59,2.27,2.03,1.79,1.49,1.21,1.02,0.86,0.68,0.53,0.39,0.3,0.25,0.19,0.14,0.11,0.09,0.07,0.06,0.05,0.04,0.03,0.02,0.02,0.02,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951237<br />GC_Content:   0<br />Percent: 0.01","Filename: SRR11951237<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951237<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  10<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951237<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951237<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951237<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951237<br />GC_Content:  15<br />Percent: 0.02","Filename: SRR11951237<br />GC_Content:  16<br />Percent: 0.03","Filename: SRR11951237<br />GC_Content:  17<br />Percent: 0.04","Filename: SRR11951237<br />GC_Content:  18<br />Percent: 0.06","Filename: SRR11951237<br />GC_Content:  19<br />Percent: 0.08","Filename: SRR11951237<br />GC_Content:  20<br />Percent: 0.12","Filename: SRR11951237<br />GC_Content:  21<br />Percent: 0.14","Filename: SRR11951237<br />GC_Content:  22<br />Percent: 0.18","Filename: SRR11951237<br />GC_Content:  23<br />Percent: 0.26","Filename: SRR11951237<br />GC_Content:  24<br />Percent: 0.36","Filename: SRR11951237<br />GC_Content:  25<br />Percent: 0.43","Filename: SRR11951237<br />GC_Content:  26<br />Percent: 0.56","Filename: SRR11951237<br />GC_Content:  27<br />Percent: 0.78","Filename: SRR11951237<br />GC_Content:  28<br />Percent: 1.01","Filename: SRR11951237<br />GC_Content:  29<br />Percent: 1.16","Filename: SRR11951237<br />GC_Content:  30<br />Percent: 1.33","Filename: SRR11951237<br />GC_Content:  31<br />Percent: 1.62","Filename: SRR11951237<br />GC_Content:  32<br />Percent: 1.88","Filename: SRR11951237<br />GC_Content:  33<br />Percent: 2.03","Filename: SRR11951237<br />GC_Content:  34<br />Percent: 2.21","Filename: SRR11951237<br />GC_Content:  35<br />Percent: 2.49","Filename: SRR11951237<br />GC_Content:  36<br />Percent: 2.72","Filename: SRR11951237<br />GC_Content:  37<br />Percent: 2.87","Filename: SRR11951237<br />GC_Content:  38<br />Percent: 3.02","Filename: SRR11951237<br />GC_Content:  39<br />Percent: 3.16","Filename: SRR11951237<br />GC_Content:  40<br />Percent: 3.23","Filename: SRR11951237<br />GC_Content:  41<br />Percent: 3.32","Filename: SRR11951237<br />GC_Content:  42<br />Percent: 3.37","Filename: SRR11951237<br />GC_Content:  43<br />Percent: 3.40","Filename: SRR11951237<br />GC_Content:  44<br />Percent: 3.43","Filename: SRR11951237<br />GC_Content:  45<br />Percent: 3.54","Filename: SRR11951237<br />GC_Content:  46<br />Percent: 3.66","Filename: SRR11951237<br />GC_Content:  47<br />Percent: 3.69","Filename: SRR11951237<br />GC_Content:  48<br />Percent: 3.68","Filename: SRR11951237<br />GC_Content:  49<br />Percent: 3.67","Filename: SRR11951237<br />GC_Content:  50<br />Percent: 3.65","Filename: SRR11951237<br />GC_Content:  51<br />Percent: 3.60","Filename: SRR11951237<br />GC_Content:  52<br />Percent: 3.46","Filename: SRR11951237<br />GC_Content:  53<br />Percent: 3.27","Filename: SRR11951237<br />GC_Content:  54<br />Percent: 3.11","Filename: SRR11951237<br />GC_Content:  55<br />Percent: 2.90","Filename: SRR11951237<br />GC_Content:  56<br />Percent: 2.59","Filename: SRR11951237<br />GC_Content:  57<br />Percent: 2.27","Filename: SRR11951237<br />GC_Content:  58<br />Percent: 2.03","Filename: SRR11951237<br />GC_Content:  59<br />Percent: 1.79","Filename: SRR11951237<br />GC_Content:  60<br />Percent: 1.49","Filename: SRR11951237<br />GC_Content:  61<br />Percent: 1.21","Filename: SRR11951237<br />GC_Content:  62<br />Percent: 1.02","Filename: SRR11951237<br />GC_Content:  63<br />Percent: 0.86","Filename: SRR11951237<br />GC_Content:  64<br />Percent: 0.68","Filename: SRR11951237<br />GC_Content:  65<br />Percent: 0.53","Filename: SRR11951237<br />GC_Content:  66<br />Percent: 0.39","Filename: SRR11951237<br />GC_Content:  67<br />Percent: 0.30","Filename: SRR11951237<br />GC_Content:  68<br />Percent: 0.25","Filename: SRR11951237<br />GC_Content:  69<br />Percent: 0.19","Filename: SRR11951237<br />GC_Content:  70<br />Percent: 0.14","Filename: SRR11951237<br />GC_Content:  71<br />Percent: 0.11","Filename: SRR11951237<br />GC_Content:  72<br />Percent: 0.09","Filename: SRR11951237<br />GC_Content:  73<br />Percent: 0.07","Filename: SRR11951237<br />GC_Content:  74<br />Percent: 0.06","Filename: SRR11951237<br />GC_Content:  75<br />Percent: 0.05","Filename: SRR11951237<br />GC_Content:  76<br />Percent: 0.04","Filename: SRR11951237<br />GC_Content:  77<br />Percent: 0.03","Filename: SRR11951237<br />GC_Content:  78<br />Percent: 0.02","Filename: SRR11951237<br />GC_Content:  79<br />Percent: 0.02","Filename: SRR11951237<br />GC_Content:  80<br />Percent: 0.02","Filename: SRR11951237<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951237<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951237<br />GC_Content:  83<br />Percent: 0.01","Filename: SRR11951237<br />GC_Content:  84<br />Percent: 0.01","Filename: SRR11951237<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  94<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951237<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951237<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(243,115,114,1)","dash":"solid"},"hoveron":"points","name":"SRR11951237","legendgroup":"SRR11951237","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.01,0.01,0,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.06,0.09,0.14,0.17,0.22,0.31,0.44,0.52,0.67,0.94,1.21,1.38,1.57,1.87,2.14,2.29,2.49,2.77,3,3.14,3.3,3.41,3.43,3.48,3.49,3.46,3.43,3.49,3.59,3.56,3.49,3.44,3.39,3.32,3.17,2.98,2.84,2.63,2.34,2.04,1.82,1.61,1.33,1.08,0.9,0.76,0.6,0.47,0.34,0.27,0.22,0.17,0.12,0.09,0.07,0.06,0.04,0.03,0.03,0.02,0.01,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951238<br />GC_Content:   0<br />Percent: 0.01","Filename: SRR11951238<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951238<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  10<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951238<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951238<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951238<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951238<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951238<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951238<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951238<br />GC_Content:  18<br />Percent: 0.06","Filename: SRR11951238<br />GC_Content:  19<br />Percent: 0.09","Filename: SRR11951238<br />GC_Content:  20<br />Percent: 0.14","Filename: SRR11951238<br />GC_Content:  21<br />Percent: 0.17","Filename: SRR11951238<br />GC_Content:  22<br />Percent: 0.22","Filename: SRR11951238<br />GC_Content:  23<br />Percent: 0.31","Filename: SRR11951238<br />GC_Content:  24<br />Percent: 0.44","Filename: SRR11951238<br />GC_Content:  25<br />Percent: 0.52","Filename: SRR11951238<br />GC_Content:  26<br />Percent: 0.67","Filename: SRR11951238<br />GC_Content:  27<br />Percent: 0.94","Filename: SRR11951238<br />GC_Content:  28<br />Percent: 1.21","Filename: SRR11951238<br />GC_Content:  29<br />Percent: 1.38","Filename: SRR11951238<br />GC_Content:  30<br />Percent: 1.57","Filename: SRR11951238<br />GC_Content:  31<br />Percent: 1.87","Filename: SRR11951238<br />GC_Content:  32<br />Percent: 2.14","Filename: SRR11951238<br />GC_Content:  33<br />Percent: 2.29","Filename: SRR11951238<br />GC_Content:  34<br />Percent: 2.49","Filename: SRR11951238<br />GC_Content:  35<br />Percent: 2.77","Filename: SRR11951238<br />GC_Content:  36<br />Percent: 3.00","Filename: SRR11951238<br />GC_Content:  37<br />Percent: 3.14","Filename: SRR11951238<br />GC_Content:  38<br />Percent: 3.30","Filename: SRR11951238<br />GC_Content:  39<br />Percent: 3.41","Filename: SRR11951238<br />GC_Content:  40<br />Percent: 3.43","Filename: SRR11951238<br />GC_Content:  41<br />Percent: 3.48","Filename: SRR11951238<br />GC_Content:  42<br />Percent: 3.49","Filename: SRR11951238<br />GC_Content:  43<br />Percent: 3.46","Filename: SRR11951238<br />GC_Content:  44<br />Percent: 3.43","Filename: SRR11951238<br />GC_Content:  45<br />Percent: 3.49","Filename: SRR11951238<br />GC_Content:  46<br />Percent: 3.59","Filename: SRR11951238<br />GC_Content:  47<br />Percent: 3.56","Filename: SRR11951238<br />GC_Content:  48<br />Percent: 3.49","Filename: SRR11951238<br />GC_Content:  49<br />Percent: 3.44","Filename: SRR11951238<br />GC_Content:  50<br />Percent: 3.39","Filename: SRR11951238<br />GC_Content:  51<br />Percent: 3.32","Filename: SRR11951238<br />GC_Content:  52<br />Percent: 3.17","Filename: SRR11951238<br />GC_Content:  53<br />Percent: 2.98","Filename: SRR11951238<br />GC_Content:  54<br />Percent: 2.84","Filename: SRR11951238<br />GC_Content:  55<br />Percent: 2.63","Filename: SRR11951238<br />GC_Content:  56<br />Percent: 2.34","Filename: SRR11951238<br />GC_Content:  57<br />Percent: 2.04","Filename: SRR11951238<br />GC_Content:  58<br />Percent: 1.82","Filename: SRR11951238<br />GC_Content:  59<br />Percent: 1.61","Filename: SRR11951238<br />GC_Content:  60<br />Percent: 1.33","Filename: SRR11951238<br />GC_Content:  61<br />Percent: 1.08","Filename: SRR11951238<br />GC_Content:  62<br />Percent: 0.90","Filename: SRR11951238<br />GC_Content:  63<br />Percent: 0.76","Filename: SRR11951238<br />GC_Content:  64<br />Percent: 0.60","Filename: SRR11951238<br />GC_Content:  65<br />Percent: 0.47","Filename: SRR11951238<br />GC_Content:  66<br />Percent: 0.34","Filename: SRR11951238<br />GC_Content:  67<br />Percent: 0.27","Filename: SRR11951238<br />GC_Content:  68<br />Percent: 0.22","Filename: SRR11951238<br />GC_Content:  69<br />Percent: 0.17","Filename: SRR11951238<br />GC_Content:  70<br />Percent: 0.12","Filename: SRR11951238<br />GC_Content:  71<br />Percent: 0.09","Filename: SRR11951238<br />GC_Content:  72<br />Percent: 0.07","Filename: SRR11951238<br />GC_Content:  73<br />Percent: 0.06","Filename: SRR11951238<br />GC_Content:  74<br />Percent: 0.04","Filename: SRR11951238<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951238<br />GC_Content:  76<br />Percent: 0.03","Filename: SRR11951238<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951238<br />GC_Content:  78<br />Percent: 0.01","Filename: SRR11951238<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951238<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951238<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951238<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951238<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  94<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951238<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951238<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(232,53,55,1)","dash":"solid"},"hoveron":"points","name":"SRR11951238","legendgroup":"SRR11951238","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.02,0.01,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.02,0.02,0.03,0.05,0.06,0.07,0.11,0.15,0.19,0.24,0.34,0.46,0.55,0.69,0.97,1.25,1.43,1.63,1.97,2.29,2.47,2.7,3.03,3.31,3.47,3.63,3.73,3.73,3.74,3.72,3.65,3.55,3.52,3.56,3.48,3.34,3.23,3.15,3.05,2.88,2.68,2.54,2.35,2.07,1.8,1.61,1.42,1.18,0.95,0.79,0.67,0.53,0.42,0.3,0.24,0.2,0.15,0.11,0.09,0.07,0.06,0.04,0.03,0.03,0.02,0.02,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951239<br />GC_Content:   0<br />Percent: 0.02","Filename: SRR11951239<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951239<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951239<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951239<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951239<br />GC_Content:  13<br />Percent: 0.02","Filename: SRR11951239<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951239<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951239<br />GC_Content:  16<br />Percent: 0.05","Filename: SRR11951239<br />GC_Content:  17<br />Percent: 0.06","Filename: SRR11951239<br />GC_Content:  18<br />Percent: 0.07","Filename: SRR11951239<br />GC_Content:  19<br />Percent: 0.11","Filename: SRR11951239<br />GC_Content:  20<br />Percent: 0.15","Filename: SRR11951239<br />GC_Content:  21<br />Percent: 0.19","Filename: SRR11951239<br />GC_Content:  22<br />Percent: 0.24","Filename: SRR11951239<br />GC_Content:  23<br />Percent: 0.34","Filename: SRR11951239<br />GC_Content:  24<br />Percent: 0.46","Filename: SRR11951239<br />GC_Content:  25<br />Percent: 0.55","Filename: SRR11951239<br />GC_Content:  26<br />Percent: 0.69","Filename: SRR11951239<br />GC_Content:  27<br />Percent: 0.97","Filename: SRR11951239<br />GC_Content:  28<br />Percent: 1.25","Filename: SRR11951239<br />GC_Content:  29<br />Percent: 1.43","Filename: SRR11951239<br />GC_Content:  30<br />Percent: 1.63","Filename: SRR11951239<br />GC_Content:  31<br />Percent: 1.97","Filename: SRR11951239<br />GC_Content:  32<br />Percent: 2.29","Filename: SRR11951239<br />GC_Content:  33<br />Percent: 2.47","Filename: SRR11951239<br />GC_Content:  34<br />Percent: 2.70","Filename: SRR11951239<br />GC_Content:  35<br />Percent: 3.03","Filename: SRR11951239<br />GC_Content:  36<br />Percent: 3.31","Filename: SRR11951239<br />GC_Content:  37<br />Percent: 3.47","Filename: SRR11951239<br />GC_Content:  38<br />Percent: 3.63","Filename: SRR11951239<br />GC_Content:  39<br />Percent: 3.73","Filename: SRR11951239<br />GC_Content:  40<br />Percent: 3.73","Filename: SRR11951239<br />GC_Content:  41<br />Percent: 3.74","Filename: SRR11951239<br />GC_Content:  42<br />Percent: 3.72","Filename: SRR11951239<br />GC_Content:  43<br />Percent: 3.65","Filename: SRR11951239<br />GC_Content:  44<br />Percent: 3.55","Filename: SRR11951239<br />GC_Content:  45<br />Percent: 3.52","Filename: SRR11951239<br />GC_Content:  46<br />Percent: 3.56","Filename: SRR11951239<br />GC_Content:  47<br />Percent: 3.48","Filename: SRR11951239<br />GC_Content:  48<br />Percent: 3.34","Filename: SRR11951239<br />GC_Content:  49<br />Percent: 3.23","Filename: SRR11951239<br />GC_Content:  50<br />Percent: 3.15","Filename: SRR11951239<br />GC_Content:  51<br />Percent: 3.05","Filename: SRR11951239<br />GC_Content:  52<br />Percent: 2.88","Filename: SRR11951239<br />GC_Content:  53<br />Percent: 2.68","Filename: SRR11951239<br />GC_Content:  54<br />Percent: 2.54","Filename: SRR11951239<br />GC_Content:  55<br />Percent: 2.35","Filename: SRR11951239<br />GC_Content:  56<br />Percent: 2.07","Filename: SRR11951239<br />GC_Content:  57<br />Percent: 1.80","Filename: SRR11951239<br />GC_Content:  58<br />Percent: 1.61","Filename: SRR11951239<br />GC_Content:  59<br />Percent: 1.42","Filename: SRR11951239<br />GC_Content:  60<br />Percent: 1.18","Filename: SRR11951239<br />GC_Content:  61<br />Percent: 0.95","Filename: SRR11951239<br />GC_Content:  62<br />Percent: 0.79","Filename: SRR11951239<br />GC_Content:  63<br />Percent: 0.67","Filename: SRR11951239<br />GC_Content:  64<br />Percent: 0.53","Filename: SRR11951239<br />GC_Content:  65<br />Percent: 0.42","Filename: SRR11951239<br />GC_Content:  66<br />Percent: 0.30","Filename: SRR11951239<br />GC_Content:  67<br />Percent: 0.24","Filename: SRR11951239<br />GC_Content:  68<br />Percent: 0.20","Filename: SRR11951239<br />GC_Content:  69<br />Percent: 0.15","Filename: SRR11951239<br />GC_Content:  70<br />Percent: 0.11","Filename: SRR11951239<br />GC_Content:  71<br />Percent: 0.09","Filename: SRR11951239<br />GC_Content:  72<br />Percent: 0.07","Filename: SRR11951239<br />GC_Content:  73<br />Percent: 0.06","Filename: SRR11951239<br />GC_Content:  74<br />Percent: 0.04","Filename: SRR11951239<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951239<br />GC_Content:  76<br />Percent: 0.03","Filename: SRR11951239<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951239<br />GC_Content:  78<br />Percent: 0.02","Filename: SRR11951239<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951239<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951239<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951239<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951239<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  94<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951239<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951239<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(233,69,49,1)","dash":"solid"},"hoveron":"points","name":"SRR11951239","legendgroup":"SRR11951239","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.01,0.01,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.07,0.11,0.15,0.19,0.24,0.36,0.5,0.6,0.76,1.06,1.38,1.58,1.8,2.17,2.49,2.67,2.89,3.2,3.45,3.6,3.74,3.81,3.78,3.76,3.72,3.63,3.49,3.45,3.47,3.37,3.21,3.09,3,2.9,2.73,2.55,2.41,2.23,1.97,1.71,1.52,1.34,1.11,0.9,0.74,0.63,0.5,0.39,0.28,0.22,0.19,0.14,0.1,0.08,0.07,0.05,0.04,0.03,0.02,0.02,0.01,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951240<br />GC_Content:   0<br />Percent: 0.01","Filename: SRR11951240<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951240<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951240<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951240<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951240<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951240<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951240<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951240<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951240<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951240<br />GC_Content:  18<br />Percent: 0.07","Filename: SRR11951240<br />GC_Content:  19<br />Percent: 0.11","Filename: SRR11951240<br />GC_Content:  20<br />Percent: 0.15","Filename: SRR11951240<br />GC_Content:  21<br />Percent: 0.19","Filename: SRR11951240<br />GC_Content:  22<br />Percent: 0.24","Filename: SRR11951240<br />GC_Content:  23<br />Percent: 0.36","Filename: SRR11951240<br />GC_Content:  24<br />Percent: 0.50","Filename: SRR11951240<br />GC_Content:  25<br />Percent: 0.60","Filename: SRR11951240<br />GC_Content:  26<br />Percent: 0.76","Filename: SRR11951240<br />GC_Content:  27<br />Percent: 1.06","Filename: SRR11951240<br />GC_Content:  28<br />Percent: 1.38","Filename: SRR11951240<br />GC_Content:  29<br />Percent: 1.58","Filename: SRR11951240<br />GC_Content:  30<br />Percent: 1.80","Filename: SRR11951240<br />GC_Content:  31<br />Percent: 2.17","Filename: SRR11951240<br />GC_Content:  32<br />Percent: 2.49","Filename: SRR11951240<br />GC_Content:  33<br />Percent: 2.67","Filename: SRR11951240<br />GC_Content:  34<br />Percent: 2.89","Filename: SRR11951240<br />GC_Content:  35<br />Percent: 3.20","Filename: SRR11951240<br />GC_Content:  36<br />Percent: 3.45","Filename: SRR11951240<br />GC_Content:  37<br />Percent: 3.60","Filename: SRR11951240<br />GC_Content:  38<br />Percent: 3.74","Filename: SRR11951240<br />GC_Content:  39<br />Percent: 3.81","Filename: SRR11951240<br />GC_Content:  40<br />Percent: 3.78","Filename: SRR11951240<br />GC_Content:  41<br />Percent: 3.76","Filename: SRR11951240<br />GC_Content:  42<br />Percent: 3.72","Filename: SRR11951240<br />GC_Content:  43<br />Percent: 3.63","Filename: SRR11951240<br />GC_Content:  44<br />Percent: 3.49","Filename: SRR11951240<br />GC_Content:  45<br />Percent: 3.45","Filename: SRR11951240<br />GC_Content:  46<br />Percent: 3.47","Filename: SRR11951240<br />GC_Content:  47<br />Percent: 3.37","Filename: SRR11951240<br />GC_Content:  48<br />Percent: 3.21","Filename: SRR11951240<br />GC_Content:  49<br />Percent: 3.09","Filename: SRR11951240<br />GC_Content:  50<br />Percent: 3.00","Filename: SRR11951240<br />GC_Content:  51<br />Percent: 2.90","Filename: SRR11951240<br />GC_Content:  52<br />Percent: 2.73","Filename: SRR11951240<br />GC_Content:  53<br />Percent: 2.55","Filename: SRR11951240<br />GC_Content:  54<br />Percent: 2.41","Filename: SRR11951240<br />GC_Content:  55<br />Percent: 2.23","Filename: SRR11951240<br />GC_Content:  56<br />Percent: 1.97","Filename: SRR11951240<br />GC_Content:  57<br />Percent: 1.71","Filename: SRR11951240<br />GC_Content:  58<br />Percent: 1.52","Filename: SRR11951240<br />GC_Content:  59<br />Percent: 1.34","Filename: SRR11951240<br />GC_Content:  60<br />Percent: 1.11","Filename: SRR11951240<br />GC_Content:  61<br />Percent: 0.90","Filename: SRR11951240<br />GC_Content:  62<br />Percent: 0.74","Filename: SRR11951240<br />GC_Content:  63<br />Percent: 0.63","Filename: SRR11951240<br />GC_Content:  64<br />Percent: 0.50","Filename: SRR11951240<br />GC_Content:  65<br />Percent: 0.39","Filename: SRR11951240<br />GC_Content:  66<br />Percent: 0.28","Filename: SRR11951240<br />GC_Content:  67<br />Percent: 0.22","Filename: SRR11951240<br />GC_Content:  68<br />Percent: 0.19","Filename: SRR11951240<br />GC_Content:  69<br />Percent: 0.14","Filename: SRR11951240<br />GC_Content:  70<br />Percent: 0.10","Filename: SRR11951240<br />GC_Content:  71<br />Percent: 0.08","Filename: SRR11951240<br />GC_Content:  72<br />Percent: 0.07","Filename: SRR11951240<br />GC_Content:  73<br />Percent: 0.05","Filename: SRR11951240<br />GC_Content:  74<br />Percent: 0.04","Filename: SRR11951240<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951240<br />GC_Content:  76<br />Percent: 0.02","Filename: SRR11951240<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951240<br />GC_Content:  78<br />Percent: 0.01","Filename: SRR11951240<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951240<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951240<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951240<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951240<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  94<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951240<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951240<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(246,147,89,1)","dash":"solid"},"hoveron":"points","name":"SRR11951240","legendgroup":"SRR11951240","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.01,0.01,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.07,0.1,0.15,0.18,0.23,0.33,0.45,0.54,0.68,0.95,1.22,1.39,1.57,1.87,2.14,2.3,2.48,2.74,2.96,3.09,3.23,3.34,3.37,3.42,3.44,3.43,3.41,3.47,3.57,3.54,3.47,3.43,3.39,3.33,3.18,2.99,2.85,2.65,2.35,2.06,1.84,1.63,1.35,1.1,0.92,0.78,0.62,0.49,0.35,0.28,0.23,0.18,0.13,0.1,0.09,0.07,0.05,0.04,0.04,0.03,0.02,0.02,0.02,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951241<br />GC_Content:   0<br />Percent: 0.01","Filename: SRR11951241<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951241<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951241<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951241<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951241<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951241<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951241<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951241<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951241<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951241<br />GC_Content:  18<br />Percent: 0.07","Filename: SRR11951241<br />GC_Content:  19<br />Percent: 0.10","Filename: SRR11951241<br />GC_Content:  20<br />Percent: 0.15","Filename: SRR11951241<br />GC_Content:  21<br />Percent: 0.18","Filename: SRR11951241<br />GC_Content:  22<br />Percent: 0.23","Filename: SRR11951241<br />GC_Content:  23<br />Percent: 0.33","Filename: SRR11951241<br />GC_Content:  24<br />Percent: 0.45","Filename: SRR11951241<br />GC_Content:  25<br />Percent: 0.54","Filename: SRR11951241<br />GC_Content:  26<br />Percent: 0.68","Filename: SRR11951241<br />GC_Content:  27<br />Percent: 0.95","Filename: SRR11951241<br />GC_Content:  28<br />Percent: 1.22","Filename: SRR11951241<br />GC_Content:  29<br />Percent: 1.39","Filename: SRR11951241<br />GC_Content:  30<br />Percent: 1.57","Filename: SRR11951241<br />GC_Content:  31<br />Percent: 1.87","Filename: SRR11951241<br />GC_Content:  32<br />Percent: 2.14","Filename: SRR11951241<br />GC_Content:  33<br />Percent: 2.30","Filename: SRR11951241<br />GC_Content:  34<br />Percent: 2.48","Filename: SRR11951241<br />GC_Content:  35<br />Percent: 2.74","Filename: SRR11951241<br />GC_Content:  36<br />Percent: 2.96","Filename: SRR11951241<br />GC_Content:  37<br />Percent: 3.09","Filename: SRR11951241<br />GC_Content:  38<br />Percent: 3.23","Filename: SRR11951241<br />GC_Content:  39<br />Percent: 3.34","Filename: SRR11951241<br />GC_Content:  40<br />Percent: 3.37","Filename: SRR11951241<br />GC_Content:  41<br />Percent: 3.42","Filename: SRR11951241<br />GC_Content:  42<br />Percent: 3.44","Filename: SRR11951241<br />GC_Content:  43<br />Percent: 3.43","Filename: SRR11951241<br />GC_Content:  44<br />Percent: 3.41","Filename: SRR11951241<br />GC_Content:  45<br />Percent: 3.47","Filename: SRR11951241<br />GC_Content:  46<br />Percent: 3.57","Filename: SRR11951241<br />GC_Content:  47<br />Percent: 3.54","Filename: SRR11951241<br />GC_Content:  48<br />Percent: 3.47","Filename: SRR11951241<br />GC_Content:  49<br />Percent: 3.43","Filename: SRR11951241<br />GC_Content:  50<br />Percent: 3.39","Filename: SRR11951241<br />GC_Content:  51<br />Percent: 3.33","Filename: SRR11951241<br />GC_Content:  52<br />Percent: 3.18","Filename: SRR11951241<br />GC_Content:  53<br />Percent: 2.99","Filename: SRR11951241<br />GC_Content:  54<br />Percent: 2.85","Filename: SRR11951241<br />GC_Content:  55<br />Percent: 2.65","Filename: SRR11951241<br />GC_Content:  56<br />Percent: 2.35","Filename: SRR11951241<br />GC_Content:  57<br />Percent: 2.06","Filename: SRR11951241<br />GC_Content:  58<br />Percent: 1.84","Filename: SRR11951241<br />GC_Content:  59<br />Percent: 1.63","Filename: SRR11951241<br />GC_Content:  60<br />Percent: 1.35","Filename: SRR11951241<br />GC_Content:  61<br />Percent: 1.10","Filename: SRR11951241<br />GC_Content:  62<br />Percent: 0.92","Filename: SRR11951241<br />GC_Content:  63<br />Percent: 0.78","Filename: SRR11951241<br />GC_Content:  64<br />Percent: 0.62","Filename: SRR11951241<br />GC_Content:  65<br />Percent: 0.49","Filename: SRR11951241<br />GC_Content:  66<br />Percent: 0.35","Filename: SRR11951241<br />GC_Content:  67<br />Percent: 0.28","Filename: SRR11951241<br />GC_Content:  68<br />Percent: 0.23","Filename: SRR11951241<br />GC_Content:  69<br />Percent: 0.18","Filename: SRR11951241<br />GC_Content:  70<br />Percent: 0.13","Filename: SRR11951241<br />GC_Content:  71<br />Percent: 0.10","Filename: SRR11951241<br />GC_Content:  72<br />Percent: 0.09","Filename: SRR11951241<br />GC_Content:  73<br />Percent: 0.07","Filename: SRR11951241<br />GC_Content:  74<br />Percent: 0.05","Filename: SRR11951241<br />GC_Content:  75<br />Percent: 0.04","Filename: SRR11951241<br />GC_Content:  76<br />Percent: 0.04","Filename: SRR11951241<br />GC_Content:  77<br />Percent: 0.03","Filename: SRR11951241<br />GC_Content:  78<br />Percent: 0.02","Filename: SRR11951241<br />GC_Content:  79<br />Percent: 0.02","Filename: SRR11951241<br />GC_Content:  80<br />Percent: 0.02","Filename: SRR11951241<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951241<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951241<br />GC_Content:  83<br />Percent: 0.01","Filename: SRR11951241<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  94<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951241<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951241<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(253,177,86,1)","dash":"solid"},"hoveron":"points","name":"SRR11951241","legendgroup":"SRR11951241","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.03,0.02,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.06,0.1,0.14,0.16,0.21,0.31,0.42,0.51,0.66,0.94,1.25,1.45,1.67,2.05,2.4,2.61,2.88,3.28,3.59,3.76,3.95,4.04,3.98,3.94,3.89,3.77,3.6,3.53,3.55,3.42,3.23,3.08,2.96,2.84,2.65,2.46,2.33,2.15,1.9,1.64,1.46,1.29,1.07,0.86,0.72,0.61,0.49,0.38,0.28,0.22,0.19,0.15,0.11,0.1,0.08,0.07,0.06,0.05,0.04,0.03,0.03,0.02,0.02,0.02,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0.01,0.01,0,0,0,0,0.01,0.02],"text":["Filename: SRR11951242<br />GC_Content:   0<br />Percent: 0.03","Filename: SRR11951242<br />GC_Content:   1<br />Percent: 0.02","Filename: SRR11951242<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951242<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951242<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951242<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951242<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951242<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951242<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951242<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951242<br />GC_Content:  18<br />Percent: 0.06","Filename: SRR11951242<br />GC_Content:  19<br />Percent: 0.10","Filename: SRR11951242<br />GC_Content:  20<br />Percent: 0.14","Filename: SRR11951242<br />GC_Content:  21<br />Percent: 0.16","Filename: SRR11951242<br />GC_Content:  22<br />Percent: 0.21","Filename: SRR11951242<br />GC_Content:  23<br />Percent: 0.31","Filename: SRR11951242<br />GC_Content:  24<br />Percent: 0.42","Filename: SRR11951242<br />GC_Content:  25<br />Percent: 0.51","Filename: SRR11951242<br />GC_Content:  26<br />Percent: 0.66","Filename: SRR11951242<br />GC_Content:  27<br />Percent: 0.94","Filename: SRR11951242<br />GC_Content:  28<br />Percent: 1.25","Filename: SRR11951242<br />GC_Content:  29<br />Percent: 1.45","Filename: SRR11951242<br />GC_Content:  30<br />Percent: 1.67","Filename: SRR11951242<br />GC_Content:  31<br />Percent: 2.05","Filename: SRR11951242<br />GC_Content:  32<br />Percent: 2.40","Filename: SRR11951242<br />GC_Content:  33<br />Percent: 2.61","Filename: SRR11951242<br />GC_Content:  34<br />Percent: 2.88","Filename: SRR11951242<br />GC_Content:  35<br />Percent: 3.28","Filename: SRR11951242<br />GC_Content:  36<br />Percent: 3.59","Filename: SRR11951242<br />GC_Content:  37<br />Percent: 3.76","Filename: SRR11951242<br />GC_Content:  38<br />Percent: 3.95","Filename: SRR11951242<br />GC_Content:  39<br />Percent: 4.04","Filename: SRR11951242<br />GC_Content:  40<br />Percent: 3.98","Filename: SRR11951242<br />GC_Content:  41<br />Percent: 3.94","Filename: SRR11951242<br />GC_Content:  42<br />Percent: 3.89","Filename: SRR11951242<br />GC_Content:  43<br />Percent: 3.77","Filename: SRR11951242<br />GC_Content:  44<br />Percent: 3.60","Filename: SRR11951242<br />GC_Content:  45<br />Percent: 3.53","Filename: SRR11951242<br />GC_Content:  46<br />Percent: 3.55","Filename: SRR11951242<br />GC_Content:  47<br />Percent: 3.42","Filename: SRR11951242<br />GC_Content:  48<br />Percent: 3.23","Filename: SRR11951242<br />GC_Content:  49<br />Percent: 3.08","Filename: SRR11951242<br />GC_Content:  50<br />Percent: 2.96","Filename: SRR11951242<br />GC_Content:  51<br />Percent: 2.84","Filename: SRR11951242<br />GC_Content:  52<br />Percent: 2.65","Filename: SRR11951242<br />GC_Content:  53<br />Percent: 2.46","Filename: SRR11951242<br />GC_Content:  54<br />Percent: 2.33","Filename: SRR11951242<br />GC_Content:  55<br />Percent: 2.15","Filename: SRR11951242<br />GC_Content:  56<br />Percent: 1.90","Filename: SRR11951242<br />GC_Content:  57<br />Percent: 1.64","Filename: SRR11951242<br />GC_Content:  58<br />Percent: 1.46","Filename: SRR11951242<br />GC_Content:  59<br />Percent: 1.29","Filename: SRR11951242<br />GC_Content:  60<br />Percent: 1.07","Filename: SRR11951242<br />GC_Content:  61<br />Percent: 0.86","Filename: SRR11951242<br />GC_Content:  62<br />Percent: 0.72","Filename: SRR11951242<br />GC_Content:  63<br />Percent: 0.61","Filename: SRR11951242<br />GC_Content:  64<br />Percent: 0.49","Filename: SRR11951242<br />GC_Content:  65<br />Percent: 0.38","Filename: SRR11951242<br />GC_Content:  66<br />Percent: 0.28","Filename: SRR11951242<br />GC_Content:  67<br />Percent: 0.22","Filename: SRR11951242<br />GC_Content:  68<br />Percent: 0.19","Filename: SRR11951242<br />GC_Content:  69<br />Percent: 0.15","Filename: SRR11951242<br />GC_Content:  70<br />Percent: 0.11","Filename: SRR11951242<br />GC_Content:  71<br />Percent: 0.10","Filename: SRR11951242<br />GC_Content:  72<br />Percent: 0.08","Filename: SRR11951242<br />GC_Content:  73<br />Percent: 0.07","Filename: SRR11951242<br />GC_Content:  74<br />Percent: 0.06","Filename: SRR11951242<br />GC_Content:  75<br />Percent: 0.05","Filename: SRR11951242<br />GC_Content:  76<br />Percent: 0.04","Filename: SRR11951242<br />GC_Content:  77<br />Percent: 0.03","Filename: SRR11951242<br />GC_Content:  78<br />Percent: 0.03","Filename: SRR11951242<br />GC_Content:  79<br />Percent: 0.02","Filename: SRR11951242<br />GC_Content:  80<br />Percent: 0.02","Filename: SRR11951242<br />GC_Content:  81<br />Percent: 0.02","Filename: SRR11951242<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951242<br />GC_Content:  83<br />Percent: 0.01","Filename: SRR11951242<br />GC_Content:  84<br />Percent: 0.01","Filename: SRR11951242<br />GC_Content:  85<br />Percent: 0.01","Filename: SRR11951242<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:  93<br />Percent: 0.01","Filename: SRR11951242<br />GC_Content:  94<br />Percent: 0.01","Filename: SRR11951242<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951242<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951242<br />GC_Content: 100<br />Percent: 0.02"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(254,146,33,1)","dash":"solid"},"hoveron":"points","name":"SRR11951242","legendgroup":"SRR11951242","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.02,0.01,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.07,0.11,0.16,0.19,0.24,0.35,0.48,0.57,0.72,0.99,1.28,1.45,1.65,1.96,2.25,2.41,2.62,2.94,3.21,3.37,3.54,3.64,3.63,3.65,3.63,3.57,3.48,3.48,3.55,3.48,3.37,3.29,3.21,3.11,2.94,2.74,2.61,2.43,2.15,1.88,1.67,1.47,1.22,0.99,0.83,0.7,0.55,0.43,0.32,0.25,0.21,0.16,0.11,0.08,0.07,0.05,0.04,0.03,0.02,0.02,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0.01,0.01,0,0,0,0,0.02,0.05],"text":["Filename: SRR11951243<br />GC_Content:   0<br />Percent: 0.02","Filename: SRR11951243<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951243<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951243<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951243<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951243<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951243<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951243<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951243<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951243<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951243<br />GC_Content:  18<br />Percent: 0.07","Filename: SRR11951243<br />GC_Content:  19<br />Percent: 0.11","Filename: SRR11951243<br />GC_Content:  20<br />Percent: 0.16","Filename: SRR11951243<br />GC_Content:  21<br />Percent: 0.19","Filename: SRR11951243<br />GC_Content:  22<br />Percent: 0.24","Filename: SRR11951243<br />GC_Content:  23<br />Percent: 0.35","Filename: SRR11951243<br />GC_Content:  24<br />Percent: 0.48","Filename: SRR11951243<br />GC_Content:  25<br />Percent: 0.57","Filename: SRR11951243<br />GC_Content:  26<br />Percent: 0.72","Filename: SRR11951243<br />GC_Content:  27<br />Percent: 0.99","Filename: SRR11951243<br />GC_Content:  28<br />Percent: 1.28","Filename: SRR11951243<br />GC_Content:  29<br />Percent: 1.45","Filename: SRR11951243<br />GC_Content:  30<br />Percent: 1.65","Filename: SRR11951243<br />GC_Content:  31<br />Percent: 1.96","Filename: SRR11951243<br />GC_Content:  32<br />Percent: 2.25","Filename: SRR11951243<br />GC_Content:  33<br />Percent: 2.41","Filename: SRR11951243<br />GC_Content:  34<br />Percent: 2.62","Filename: SRR11951243<br />GC_Content:  35<br />Percent: 2.94","Filename: SRR11951243<br />GC_Content:  36<br />Percent: 3.21","Filename: SRR11951243<br />GC_Content:  37<br />Percent: 3.37","Filename: SRR11951243<br />GC_Content:  38<br />Percent: 3.54","Filename: SRR11951243<br />GC_Content:  39<br />Percent: 3.64","Filename: SRR11951243<br />GC_Content:  40<br />Percent: 3.63","Filename: SRR11951243<br />GC_Content:  41<br />Percent: 3.65","Filename: SRR11951243<br />GC_Content:  42<br />Percent: 3.63","Filename: SRR11951243<br />GC_Content:  43<br />Percent: 3.57","Filename: SRR11951243<br />GC_Content:  44<br />Percent: 3.48","Filename: SRR11951243<br />GC_Content:  45<br />Percent: 3.48","Filename: SRR11951243<br />GC_Content:  46<br />Percent: 3.55","Filename: SRR11951243<br />GC_Content:  47<br />Percent: 3.48","Filename: SRR11951243<br />GC_Content:  48<br />Percent: 3.37","Filename: SRR11951243<br />GC_Content:  49<br />Percent: 3.29","Filename: SRR11951243<br />GC_Content:  50<br />Percent: 3.21","Filename: SRR11951243<br />GC_Content:  51<br />Percent: 3.11","Filename: SRR11951243<br />GC_Content:  52<br />Percent: 2.94","Filename: SRR11951243<br />GC_Content:  53<br />Percent: 2.74","Filename: SRR11951243<br />GC_Content:  54<br />Percent: 2.61","Filename: SRR11951243<br />GC_Content:  55<br />Percent: 2.43","Filename: SRR11951243<br />GC_Content:  56<br />Percent: 2.15","Filename: SRR11951243<br />GC_Content:  57<br />Percent: 1.88","Filename: SRR11951243<br />GC_Content:  58<br />Percent: 1.67","Filename: SRR11951243<br />GC_Content:  59<br />Percent: 1.47","Filename: SRR11951243<br />GC_Content:  60<br />Percent: 1.22","Filename: SRR11951243<br />GC_Content:  61<br />Percent: 0.99","Filename: SRR11951243<br />GC_Content:  62<br />Percent: 0.83","Filename: SRR11951243<br />GC_Content:  63<br />Percent: 0.70","Filename: SRR11951243<br />GC_Content:  64<br />Percent: 0.55","Filename: SRR11951243<br />GC_Content:  65<br />Percent: 0.43","Filename: SRR11951243<br />GC_Content:  66<br />Percent: 0.32","Filename: SRR11951243<br />GC_Content:  67<br />Percent: 0.25","Filename: SRR11951243<br />GC_Content:  68<br />Percent: 0.21","Filename: SRR11951243<br />GC_Content:  69<br />Percent: 0.16","Filename: SRR11951243<br />GC_Content:  70<br />Percent: 0.11","Filename: SRR11951243<br />GC_Content:  71<br />Percent: 0.08","Filename: SRR11951243<br />GC_Content:  72<br />Percent: 0.07","Filename: SRR11951243<br />GC_Content:  73<br />Percent: 0.05","Filename: SRR11951243<br />GC_Content:  74<br />Percent: 0.04","Filename: SRR11951243<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951243<br />GC_Content:  76<br />Percent: 0.02","Filename: SRR11951243<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951243<br />GC_Content:  78<br />Percent: 0.01","Filename: SRR11951243<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951243<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951243<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951243<br />GC_Content:  82<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  93<br />Percent: 0.01","Filename: SRR11951243<br />GC_Content:  94<br />Percent: 0.01","Filename: SRR11951243<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951243<br />GC_Content:  99<br />Percent: 0.02","Filename: SRR11951243<br />GC_Content: 100<br />Percent: 0.05"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(245,135,37,1)","dash":"solid"},"hoveron":"points","name":"SRR11951243","legendgroup":"SRR11951243","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.01,0,0,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.02,0.02,0.03,0.04,0.05,0.08,0.12,0.14,0.18,0.27,0.37,0.44,0.56,0.79,1.03,1.18,1.35,1.65,1.91,2.06,2.25,2.53,2.76,2.9,3.05,3.19,3.26,3.35,3.4,3.43,3.46,3.55,3.67,3.69,3.67,3.65,3.64,3.57,3.42,3.22,3.07,2.87,2.56,2.25,2.01,1.78,1.48,1.2,1,0.85,0.67,0.52,0.38,0.3,0.25,0.19,0.13,0.1,0.08,0.06,0.05,0.03,0.03,0.02,0.02,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951244<br />GC_Content:   0<br />Percent: 0.01","Filename: SRR11951244<br />GC_Content:   1<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  10<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951244<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951244<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951244<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951244<br />GC_Content:  15<br />Percent: 0.02","Filename: SRR11951244<br />GC_Content:  16<br />Percent: 0.03","Filename: SRR11951244<br />GC_Content:  17<br />Percent: 0.04","Filename: SRR11951244<br />GC_Content:  18<br />Percent: 0.05","Filename: SRR11951244<br />GC_Content:  19<br />Percent: 0.08","Filename: SRR11951244<br />GC_Content:  20<br />Percent: 0.12","Filename: SRR11951244<br />GC_Content:  21<br />Percent: 0.14","Filename: SRR11951244<br />GC_Content:  22<br />Percent: 0.18","Filename: SRR11951244<br />GC_Content:  23<br />Percent: 0.27","Filename: SRR11951244<br />GC_Content:  24<br />Percent: 0.37","Filename: SRR11951244<br />GC_Content:  25<br />Percent: 0.44","Filename: SRR11951244<br />GC_Content:  26<br />Percent: 0.56","Filename: SRR11951244<br />GC_Content:  27<br />Percent: 0.79","Filename: SRR11951244<br />GC_Content:  28<br />Percent: 1.03","Filename: SRR11951244<br />GC_Content:  29<br />Percent: 1.18","Filename: SRR11951244<br />GC_Content:  30<br />Percent: 1.35","Filename: SRR11951244<br />GC_Content:  31<br />Percent: 1.65","Filename: SRR11951244<br />GC_Content:  32<br />Percent: 1.91","Filename: SRR11951244<br />GC_Content:  33<br />Percent: 2.06","Filename: SRR11951244<br />GC_Content:  34<br />Percent: 2.25","Filename: SRR11951244<br />GC_Content:  35<br />Percent: 2.53","Filename: SRR11951244<br />GC_Content:  36<br />Percent: 2.76","Filename: SRR11951244<br />GC_Content:  37<br />Percent: 2.90","Filename: SRR11951244<br />GC_Content:  38<br />Percent: 3.05","Filename: SRR11951244<br />GC_Content:  39<br />Percent: 3.19","Filename: SRR11951244<br />GC_Content:  40<br />Percent: 3.26","Filename: SRR11951244<br />GC_Content:  41<br />Percent: 3.35","Filename: SRR11951244<br />GC_Content:  42<br />Percent: 3.40","Filename: SRR11951244<br />GC_Content:  43<br />Percent: 3.43","Filename: SRR11951244<br />GC_Content:  44<br />Percent: 3.46","Filename: SRR11951244<br />GC_Content:  45<br />Percent: 3.55","Filename: SRR11951244<br />GC_Content:  46<br />Percent: 3.67","Filename: SRR11951244<br />GC_Content:  47<br />Percent: 3.69","Filename: SRR11951244<br />GC_Content:  48<br />Percent: 3.67","Filename: SRR11951244<br />GC_Content:  49<br />Percent: 3.65","Filename: SRR11951244<br />GC_Content:  50<br />Percent: 3.64","Filename: SRR11951244<br />GC_Content:  51<br />Percent: 3.57","Filename: SRR11951244<br />GC_Content:  52<br />Percent: 3.42","Filename: SRR11951244<br />GC_Content:  53<br />Percent: 3.22","Filename: SRR11951244<br />GC_Content:  54<br />Percent: 3.07","Filename: SRR11951244<br />GC_Content:  55<br />Percent: 2.87","Filename: SRR11951244<br />GC_Content:  56<br />Percent: 2.56","Filename: SRR11951244<br />GC_Content:  57<br />Percent: 2.25","Filename: SRR11951244<br />GC_Content:  58<br />Percent: 2.01","Filename: SRR11951244<br />GC_Content:  59<br />Percent: 1.78","Filename: SRR11951244<br />GC_Content:  60<br />Percent: 1.48","Filename: SRR11951244<br />GC_Content:  61<br />Percent: 1.20","Filename: SRR11951244<br />GC_Content:  62<br />Percent: 1.00","Filename: SRR11951244<br />GC_Content:  63<br />Percent: 0.85","Filename: SRR11951244<br />GC_Content:  64<br />Percent: 0.67","Filename: SRR11951244<br />GC_Content:  65<br />Percent: 0.52","Filename: SRR11951244<br />GC_Content:  66<br />Percent: 0.38","Filename: SRR11951244<br />GC_Content:  67<br />Percent: 0.30","Filename: SRR11951244<br />GC_Content:  68<br />Percent: 0.25","Filename: SRR11951244<br />GC_Content:  69<br />Percent: 0.19","Filename: SRR11951244<br />GC_Content:  70<br />Percent: 0.13","Filename: SRR11951244<br />GC_Content:  71<br />Percent: 0.10","Filename: SRR11951244<br />GC_Content:  72<br />Percent: 0.08","Filename: SRR11951244<br />GC_Content:  73<br />Percent: 0.06","Filename: SRR11951244<br />GC_Content:  74<br />Percent: 0.05","Filename: SRR11951244<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951244<br />GC_Content:  76<br />Percent: 0.03","Filename: SRR11951244<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951244<br />GC_Content:  78<br />Percent: 0.02","Filename: SRR11951244<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951244<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951244<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951244<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951244<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  94<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951244<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951244<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(220,160,139,1)","dash":"solid"},"hoveron":"points","name":"SRR11951244","legendgroup":"SRR11951244","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.02,0.01,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.07,0.1,0.15,0.18,0.23,0.33,0.46,0.54,0.69,0.96,1.24,1.42,1.62,1.96,2.27,2.44,2.66,2.99,3.25,3.41,3.57,3.68,3.68,3.69,3.67,3.61,3.51,3.5,3.56,3.48,3.36,3.26,3.18,3.09,2.93,2.73,2.59,2.4,2.13,1.86,1.66,1.47,1.22,0.99,0.83,0.7,0.55,0.43,0.31,0.25,0.21,0.16,0.11,0.08,0.07,0.05,0.04,0.03,0.02,0.02,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951245<br />GC_Content:   0<br />Percent: 0.02","Filename: SRR11951245<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951245<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951245<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951245<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951245<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951245<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951245<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951245<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951245<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951245<br />GC_Content:  18<br />Percent: 0.07","Filename: SRR11951245<br />GC_Content:  19<br />Percent: 0.10","Filename: SRR11951245<br />GC_Content:  20<br />Percent: 0.15","Filename: SRR11951245<br />GC_Content:  21<br />Percent: 0.18","Filename: SRR11951245<br />GC_Content:  22<br />Percent: 0.23","Filename: SRR11951245<br />GC_Content:  23<br />Percent: 0.33","Filename: SRR11951245<br />GC_Content:  24<br />Percent: 0.46","Filename: SRR11951245<br />GC_Content:  25<br />Percent: 0.54","Filename: SRR11951245<br />GC_Content:  26<br />Percent: 0.69","Filename: SRR11951245<br />GC_Content:  27<br />Percent: 0.96","Filename: SRR11951245<br />GC_Content:  28<br />Percent: 1.24","Filename: SRR11951245<br />GC_Content:  29<br />Percent: 1.42","Filename: SRR11951245<br />GC_Content:  30<br />Percent: 1.62","Filename: SRR11951245<br />GC_Content:  31<br />Percent: 1.96","Filename: SRR11951245<br />GC_Content:  32<br />Percent: 2.27","Filename: SRR11951245<br />GC_Content:  33<br />Percent: 2.44","Filename: SRR11951245<br />GC_Content:  34<br />Percent: 2.66","Filename: SRR11951245<br />GC_Content:  35<br />Percent: 2.99","Filename: SRR11951245<br />GC_Content:  36<br />Percent: 3.25","Filename: SRR11951245<br />GC_Content:  37<br />Percent: 3.41","Filename: SRR11951245<br />GC_Content:  38<br />Percent: 3.57","Filename: SRR11951245<br />GC_Content:  39<br />Percent: 3.68","Filename: SRR11951245<br />GC_Content:  40<br />Percent: 3.68","Filename: SRR11951245<br />GC_Content:  41<br />Percent: 3.69","Filename: SRR11951245<br />GC_Content:  42<br />Percent: 3.67","Filename: SRR11951245<br />GC_Content:  43<br />Percent: 3.61","Filename: SRR11951245<br />GC_Content:  44<br />Percent: 3.51","Filename: SRR11951245<br />GC_Content:  45<br />Percent: 3.50","Filename: SRR11951245<br />GC_Content:  46<br />Percent: 3.56","Filename: SRR11951245<br />GC_Content:  47<br />Percent: 3.48","Filename: SRR11951245<br />GC_Content:  48<br />Percent: 3.36","Filename: SRR11951245<br />GC_Content:  49<br />Percent: 3.26","Filename: SRR11951245<br />GC_Content:  50<br />Percent: 3.18","Filename: SRR11951245<br />GC_Content:  51<br />Percent: 3.09","Filename: SRR11951245<br />GC_Content:  52<br />Percent: 2.93","Filename: SRR11951245<br />GC_Content:  53<br />Percent: 2.73","Filename: SRR11951245<br />GC_Content:  54<br />Percent: 2.59","Filename: SRR11951245<br />GC_Content:  55<br />Percent: 2.40","Filename: SRR11951245<br />GC_Content:  56<br />Percent: 2.13","Filename: SRR11951245<br />GC_Content:  57<br />Percent: 1.86","Filename: SRR11951245<br />GC_Content:  58<br />Percent: 1.66","Filename: SRR11951245<br />GC_Content:  59<br />Percent: 1.47","Filename: SRR11951245<br />GC_Content:  60<br />Percent: 1.22","Filename: SRR11951245<br />GC_Content:  61<br />Percent: 0.99","Filename: SRR11951245<br />GC_Content:  62<br />Percent: 0.83","Filename: SRR11951245<br />GC_Content:  63<br />Percent: 0.70","Filename: SRR11951245<br />GC_Content:  64<br />Percent: 0.55","Filename: SRR11951245<br />GC_Content:  65<br />Percent: 0.43","Filename: SRR11951245<br />GC_Content:  66<br />Percent: 0.31","Filename: SRR11951245<br />GC_Content:  67<br />Percent: 0.25","Filename: SRR11951245<br />GC_Content:  68<br />Percent: 0.21","Filename: SRR11951245<br />GC_Content:  69<br />Percent: 0.16","Filename: SRR11951245<br />GC_Content:  70<br />Percent: 0.11","Filename: SRR11951245<br />GC_Content:  71<br />Percent: 0.08","Filename: SRR11951245<br />GC_Content:  72<br />Percent: 0.07","Filename: SRR11951245<br />GC_Content:  73<br />Percent: 0.05","Filename: SRR11951245<br />GC_Content:  74<br />Percent: 0.04","Filename: SRR11951245<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951245<br />GC_Content:  76<br />Percent: 0.02","Filename: SRR11951245<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951245<br />GC_Content:  78<br />Percent: 0.01","Filename: SRR11951245<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951245<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951245<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951245<br />GC_Content:  82<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  94<br />Percent: 0.01","Filename: SRR11951245<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951245<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951245<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(189,162,206,1)","dash":"solid"},"hoveron":"points","name":"SRR11951245","legendgroup":"SRR11951245","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.02,0.01,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.07,0.1,0.14,0.17,0.21,0.31,0.43,0.51,0.67,0.97,1.3,1.51,1.75,2.16,2.54,2.77,3.08,3.56,3.95,4.18,4.41,4.5,4.39,4.31,4.19,3.99,3.7,3.57,3.56,3.35,3.07,2.86,2.69,2.53,2.31,2.13,2.02,1.86,1.62,1.38,1.22,1.07,0.88,0.71,0.59,0.49,0.39,0.3,0.22,0.18,0.15,0.12,0.09,0.08,0.07,0.06,0.05,0.04,0.03,0.03,0.03,0.02,0.02,0.02,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0.01,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951246<br />GC_Content:   0<br />Percent: 0.02","Filename: SRR11951246<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951246<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951246<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951246<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951246<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951246<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951246<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951246<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951246<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951246<br />GC_Content:  18<br />Percent: 0.07","Filename: SRR11951246<br />GC_Content:  19<br />Percent: 0.10","Filename: SRR11951246<br />GC_Content:  20<br />Percent: 0.14","Filename: SRR11951246<br />GC_Content:  21<br />Percent: 0.17","Filename: SRR11951246<br />GC_Content:  22<br />Percent: 0.21","Filename: SRR11951246<br />GC_Content:  23<br />Percent: 0.31","Filename: SRR11951246<br />GC_Content:  24<br />Percent: 0.43","Filename: SRR11951246<br />GC_Content:  25<br />Percent: 0.51","Filename: SRR11951246<br />GC_Content:  26<br />Percent: 0.67","Filename: SRR11951246<br />GC_Content:  27<br />Percent: 0.97","Filename: SRR11951246<br />GC_Content:  28<br />Percent: 1.30","Filename: SRR11951246<br />GC_Content:  29<br />Percent: 1.51","Filename: SRR11951246<br />GC_Content:  30<br />Percent: 1.75","Filename: SRR11951246<br />GC_Content:  31<br />Percent: 2.16","Filename: SRR11951246<br />GC_Content:  32<br />Percent: 2.54","Filename: SRR11951246<br />GC_Content:  33<br />Percent: 2.77","Filename: SRR11951246<br />GC_Content:  34<br />Percent: 3.08","Filename: SRR11951246<br />GC_Content:  35<br />Percent: 3.56","Filename: SRR11951246<br />GC_Content:  36<br />Percent: 3.95","Filename: SRR11951246<br />GC_Content:  37<br />Percent: 4.18","Filename: SRR11951246<br />GC_Content:  38<br />Percent: 4.41","Filename: SRR11951246<br />GC_Content:  39<br />Percent: 4.50","Filename: SRR11951246<br />GC_Content:  40<br />Percent: 4.39","Filename: SRR11951246<br />GC_Content:  41<br />Percent: 4.31","Filename: SRR11951246<br />GC_Content:  42<br />Percent: 4.19","Filename: SRR11951246<br />GC_Content:  43<br />Percent: 3.99","Filename: SRR11951246<br />GC_Content:  44<br />Percent: 3.70","Filename: SRR11951246<br />GC_Content:  45<br />Percent: 3.57","Filename: SRR11951246<br />GC_Content:  46<br />Percent: 3.56","Filename: SRR11951246<br />GC_Content:  47<br />Percent: 3.35","Filename: SRR11951246<br />GC_Content:  48<br />Percent: 3.07","Filename: SRR11951246<br />GC_Content:  49<br />Percent: 2.86","Filename: SRR11951246<br />GC_Content:  50<br />Percent: 2.69","Filename: SRR11951246<br />GC_Content:  51<br />Percent: 2.53","Filename: SRR11951246<br />GC_Content:  52<br />Percent: 2.31","Filename: SRR11951246<br />GC_Content:  53<br />Percent: 2.13","Filename: SRR11951246<br />GC_Content:  54<br />Percent: 2.02","Filename: SRR11951246<br />GC_Content:  55<br />Percent: 1.86","Filename: SRR11951246<br />GC_Content:  56<br />Percent: 1.62","Filename: SRR11951246<br />GC_Content:  57<br />Percent: 1.38","Filename: SRR11951246<br />GC_Content:  58<br />Percent: 1.22","Filename: SRR11951246<br />GC_Content:  59<br />Percent: 1.07","Filename: SRR11951246<br />GC_Content:  60<br />Percent: 0.88","Filename: SRR11951246<br />GC_Content:  61<br />Percent: 0.71","Filename: SRR11951246<br />GC_Content:  62<br />Percent: 0.59","Filename: SRR11951246<br />GC_Content:  63<br />Percent: 0.49","Filename: SRR11951246<br />GC_Content:  64<br />Percent: 0.39","Filename: SRR11951246<br />GC_Content:  65<br />Percent: 0.30","Filename: SRR11951246<br />GC_Content:  66<br />Percent: 0.22","Filename: SRR11951246<br />GC_Content:  67<br />Percent: 0.18","Filename: SRR11951246<br />GC_Content:  68<br />Percent: 0.15","Filename: SRR11951246<br />GC_Content:  69<br />Percent: 0.12","Filename: SRR11951246<br />GC_Content:  70<br />Percent: 0.09","Filename: SRR11951246<br />GC_Content:  71<br />Percent: 0.08","Filename: SRR11951246<br />GC_Content:  72<br />Percent: 0.07","Filename: SRR11951246<br />GC_Content:  73<br />Percent: 0.06","Filename: SRR11951246<br />GC_Content:  74<br />Percent: 0.05","Filename: SRR11951246<br />GC_Content:  75<br />Percent: 0.04","Filename: SRR11951246<br />GC_Content:  76<br />Percent: 0.03","Filename: SRR11951246<br />GC_Content:  77<br />Percent: 0.03","Filename: SRR11951246<br />GC_Content:  78<br />Percent: 0.03","Filename: SRR11951246<br />GC_Content:  79<br />Percent: 0.02","Filename: SRR11951246<br />GC_Content:  80<br />Percent: 0.02","Filename: SRR11951246<br />GC_Content:  81<br />Percent: 0.02","Filename: SRR11951246<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951246<br />GC_Content:  83<br />Percent: 0.01","Filename: SRR11951246<br />GC_Content:  84<br />Percent: 0.01","Filename: SRR11951246<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:  94<br />Percent: 0.01","Filename: SRR11951246<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951246<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951246<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(143,106,177,1)","dash":"solid"},"hoveron":"points","name":"SRR11951246","legendgroup":"SRR11951246","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.02,0.01,0,0,0,0,0,0,0,0.01,0.01,0.01,0.02,0.02,0.03,0.04,0.05,0.06,0.08,0.11,0.15,0.17,0.22,0.31,0.43,0.51,0.65,0.92,1.22,1.4,1.63,2,2.33,2.53,2.79,3.17,3.47,3.65,3.83,3.93,3.9,3.89,3.85,3.74,3.59,3.54,3.58,3.47,3.3,3.17,3.06,2.95,2.76,2.56,2.42,2.24,1.97,1.71,1.53,1.35,1.12,0.9,0.75,0.63,0.5,0.38,0.28,0.22,0.19,0.14,0.1,0.08,0.07,0.05,0.04,0.03,0.03,0.02,0.01,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0.01,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951247<br />GC_Content:   0<br />Percent: 0.02","Filename: SRR11951247<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951247<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:   9<br />Percent: 0.01","Filename: SRR11951247<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951247<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951247<br />GC_Content:  12<br />Percent: 0.02","Filename: SRR11951247<br />GC_Content:  13<br />Percent: 0.02","Filename: SRR11951247<br />GC_Content:  14<br />Percent: 0.03","Filename: SRR11951247<br />GC_Content:  15<br />Percent: 0.04","Filename: SRR11951247<br />GC_Content:  16<br />Percent: 0.05","Filename: SRR11951247<br />GC_Content:  17<br />Percent: 0.06","Filename: SRR11951247<br />GC_Content:  18<br />Percent: 0.08","Filename: SRR11951247<br />GC_Content:  19<br />Percent: 0.11","Filename: SRR11951247<br />GC_Content:  20<br />Percent: 0.15","Filename: SRR11951247<br />GC_Content:  21<br />Percent: 0.17","Filename: SRR11951247<br />GC_Content:  22<br />Percent: 0.22","Filename: SRR11951247<br />GC_Content:  23<br />Percent: 0.31","Filename: SRR11951247<br />GC_Content:  24<br />Percent: 0.43","Filename: SRR11951247<br />GC_Content:  25<br />Percent: 0.51","Filename: SRR11951247<br />GC_Content:  26<br />Percent: 0.65","Filename: SRR11951247<br />GC_Content:  27<br />Percent: 0.92","Filename: SRR11951247<br />GC_Content:  28<br />Percent: 1.22","Filename: SRR11951247<br />GC_Content:  29<br />Percent: 1.40","Filename: SRR11951247<br />GC_Content:  30<br />Percent: 1.63","Filename: SRR11951247<br />GC_Content:  31<br />Percent: 2.00","Filename: SRR11951247<br />GC_Content:  32<br />Percent: 2.33","Filename: SRR11951247<br />GC_Content:  33<br />Percent: 2.53","Filename: SRR11951247<br />GC_Content:  34<br />Percent: 2.79","Filename: SRR11951247<br />GC_Content:  35<br />Percent: 3.17","Filename: SRR11951247<br />GC_Content:  36<br />Percent: 3.47","Filename: SRR11951247<br />GC_Content:  37<br />Percent: 3.65","Filename: SRR11951247<br />GC_Content:  38<br />Percent: 3.83","Filename: SRR11951247<br />GC_Content:  39<br />Percent: 3.93","Filename: SRR11951247<br />GC_Content:  40<br />Percent: 3.90","Filename: SRR11951247<br />GC_Content:  41<br />Percent: 3.89","Filename: SRR11951247<br />GC_Content:  42<br />Percent: 3.85","Filename: SRR11951247<br />GC_Content:  43<br />Percent: 3.74","Filename: SRR11951247<br />GC_Content:  44<br />Percent: 3.59","Filename: SRR11951247<br />GC_Content:  45<br />Percent: 3.54","Filename: SRR11951247<br />GC_Content:  46<br />Percent: 3.58","Filename: SRR11951247<br />GC_Content:  47<br />Percent: 3.47","Filename: SRR11951247<br />GC_Content:  48<br />Percent: 3.30","Filename: SRR11951247<br />GC_Content:  49<br />Percent: 3.17","Filename: SRR11951247<br />GC_Content:  50<br />Percent: 3.06","Filename: SRR11951247<br />GC_Content:  51<br />Percent: 2.95","Filename: SRR11951247<br />GC_Content:  52<br />Percent: 2.76","Filename: SRR11951247<br />GC_Content:  53<br />Percent: 2.56","Filename: SRR11951247<br />GC_Content:  54<br />Percent: 2.42","Filename: SRR11951247<br />GC_Content:  55<br />Percent: 2.24","Filename: SRR11951247<br />GC_Content:  56<br />Percent: 1.97","Filename: SRR11951247<br />GC_Content:  57<br />Percent: 1.71","Filename: SRR11951247<br />GC_Content:  58<br />Percent: 1.53","Filename: SRR11951247<br />GC_Content:  59<br />Percent: 1.35","Filename: SRR11951247<br />GC_Content:  60<br />Percent: 1.12","Filename: SRR11951247<br />GC_Content:  61<br />Percent: 0.90","Filename: SRR11951247<br />GC_Content:  62<br />Percent: 0.75","Filename: SRR11951247<br />GC_Content:  63<br />Percent: 0.63","Filename: SRR11951247<br />GC_Content:  64<br />Percent: 0.50","Filename: SRR11951247<br />GC_Content:  65<br />Percent: 0.38","Filename: SRR11951247<br />GC_Content:  66<br />Percent: 0.28","Filename: SRR11951247<br />GC_Content:  67<br />Percent: 0.22","Filename: SRR11951247<br />GC_Content:  68<br />Percent: 0.19","Filename: SRR11951247<br />GC_Content:  69<br />Percent: 0.14","Filename: SRR11951247<br />GC_Content:  70<br />Percent: 0.10","Filename: SRR11951247<br />GC_Content:  71<br />Percent: 0.08","Filename: SRR11951247<br />GC_Content:  72<br />Percent: 0.07","Filename: SRR11951247<br />GC_Content:  73<br />Percent: 0.05","Filename: SRR11951247<br />GC_Content:  74<br />Percent: 0.04","Filename: SRR11951247<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951247<br />GC_Content:  76<br />Percent: 0.03","Filename: SRR11951247<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951247<br />GC_Content:  78<br />Percent: 0.01","Filename: SRR11951247<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951247<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951247<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951247<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951247<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:  94<br />Percent: 0.01","Filename: SRR11951247<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951247<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951247<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(118,77,153,1)","dash":"solid"},"hoveron":"points","name":"SRR11951247","legendgroup":"SRR11951247","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.01,0.01,0,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.08,0.11,0.13,0.17,0.25,0.34,0.41,0.55,0.8,1.06,1.22,1.41,1.73,2.03,2.2,2.4,2.72,3.02,3.19,3.39,3.53,3.54,3.61,3.62,3.58,3.53,3.58,3.69,3.65,3.56,3.51,3.46,3.38,3.22,3.02,2.87,2.67,2.37,2.07,1.85,1.62,1.34,1.08,0.89,0.75,0.59,0.46,0.34,0.27,0.22,0.16,0.11,0.09,0.07,0.05,0.04,0.03,0.02,0.02,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951248<br />GC_Content:   0<br />Percent: 0.01","Filename: SRR11951248<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951248<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  10<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951248<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951248<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951248<br />GC_Content:  14<br />Percent: 0.01","Filename: SRR11951248<br />GC_Content:  15<br />Percent: 0.02","Filename: SRR11951248<br />GC_Content:  16<br />Percent: 0.03","Filename: SRR11951248<br />GC_Content:  17<br />Percent: 0.04","Filename: SRR11951248<br />GC_Content:  18<br />Percent: 0.05","Filename: SRR11951248<br />GC_Content:  19<br />Percent: 0.08","Filename: SRR11951248<br />GC_Content:  20<br />Percent: 0.11","Filename: SRR11951248<br />GC_Content:  21<br />Percent: 0.13","Filename: SRR11951248<br />GC_Content:  22<br />Percent: 0.17","Filename: SRR11951248<br />GC_Content:  23<br />Percent: 0.25","Filename: SRR11951248<br />GC_Content:  24<br />Percent: 0.34","Filename: SRR11951248<br />GC_Content:  25<br />Percent: 0.41","Filename: SRR11951248<br />GC_Content:  26<br />Percent: 0.55","Filename: SRR11951248<br />GC_Content:  27<br />Percent: 0.80","Filename: SRR11951248<br />GC_Content:  28<br />Percent: 1.06","Filename: SRR11951248<br />GC_Content:  29<br />Percent: 1.22","Filename: SRR11951248<br />GC_Content:  30<br />Percent: 1.41","Filename: SRR11951248<br />GC_Content:  31<br />Percent: 1.73","Filename: SRR11951248<br />GC_Content:  32<br />Percent: 2.03","Filename: SRR11951248<br />GC_Content:  33<br />Percent: 2.20","Filename: SRR11951248<br />GC_Content:  34<br />Percent: 2.40","Filename: SRR11951248<br />GC_Content:  35<br />Percent: 2.72","Filename: SRR11951248<br />GC_Content:  36<br />Percent: 3.02","Filename: SRR11951248<br />GC_Content:  37<br />Percent: 3.19","Filename: SRR11951248<br />GC_Content:  38<br />Percent: 3.39","Filename: SRR11951248<br />GC_Content:  39<br />Percent: 3.53","Filename: SRR11951248<br />GC_Content:  40<br />Percent: 3.54","Filename: SRR11951248<br />GC_Content:  41<br />Percent: 3.61","Filename: SRR11951248<br />GC_Content:  42<br />Percent: 3.62","Filename: SRR11951248<br />GC_Content:  43<br />Percent: 3.58","Filename: SRR11951248<br />GC_Content:  44<br />Percent: 3.53","Filename: SRR11951248<br />GC_Content:  45<br />Percent: 3.58","Filename: SRR11951248<br />GC_Content:  46<br />Percent: 3.69","Filename: SRR11951248<br />GC_Content:  47<br />Percent: 3.65","Filename: SRR11951248<br />GC_Content:  48<br />Percent: 3.56","Filename: SRR11951248<br />GC_Content:  49<br />Percent: 3.51","Filename: SRR11951248<br />GC_Content:  50<br />Percent: 3.46","Filename: SRR11951248<br />GC_Content:  51<br />Percent: 3.38","Filename: SRR11951248<br />GC_Content:  52<br />Percent: 3.22","Filename: SRR11951248<br />GC_Content:  53<br />Percent: 3.02","Filename: SRR11951248<br />GC_Content:  54<br />Percent: 2.87","Filename: SRR11951248<br />GC_Content:  55<br />Percent: 2.67","Filename: SRR11951248<br />GC_Content:  56<br />Percent: 2.37","Filename: SRR11951248<br />GC_Content:  57<br />Percent: 2.07","Filename: SRR11951248<br />GC_Content:  58<br />Percent: 1.85","Filename: SRR11951248<br />GC_Content:  59<br />Percent: 1.62","Filename: SRR11951248<br />GC_Content:  60<br />Percent: 1.34","Filename: SRR11951248<br />GC_Content:  61<br />Percent: 1.08","Filename: SRR11951248<br />GC_Content:  62<br />Percent: 0.89","Filename: SRR11951248<br />GC_Content:  63<br />Percent: 0.75","Filename: SRR11951248<br />GC_Content:  64<br />Percent: 0.59","Filename: SRR11951248<br />GC_Content:  65<br />Percent: 0.46","Filename: SRR11951248<br />GC_Content:  66<br />Percent: 0.34","Filename: SRR11951248<br />GC_Content:  67<br />Percent: 0.27","Filename: SRR11951248<br />GC_Content:  68<br />Percent: 0.22","Filename: SRR11951248<br />GC_Content:  69<br />Percent: 0.16","Filename: SRR11951248<br />GC_Content:  70<br />Percent: 0.11","Filename: SRR11951248<br />GC_Content:  71<br />Percent: 0.09","Filename: SRR11951248<br />GC_Content:  72<br />Percent: 0.07","Filename: SRR11951248<br />GC_Content:  73<br />Percent: 0.05","Filename: SRR11951248<br />GC_Content:  74<br />Percent: 0.04","Filename: SRR11951248<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951248<br />GC_Content:  76<br />Percent: 0.02","Filename: SRR11951248<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951248<br />GC_Content:  78<br />Percent: 0.01","Filename: SRR11951248<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951248<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951248<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951248<br />GC_Content:  82<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  94<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951248<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951248<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(190,170,153,1)","dash":"solid"},"hoveron":"points","name":"SRR11951248","legendgroup":"SRR11951248","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.03,0.02,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.06,0.09,0.13,0.15,0.2,0.28,0.39,0.47,0.6,0.86,1.13,1.3,1.51,1.85,2.17,2.36,2.59,2.94,3.22,3.38,3.56,3.68,3.69,3.71,3.7,3.64,3.56,3.56,3.63,3.56,3.45,3.35,3.28,3.18,3,2.8,2.66,2.47,2.18,1.91,1.7,1.5,1.25,1.01,0.84,0.71,0.56,0.44,0.32,0.25,0.21,0.16,0.11,0.09,0.07,0.05,0.04,0.03,0.02,0.02,0.01,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951249<br />GC_Content:   0<br />Percent: 0.03","Filename: SRR11951249<br />GC_Content:   1<br />Percent: 0.02","Filename: SRR11951249<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951249<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951249<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951249<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951249<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951249<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951249<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951249<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951249<br />GC_Content:  18<br />Percent: 0.06","Filename: SRR11951249<br />GC_Content:  19<br />Percent: 0.09","Filename: SRR11951249<br />GC_Content:  20<br />Percent: 0.13","Filename: SRR11951249<br />GC_Content:  21<br />Percent: 0.15","Filename: SRR11951249<br />GC_Content:  22<br />Percent: 0.20","Filename: SRR11951249<br />GC_Content:  23<br />Percent: 0.28","Filename: SRR11951249<br />GC_Content:  24<br />Percent: 0.39","Filename: SRR11951249<br />GC_Content:  25<br />Percent: 0.47","Filename: SRR11951249<br />GC_Content:  26<br />Percent: 0.60","Filename: SRR11951249<br />GC_Content:  27<br />Percent: 0.86","Filename: SRR11951249<br />GC_Content:  28<br />Percent: 1.13","Filename: SRR11951249<br />GC_Content:  29<br />Percent: 1.30","Filename: SRR11951249<br />GC_Content:  30<br />Percent: 1.51","Filename: SRR11951249<br />GC_Content:  31<br />Percent: 1.85","Filename: SRR11951249<br />GC_Content:  32<br />Percent: 2.17","Filename: SRR11951249<br />GC_Content:  33<br />Percent: 2.36","Filename: SRR11951249<br />GC_Content:  34<br />Percent: 2.59","Filename: SRR11951249<br />GC_Content:  35<br />Percent: 2.94","Filename: SRR11951249<br />GC_Content:  36<br />Percent: 3.22","Filename: SRR11951249<br />GC_Content:  37<br />Percent: 3.38","Filename: SRR11951249<br />GC_Content:  38<br />Percent: 3.56","Filename: SRR11951249<br />GC_Content:  39<br />Percent: 3.68","Filename: SRR11951249<br />GC_Content:  40<br />Percent: 3.69","Filename: SRR11951249<br />GC_Content:  41<br />Percent: 3.71","Filename: SRR11951249<br />GC_Content:  42<br />Percent: 3.70","Filename: SRR11951249<br />GC_Content:  43<br />Percent: 3.64","Filename: SRR11951249<br />GC_Content:  44<br />Percent: 3.56","Filename: SRR11951249<br />GC_Content:  45<br />Percent: 3.56","Filename: SRR11951249<br />GC_Content:  46<br />Percent: 3.63","Filename: SRR11951249<br />GC_Content:  47<br />Percent: 3.56","Filename: SRR11951249<br />GC_Content:  48<br />Percent: 3.45","Filename: SRR11951249<br />GC_Content:  49<br />Percent: 3.35","Filename: SRR11951249<br />GC_Content:  50<br />Percent: 3.28","Filename: SRR11951249<br />GC_Content:  51<br />Percent: 3.18","Filename: SRR11951249<br />GC_Content:  52<br />Percent: 3.00","Filename: SRR11951249<br />GC_Content:  53<br />Percent: 2.80","Filename: SRR11951249<br />GC_Content:  54<br />Percent: 2.66","Filename: SRR11951249<br />GC_Content:  55<br />Percent: 2.47","Filename: SRR11951249<br />GC_Content:  56<br />Percent: 2.18","Filename: SRR11951249<br />GC_Content:  57<br />Percent: 1.91","Filename: SRR11951249<br />GC_Content:  58<br />Percent: 1.70","Filename: SRR11951249<br />GC_Content:  59<br />Percent: 1.50","Filename: SRR11951249<br />GC_Content:  60<br />Percent: 1.25","Filename: SRR11951249<br />GC_Content:  61<br />Percent: 1.01","Filename: SRR11951249<br />GC_Content:  62<br />Percent: 0.84","Filename: SRR11951249<br />GC_Content:  63<br />Percent: 0.71","Filename: SRR11951249<br />GC_Content:  64<br />Percent: 0.56","Filename: SRR11951249<br />GC_Content:  65<br />Percent: 0.44","Filename: SRR11951249<br />GC_Content:  66<br />Percent: 0.32","Filename: SRR11951249<br />GC_Content:  67<br />Percent: 0.25","Filename: SRR11951249<br />GC_Content:  68<br />Percent: 0.21","Filename: SRR11951249<br />GC_Content:  69<br />Percent: 0.16","Filename: SRR11951249<br />GC_Content:  70<br />Percent: 0.11","Filename: SRR11951249<br />GC_Content:  71<br />Percent: 0.09","Filename: SRR11951249<br />GC_Content:  72<br />Percent: 0.07","Filename: SRR11951249<br />GC_Content:  73<br />Percent: 0.05","Filename: SRR11951249<br />GC_Content:  74<br />Percent: 0.04","Filename: SRR11951249<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951249<br />GC_Content:  76<br />Percent: 0.02","Filename: SRR11951249<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951249<br />GC_Content:  78<br />Percent: 0.01","Filename: SRR11951249<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951249<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951249<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951249<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951249<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  94<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951249<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951249<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(251,247,148,1)","dash":"solid"},"hoveron":"points","name":"SRR11951249","legendgroup":"SRR11951249","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.04,0.02,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.06,0.09,0.13,0.16,0.2,0.29,0.4,0.48,0.62,0.9,1.18,1.35,1.55,1.88,2.17,2.33,2.56,2.89,3.18,3.35,3.53,3.65,3.65,3.68,3.68,3.61,3.53,3.57,3.67,3.61,3.49,3.4,3.32,3.22,3.04,2.83,2.68,2.49,2.2,1.91,1.7,1.49,1.22,0.99,0.82,0.69,0.54,0.42,0.3,0.24,0.2,0.15,0.1,0.08,0.06,0.05,0.03,0.03,0.02,0.02,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951250<br />GC_Content:   0<br />Percent: 0.04","Filename: SRR11951250<br />GC_Content:   1<br />Percent: 0.02","Filename: SRR11951250<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951250<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951250<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951250<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951250<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951250<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951250<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951250<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951250<br />GC_Content:  18<br />Percent: 0.06","Filename: SRR11951250<br />GC_Content:  19<br />Percent: 0.09","Filename: SRR11951250<br />GC_Content:  20<br />Percent: 0.13","Filename: SRR11951250<br />GC_Content:  21<br />Percent: 0.16","Filename: SRR11951250<br />GC_Content:  22<br />Percent: 0.20","Filename: SRR11951250<br />GC_Content:  23<br />Percent: 0.29","Filename: SRR11951250<br />GC_Content:  24<br />Percent: 0.40","Filename: SRR11951250<br />GC_Content:  25<br />Percent: 0.48","Filename: SRR11951250<br />GC_Content:  26<br />Percent: 0.62","Filename: SRR11951250<br />GC_Content:  27<br />Percent: 0.90","Filename: SRR11951250<br />GC_Content:  28<br />Percent: 1.18","Filename: SRR11951250<br />GC_Content:  29<br />Percent: 1.35","Filename: SRR11951250<br />GC_Content:  30<br />Percent: 1.55","Filename: SRR11951250<br />GC_Content:  31<br />Percent: 1.88","Filename: SRR11951250<br />GC_Content:  32<br />Percent: 2.17","Filename: SRR11951250<br />GC_Content:  33<br />Percent: 2.33","Filename: SRR11951250<br />GC_Content:  34<br />Percent: 2.56","Filename: SRR11951250<br />GC_Content:  35<br />Percent: 2.89","Filename: SRR11951250<br />GC_Content:  36<br />Percent: 3.18","Filename: SRR11951250<br />GC_Content:  37<br />Percent: 3.35","Filename: SRR11951250<br />GC_Content:  38<br />Percent: 3.53","Filename: SRR11951250<br />GC_Content:  39<br />Percent: 3.65","Filename: SRR11951250<br />GC_Content:  40<br />Percent: 3.65","Filename: SRR11951250<br />GC_Content:  41<br />Percent: 3.68","Filename: SRR11951250<br />GC_Content:  42<br />Percent: 3.68","Filename: SRR11951250<br />GC_Content:  43<br />Percent: 3.61","Filename: SRR11951250<br />GC_Content:  44<br />Percent: 3.53","Filename: SRR11951250<br />GC_Content:  45<br />Percent: 3.57","Filename: SRR11951250<br />GC_Content:  46<br />Percent: 3.67","Filename: SRR11951250<br />GC_Content:  47<br />Percent: 3.61","Filename: SRR11951250<br />GC_Content:  48<br />Percent: 3.49","Filename: SRR11951250<br />GC_Content:  49<br />Percent: 3.40","Filename: SRR11951250<br />GC_Content:  50<br />Percent: 3.32","Filename: SRR11951250<br />GC_Content:  51<br />Percent: 3.22","Filename: SRR11951250<br />GC_Content:  52<br />Percent: 3.04","Filename: SRR11951250<br />GC_Content:  53<br />Percent: 2.83","Filename: SRR11951250<br />GC_Content:  54<br />Percent: 2.68","Filename: SRR11951250<br />GC_Content:  55<br />Percent: 2.49","Filename: SRR11951250<br />GC_Content:  56<br />Percent: 2.20","Filename: SRR11951250<br />GC_Content:  57<br />Percent: 1.91","Filename: SRR11951250<br />GC_Content:  58<br />Percent: 1.70","Filename: SRR11951250<br />GC_Content:  59<br />Percent: 1.49","Filename: SRR11951250<br />GC_Content:  60<br />Percent: 1.22","Filename: SRR11951250<br />GC_Content:  61<br />Percent: 0.99","Filename: SRR11951250<br />GC_Content:  62<br />Percent: 0.82","Filename: SRR11951250<br />GC_Content:  63<br />Percent: 0.69","Filename: SRR11951250<br />GC_Content:  64<br />Percent: 0.54","Filename: SRR11951250<br />GC_Content:  65<br />Percent: 0.42","Filename: SRR11951250<br />GC_Content:  66<br />Percent: 0.30","Filename: SRR11951250<br />GC_Content:  67<br />Percent: 0.24","Filename: SRR11951250<br />GC_Content:  68<br />Percent: 0.20","Filename: SRR11951250<br />GC_Content:  69<br />Percent: 0.15","Filename: SRR11951250<br />GC_Content:  70<br />Percent: 0.10","Filename: SRR11951250<br />GC_Content:  71<br />Percent: 0.08","Filename: SRR11951250<br />GC_Content:  72<br />Percent: 0.06","Filename: SRR11951250<br />GC_Content:  73<br />Percent: 0.05","Filename: SRR11951250<br />GC_Content:  74<br />Percent: 0.03","Filename: SRR11951250<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951250<br />GC_Content:  76<br />Percent: 0.02","Filename: SRR11951250<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951250<br />GC_Content:  78<br />Percent: 0.01","Filename: SRR11951250<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951250<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951250<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951250<br />GC_Content:  82<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  94<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951250<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951250<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(214,168,94,1)","dash":"solid"},"hoveron":"points","name":"SRR11951250","legendgroup":"SRR11951250","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0.01,0.01,0,0,0,0,0,0,0,0,0.01,0.01,0.01,0.01,0.02,0.03,0.04,0.05,0.07,0.1,0.15,0.18,0.23,0.33,0.45,0.53,0.68,0.94,1.21,1.38,1.57,1.88,2.16,2.32,2.52,2.8,3.03,3.17,3.32,3.42,3.45,3.5,3.52,3.5,3.46,3.51,3.59,3.56,3.49,3.42,3.37,3.29,3.13,2.93,2.78,2.59,2.3,2.01,1.8,1.58,1.31,1.06,0.89,0.75,0.59,0.46,0.33,0.26,0.22,0.16,0.12,0.09,0.07,0.05,0.04,0.03,0.02,0.02,0.01,0.01,0.01,0.01,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0.01],"text":["Filename: SRR11951251<br />GC_Content:   0<br />Percent: 0.01","Filename: SRR11951251<br />GC_Content:   1<br />Percent: 0.01","Filename: SRR11951251<br />GC_Content:   2<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:   3<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:   4<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:   5<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:   6<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:   7<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:   8<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:   9<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  10<br />Percent: 0.01","Filename: SRR11951251<br />GC_Content:  11<br />Percent: 0.01","Filename: SRR11951251<br />GC_Content:  12<br />Percent: 0.01","Filename: SRR11951251<br />GC_Content:  13<br />Percent: 0.01","Filename: SRR11951251<br />GC_Content:  14<br />Percent: 0.02","Filename: SRR11951251<br />GC_Content:  15<br />Percent: 0.03","Filename: SRR11951251<br />GC_Content:  16<br />Percent: 0.04","Filename: SRR11951251<br />GC_Content:  17<br />Percent: 0.05","Filename: SRR11951251<br />GC_Content:  18<br />Percent: 0.07","Filename: SRR11951251<br />GC_Content:  19<br />Percent: 0.10","Filename: SRR11951251<br />GC_Content:  20<br />Percent: 0.15","Filename: SRR11951251<br />GC_Content:  21<br />Percent: 0.18","Filename: SRR11951251<br />GC_Content:  22<br />Percent: 0.23","Filename: SRR11951251<br />GC_Content:  23<br />Percent: 0.33","Filename: SRR11951251<br />GC_Content:  24<br />Percent: 0.45","Filename: SRR11951251<br />GC_Content:  25<br />Percent: 0.53","Filename: SRR11951251<br />GC_Content:  26<br />Percent: 0.68","Filename: SRR11951251<br />GC_Content:  27<br />Percent: 0.94","Filename: SRR11951251<br />GC_Content:  28<br />Percent: 1.21","Filename: SRR11951251<br />GC_Content:  29<br />Percent: 1.38","Filename: SRR11951251<br />GC_Content:  30<br />Percent: 1.57","Filename: SRR11951251<br />GC_Content:  31<br />Percent: 1.88","Filename: SRR11951251<br />GC_Content:  32<br />Percent: 2.16","Filename: SRR11951251<br />GC_Content:  33<br />Percent: 2.32","Filename: SRR11951251<br />GC_Content:  34<br />Percent: 2.52","Filename: SRR11951251<br />GC_Content:  35<br />Percent: 2.80","Filename: SRR11951251<br />GC_Content:  36<br />Percent: 3.03","Filename: SRR11951251<br />GC_Content:  37<br />Percent: 3.17","Filename: SRR11951251<br />GC_Content:  38<br />Percent: 3.32","Filename: SRR11951251<br />GC_Content:  39<br />Percent: 3.42","Filename: SRR11951251<br />GC_Content:  40<br />Percent: 3.45","Filename: SRR11951251<br />GC_Content:  41<br />Percent: 3.50","Filename: SRR11951251<br />GC_Content:  42<br />Percent: 3.52","Filename: SRR11951251<br />GC_Content:  43<br />Percent: 3.50","Filename: SRR11951251<br />GC_Content:  44<br />Percent: 3.46","Filename: SRR11951251<br />GC_Content:  45<br />Percent: 3.51","Filename: SRR11951251<br />GC_Content:  46<br />Percent: 3.59","Filename: SRR11951251<br />GC_Content:  47<br />Percent: 3.56","Filename: SRR11951251<br />GC_Content:  48<br />Percent: 3.49","Filename: SRR11951251<br />GC_Content:  49<br />Percent: 3.42","Filename: SRR11951251<br />GC_Content:  50<br />Percent: 3.37","Filename: SRR11951251<br />GC_Content:  51<br />Percent: 3.29","Filename: SRR11951251<br />GC_Content:  52<br />Percent: 3.13","Filename: SRR11951251<br />GC_Content:  53<br />Percent: 2.93","Filename: SRR11951251<br />GC_Content:  54<br />Percent: 2.78","Filename: SRR11951251<br />GC_Content:  55<br />Percent: 2.59","Filename: SRR11951251<br />GC_Content:  56<br />Percent: 2.30","Filename: SRR11951251<br />GC_Content:  57<br />Percent: 2.01","Filename: SRR11951251<br />GC_Content:  58<br />Percent: 1.80","Filename: SRR11951251<br />GC_Content:  59<br />Percent: 1.58","Filename: SRR11951251<br />GC_Content:  60<br />Percent: 1.31","Filename: SRR11951251<br />GC_Content:  61<br />Percent: 1.06","Filename: SRR11951251<br />GC_Content:  62<br />Percent: 0.89","Filename: SRR11951251<br />GC_Content:  63<br />Percent: 0.75","Filename: SRR11951251<br />GC_Content:  64<br />Percent: 0.59","Filename: SRR11951251<br />GC_Content:  65<br />Percent: 0.46","Filename: SRR11951251<br />GC_Content:  66<br />Percent: 0.33","Filename: SRR11951251<br />GC_Content:  67<br />Percent: 0.26","Filename: SRR11951251<br />GC_Content:  68<br />Percent: 0.22","Filename: SRR11951251<br />GC_Content:  69<br />Percent: 0.16","Filename: SRR11951251<br />GC_Content:  70<br />Percent: 0.12","Filename: SRR11951251<br />GC_Content:  71<br />Percent: 0.09","Filename: SRR11951251<br />GC_Content:  72<br />Percent: 0.07","Filename: SRR11951251<br />GC_Content:  73<br />Percent: 0.05","Filename: SRR11951251<br />GC_Content:  74<br />Percent: 0.04","Filename: SRR11951251<br />GC_Content:  75<br />Percent: 0.03","Filename: SRR11951251<br />GC_Content:  76<br />Percent: 0.02","Filename: SRR11951251<br />GC_Content:  77<br />Percent: 0.02","Filename: SRR11951251<br />GC_Content:  78<br />Percent: 0.01","Filename: SRR11951251<br />GC_Content:  79<br />Percent: 0.01","Filename: SRR11951251<br />GC_Content:  80<br />Percent: 0.01","Filename: SRR11951251<br />GC_Content:  81<br />Percent: 0.01","Filename: SRR11951251<br />GC_Content:  82<br />Percent: 0.01","Filename: SRR11951251<br />GC_Content:  83<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  84<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  85<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  86<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  87<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  88<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  89<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  90<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  91<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  92<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  93<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  94<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  95<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  96<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  97<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  98<br />Percent: 0.00","Filename: SRR11951251<br />GC_Content:  99<br />Percent: 0.01","Filename: SRR11951251<br />GC_Content: 100<br />Percent: 0.01"],"type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(177,89,40,1)","dash":"solid"},"hoveron":"points","name":"SRR11951251","legendgroup":"SRR11951251","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":25.7412480974125,"r":7.30593607305936,"b":39.6955859969559,"l":37.2602739726027},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-2,102],"tickmode":"array","ticktext":["0","10","20","30","40","50","60","70","80","90","100"],"tickvals":[0,10,20,30,40,50,60,70,80,90,100],"categoryorder":"array","categoryarray":["0","10","20","30","40","50","60","70","80","90","100"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":{"text":"GC Content (%)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.2455,5.1555],"tickmode":"array","ticktext":["0%","1%","2%","3%","4%","5%"],"tickvals":[0,1,2,3,4,5],"categoryorder":"array","categoryarray":["0%","1%","2%","3%","4%","5%"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"Reads (%)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"19e8be1077ea6e":{"colour":{},"x":{},"y":{},"type":"scatter"}},"cur_data":"19e8be1077ea6e","visdat":{"19e8be1077ea6e":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p class="caption">
<em>GC content distributions for all samples. Hover for more details.</em>
</p>
</div>
</div>
</div>
<div id="trimmed-data" class="section level2">
<h2>Trimmed data</h2>
<p>Trimming software was setup carefully with the intention to perform downstream detection of short variants following the <code>GATK</code> workflow. Part of this process involves the removal of duplicate reads which relies on detection of duplicates by comparing sequences in the 5’ positions. Trimming can therefore be detrimental to downstream analysis. As such, the trimming procedure was performed more so as a filtering step, where reads were discarded if:</p>
<ol style="list-style-type: decimal">
<li>40% of the bases did not meet the threshold of a 20 phred quality score.</li>
<li>reads were shorter than 35 base pairs in length.</li>
</ol>
<pre class="r"><code>trimFqc &lt;- list.files(
  path = here(&quot;01_trim/FastQC/&quot;),
  pattern = &quot;zip&quot;,
  full.names = TRUE
) %&gt;%
  FastqcDataList()
trimStats &lt;- readTotals(rawFqc) %&gt;%
  dplyr::rename(Raw = Total_Sequences) %&gt;%
  left_join(readTotals(trimFqc), by = &quot;Filename&quot;) %&gt;%
  dplyr::rename(Trimmed = Total_Sequences) %&gt;%
  mutate(
    Discarded = 1 - Trimmed/Raw,
    Retained = Trimmed / Raw
  )</code></pre>
<p>After trimming between <em>0.48%</em> and <em>0.83%</em> of reads were discarded.</p>
</div>
<div id="aligned-data" class="section level2">
<h2>Aligned data</h2>
<p>Trimmed reads were aligned to GRCz11 reference genome (Ensembl release 101) with <code>STAR 2.7.7a</code>. <code>STAR</code>’s two-pass mode was implemented to achieve better alignments around novel splice junctions.</p>
</div>
</div>
<div id="gatk-short-variant-discovery" class="section level1">
<h1>GATK short variant discovery</h1>
<div id="duplicates" class="section level2">
<h2>Duplicates</h2>
<p>Marking duplicates is important because almost all statistical models for variant calling assume some sort of independence between measurements. Duplicates that arise from PCR artifacts are not independent. <code>Picard MarkDuplicates</code> tool classifies duplicates by comparing sequences in the 5 prime position of reads and read pairs in a <code>BAM</code> file. This is indeed a conservative approach as there remains the possibility of marking non-technical duplicates that appear to be duplicates by chance.</p>
<p>Duplicates were identified with <code>Picard MarkDuplicates</code> and the representative read was chosen by random, as opposed to a base quality scoring strategy, such that reference allele mapping bias was avoided for downstream Allele Specific Expression (ASE) testing.</p>
<pre class="r"><code>dupeMetrics &lt;- list.files(here(&quot;03_markDuplicates/log/&quot;), full.names = TRUE) %&gt;%
  lapply(function(x){
    sample &lt;- basename(x) %&gt;%
      str_remove(&quot;.tsv&quot;)
    read_tsv(x, comment = &quot;#&quot;) %&gt;%
      mutate(sample = sample)
  }) %&gt;%
  bind_rows() %&gt;%
  dplyr::select(
    sample, 
    reads = UNPAIRED_READS_EXAMINED, 
    secondary = SECONDARY_OR_SUPPLEMENTARY_RDS,
    percent.duplication = PERCENT_DUPLICATION
  )</code></pre>
<p>Between <em>63.54%</em> and <em>79.44%</em> of reads were marked as duplicates across all samples. These large proportions were noted as a concern for downstream processing due to a substantial loss of information when duplicates are excluded. Such large estimates are likely due to the conservative nature of the duplicate detection strategy, which is hard to avoid when a dataset consists of single-end reads, and therefore the software can only classify dulplicates based on a single position.</p>
</div>
<div id="base-quality-score-recalibration" class="section level2">
<h2>Base quality score recalibration</h2>
<p>Base quality score recalibration (BQSR) is a recommended technique in the <code>GATK</code> workflow which is used to assign accurate confidence scores to each sequenced base. Real data contains systematic biases due to artifacts of sample preparation, sequencing and mapping. BQSR aims to correct for the range of these types of artifacts, as downstream tools rely on base qualities to weigh the variant calls they make. The process involves two keys steps performed by tools <code>BaseRecalibrator</code> and <code>ApplyBQSR</code> in the <code>GATK</code> suite.</p>
<p><strong>BaseRecalibrator</strong></p>
<p><code>BaseRecalibrator</code> builds a model of covariation based on the input data and a set of known variants. Since this dataset originates from zebrafish which are highly polymorphic, the set of known variants was generated directly from the data as opposed to obtaining it from an online database. Details about this procedure are described <a href="https://gatk.broadinstitute.org/hc/en-us/articles/360035890531-Base-Quality-Score-Recalibration-BQSR-">here</a>. <code>BaseRecalibrator</code> records data about the following features of aligned bases:</p>
<ul>
<li>Read group</li>
<li>Quality score</li>
<li>Machine cycle producing the base (Nth cycle = Nth base from the start of the read)</li>
<li>Dinucleotide context (current base + previous base)</li>
</ul>
<p>For each bin, the number of bases and how often such bases mistmatch the reference are counted and exported as a recalibration report.</p>
<p><strong>ApplyBQSR</strong></p>
<p><code>ApplyBQSR</code> uses the recalibration file to go through the reads and adjust each base’s score depending on what bins it falls in. The new quality score is effectively determined by:</p>
<ul>
<li>The sum of the global difference between reported quality scores and the empirical quality</li>
<li>plus the quality bin specific shift</li>
<li>plus the cycle x quality and dinucleotide x quality effect</li>
</ul>
<p>Recalibration should cause the read quality scores to match their empirical scores much closer than before. This allows for more statistically robust downstream variant calling.</p>
<pre class="r"><code>files &lt;- list.files(
  &quot;/hpcfs/users/a1647910/210216_sorl1_snv/05_addRG/samstats&quot;,
  full.names = TRUE
)
beforeRecalMetrics &lt;- lapply(files, read_lines) %&gt;%
  set_names(str_remove(basename(files), &quot;.tsv&quot;))
beforeRecalSummary &lt;- map2(beforeRecalMetrics, files, function(.x, .y){
  grep(&quot;^SN&quot;, .x, value = TRUE) %&gt;%
    str_split(&quot;\t&quot;, simplify = TRUE) %&gt;%
    .[, 2:3] %&gt;%
    set_colnames(c(&quot;metric&quot;, &quot;value&quot;)) %&gt;%
    as_tibble() %&gt;%
    mutate(metric = str_remove(metric, &quot;:$&quot;)) %&gt;%
    pivot_wider(names_from = &quot;metric&quot;, values_from = &quot;value&quot;) %&gt;%
    mutate(
      across(everything(), as.numeric),
      sample = str_remove(basename(.y), &quot;.tsv&quot;)
    )
}) %&gt;%
  bind_rows()
beforeRecalQuality &lt;- map2(beforeRecalMetrics, files, function(.x, .y){
  grep(&quot;^FFQ&quot;, .x, value = TRUE) %&gt;%
    str_split(&quot;\t&quot;, simplify = TRUE) %&gt;%
    .[,-1] %&gt;%
    set_colnames(c(&quot;cycle&quot;, paste0(&quot;BQ&quot;, seq(0, ncol(.) - 2)))) %&gt;%
    as_tibble() %&gt;%
    mutate(across(everything(), as.numeric)) %&gt;%
    .[, colSums(.) != 0] %&gt;%
    mutate(
      cycle = as.factor(cycle),
      sample = str_remove(basename(.y), &quot;.tsv&quot;)
    ) 
})</code></pre>
<pre class="r"><code>files &lt;- list.files(
  &quot;/hpcfs/users/a1647910/210216_sorl1_snv/07_recalBases/samstats&quot;,
  full.names = TRUE
)
afterRecalMetrics &lt;- lapply(files, read_lines) %&gt;%
  set_names(str_remove(basename(files), &quot;.tsv&quot;))
afterRecalSummary &lt;- map2(afterRecalMetrics, files, function(.x, .y){
  grep(&quot;^SN&quot;, .x, value = TRUE) %&gt;%
    str_split(&quot;\t&quot;, simplify = TRUE) %&gt;%
    .[, 2:3] %&gt;%
    set_colnames(c(&quot;metric&quot;, &quot;value&quot;)) %&gt;%
    as_tibble() %&gt;%
    mutate(metric = str_remove(metric, &quot;:$&quot;)) %&gt;%
    pivot_wider(names_from = &quot;metric&quot;, values_from = &quot;value&quot;) %&gt;%
    mutate(
      across(everything(), as.numeric),
      sample = str_remove(basename(.y), &quot;.tsv&quot;)
    )
}) %&gt;%
  bind_rows()
afterRecalQuality &lt;- map2(afterRecalMetrics, files, function(.x, .y){
  grep(&quot;^FFQ&quot;, .x, value = TRUE) %&gt;%
    str_split(&quot;\t&quot;, simplify = TRUE) %&gt;%
    .[,-1] %&gt;%
    set_colnames(c(&quot;cycle&quot;, paste0(&quot;BQ&quot;, seq(0, ncol(.) - 2)))) %&gt;%
    as_tibble() %&gt;%
    mutate(across(everything(), as.numeric)) %&gt;%
    .[, colSums(.) != 0] %&gt;%
    mutate(
      cycle = as.factor(cycle),
      sample = str_remove(basename(.y), &quot;.tsv&quot;)
    ) 
})</code></pre>
<p>For successful recalibration it is critical that each read group (sample in this case) has more than 100 million aligned bases. For this dataset the number of aligned bases ranged between <em>451,641,994</em> and <em>961,215,281</em> across all samples, and therefore base recalibration was performed.</p>
<pre class="r"><code>files &lt;- list.files(
  &quot;/hpcfs/users/a1647910/210216_sorl1_snv/07_recalBases/recal&quot;,
  pattern = &quot;analyzeCovariates&quot;,
  full.names = TRUE
)
recalCovariates &lt;- lapply(files, function(x){
  read_csv(x) %&gt;%
    mutate(sample = str_remove(basename(x), &quot;.analyzeCovariates.csv&quot;))
})</code></pre>
<p>To assess whether base recalibration was successful, base qualities were assessed before and after recalibration. As all samples showed similar profiles, a representative sample for each genotype is shown below.</p>
<pre class="r"><code>lapply(recalCovariates[c(1, 9, 15, 19)], function(x){
  x %&gt;%
    dplyr::filter(CovariateName == &quot;QualityScore&quot;) %&gt;%
    left_join(metadata[, c(&quot;sample&quot;, &quot;genotype&quot;)]) %&gt;%
    mutate(facet = paste(sample, &quot;-&quot;, genotype))
}) %&gt;%
  bind_rows() %&gt;%
  ggplot(aes(AverageReportedQuality, EmpiricalQuality)) +
  geom_point(aes(colour = Recalibration), alpha = 0.7) +
  geom_abline(intercept = 0, slope = 1, linetype = 2) + 
  xlab(&quot;Reported Quality Score&quot;) +
  ylab(&quot;Empirical Quality Score&quot;) +
  scale_colour_manual(values = c(&quot;Before&quot; = &quot;maroon1&quot;, &quot;After&quot; = &quot;blue&quot;)) +
  facet_wrap(~facet, nrow = 1) +
  theme(
    legend.position = &quot;bottom&quot;,
    strip.text.x = element_text(size = 7)
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="ase_qc_files/figure-html/unnamed-chunk-13-1.png" alt="*Reported base quality plotted against empirical quality. Base qualities before recalibration showed patterns of over-estimation by the sequencer. Post-recalibration quality scores fitted the empirically derived quality scores better than pre-recalibration. Additionally a greater distribution of quality scores was observed after recalibration procedures.*" width="768" />
<p class="caption">
<em>Reported base quality plotted against empirical quality. Base qualities before recalibration showed patterns of over-estimation by the sequencer. Post-recalibration quality scores fitted the empirically derived quality scores better than pre-recalibration. Additionally a greater distribution of quality scores was observed after recalibration procedures.</em>
</p>
</div>
<pre class="r"><code>lapply(recalCovariates[c(1, 9, 15, 19)], function(x){
  x %&gt;%
    dplyr::filter(CovariateName == &quot;Cycle&quot;) %&gt;%
    left_join(metadata[, c(&quot;sample&quot;, &quot;genotype&quot;)]) %&gt;%
    mutate(
      CovariateValue = factor(CovariateValue, levels = unique(CovariateValue)),
      facet = paste(sample, &quot;-&quot;, genotype)
    )
}) %&gt;%
  bind_rows() %&gt;%
  ggplot(aes(CovariateValue, Accuracy)) +
  geom_point(aes(colour = Recalibration), alpha = 0.7)+
  geom_abline(intercept = 0, slope = 0, linetype = 2) + 
  xlab(&quot;Machine Cycle&quot;) +
  ylab(&quot;Quality Score Accuracy (Empirical - Reported)&quot;) +
  scale_colour_manual(values = c(&quot;Before&quot; = &quot;maroon1&quot;, &quot;After&quot; = &quot;blue&quot;)) +
  scale_x_discrete(breaks = seq(0, 100, 5)) +
  coord_cartesian(ylim = c(-10, 10)) +
  facet_wrap(~facet, nrow = 1) +
  theme(
    legend.position = &quot;bottom&quot;,
    strip.text.x = element_text(size = 7),
    axis.text.x = element_text(size = 6, angle = 270, hjust = 0)
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="ase_qc_files/figure-html/unnamed-chunk-14-1.png" alt="*Residual error by machine cycle. Observations before recalibration are coloured pink and after recalibration are blue. Most cycles showed an overestimation of base qualities which was improved after recalibration.*" width="768" />
<p class="caption">
<em>Residual error by machine cycle. Observations before recalibration are coloured pink and after recalibration are blue. Most cycles showed an overestimation of base qualities which was improved after recalibration.</em>
</p>
</div>
<pre class="r"><code>lapply(recalCovariates[c(1, 9, 15, 19)], function(x){
  x %&gt;%
    dplyr::filter(CovariateName == &quot;Context&quot;) %&gt;%
    left_join(metadata[, c(&quot;sample&quot;, &quot;genotype&quot;)]) %&gt;%
    mutate(
      CovariateValue = factor(CovariateValue, levels = unique(CovariateValue)),
      facet = paste(sample, &quot;-&quot;, genotype)
    )
}) %&gt;%
  bind_rows() %&gt;%
  ggplot(aes(CovariateValue, Accuracy, colour = Recalibration)) +
  geom_point(alpha = 0.7, shape = &quot;\U2012&quot;, size = 5)+
  geom_abline(intercept = 0, slope = 0, linetype = 2) + 
  xlab(&quot;Dinucleotide&quot;) +
  ylab(&quot;Quality Score Accuracy (Empirical - Reported)&quot;) +
  scale_colour_manual(values = c(&quot;Before&quot; = &quot;maroon1&quot;, &quot;After&quot; = &quot;blue&quot;)) +
  coord_cartesian(ylim = c(-10, 10)) +
  facet_wrap(~facet, nrow = 1) +
  theme(
    legend.position = &quot;bottom&quot;,
    strip.text.x = element_text(size = 7),
    axis.text.x = element_text(size = 6, angle = 270, hjust = 0)
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="ase_qc_files/figure-html/unnamed-chunk-15-1.png" alt="*Residual error by dinucleotide. Observations before recalibration are coloured pink and after recalibration are blue. Dinucleotide contexts showed over-estimation of base qualities pre-recalibration but were substantially improved after recalibration.*" width="768" />
<p class="caption">
<em>Residual error by dinucleotide. Observations before recalibration are coloured pink and after recalibration are blue. Dinucleotide contexts showed over-estimation of base qualities pre-recalibration but were substantially improved after recalibration.</em>
</p>
</div>
</div>
<div id="genomic-location" class="section level2">
<h2>Genomic location</h2>
<p>Variants were restricted to those that lie only in exonic regions of the Ensembl version 101 primary assembly. For zebrafish this consists of chromosomes 1 to 25. To perform this inside of <code>GATK</code>’s variant calling <code>HaploypeCaller</code> command, an interval list was created containing the exonic ranges. A number of interval list formats are supported by GATK as described <a href="https://gatk.broadinstitute.org/hc/en-us/articles/360035531852-Intervals-and-interval-lists">here</a>. The GATK-style <code>.intervals</code> is poorly described but requires the format <code>&lt;chr&gt;:&lt;start&gt;-&lt;end&gt;</code> e.g. <code>1:1367-1367</code> for a singular base position. The nomenclature must match that of the chosen reference, for example, Ensembl labels chromosome 1 as “1”, not “chr1”. The GATK-style format was chosen for its simplicity.</p>
<p>Defining intervals before calling variants also has the advantage of a large speed-up in computational processing time. <code>HaplotypeCaller</code> is the longest step in the <code>GATK</code> short variant discovery pipeline, sometimes taking over a day to process for large datasets on a HPC system when interval lists are not utilised.</p>
<pre class="r"><code>exonRanges &lt;- exons %&gt;%
  unlist() %&gt;%
  GenomicRanges::reduce() %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(seqnames %in% drChrs) %&gt;%
  dplyr::select(chromosome = seqnames, start, end)</code></pre>
<pre class="r"><code>intervalPath &lt;- &quot;/hpcfs/users/a1647910/210216_sorl1_snv/refs/exons.intervals&quot;
makeIntervals &lt;- !file.exists(intervalPath)
if (makeIntervals) {
  if (!dir.exists(dirname(intervalPath))) {
    dir.create(dirname(intervalPath), recursive = TRUE)
  }
  tibble(
    interval = paste0(
      exonRanges$chromosome,
      &quot;:&quot;,
      exonRanges$start,
      &quot;-&quot;,
      exonRanges$end
    )
  ) %&gt;%
    write.table(
      file = intervalPath,
      col.names = FALSE,
      row.names = FALSE,
      quote = FALSE
    )
}</code></pre>
<p>The intervals were also exported as a <code>BED</code> file as some software prefer this format.</p>
<pre class="r"><code>bedPath &lt;- &quot;/hpcfs/users/a1647910/210216_sorl1_snv/refs/exons.bed&quot;
makeBed &lt;- !file.exists(bedPath)
if (makeBed) {
  if (!dir.exists(dirname(bedPath))) {
    dir.create(dirname(bedPath), recursive = TRUE)
  }
  tibble(
    seqnames = exonRanges$chromosome,
    start = exonRanges$start,
    end = exonRanges$end
  ) %&gt;%
    write.table(
      file = bedPath,
      quote = FALSE,
      sep = &quot;\t&quot;,
      col.names = FALSE,
      row.names = FALSE
    )
}</code></pre>
</div>
<div id="variant-calling" class="section level2">
<h2>Variant calling</h2>
<p>With the to perform downstream ASE analysis, initial variants were restricted to only biallelic SNVs. Additionally, for RNA-seq data, <code>GATK</code> recommends <a href="https://gatk.broadinstitute.org/hc/en-us/articles/360035890471-Hard-filtering-germline-short-variants">specific hard filters</a>. Variants were removed with the following filters:</p>
<ul>
<li>QualByDepth: QD &lt; 2.0
<ul>
<li>Variant confidence (QUAL field) divided by the unfiltered depth of non-homozygous reference samples. Removing variants that have a low QD normalises the variant quality and avoids inflation caused by deep coverage.</li>
</ul></li>
<li>FisherStrand: FS &gt; 60.0
<ul>
<li>The Phred-scaled probability there there is strand bias at the site. Strand bias indicates whether the alternate allele was seen more or less often on the forward or reverse strand than the reference allele. When there is little to no strand bias at the site, the FS value is close to 0.</li>
</ul></li>
<li>StrandOddsRatio: SOR &gt; 4.0
<ul>
<li>An alternative way to estimate strand bias using a test similar to the symmetric odds ratio test.</li>
</ul></li>
<li>RMSMappingQuality: MQ &lt; 40.0
<ul>
<li>Root mean square mapping quality over all the reads at the site.</li>
</ul></li>
<li>MappingQualityRankSumTest: MQRankSum &lt; -12.5
<ul>
<li>u-based z-approximation from the Rank Sum Test for mapping qualities. This compares the mapping qualities of the reads supporting the reference allele and the alternate allele.</li>
</ul></li>
<li>ReadPosRankSumTest: ReadPosRankSum &lt; -8.0
<ul>
<li>u-based z-approximation from the Rank Sum Test for site position within reads. This compares whether the positions of the reference and alternate alleles are different within the reads.</li>
</ul></li>
</ul>
<pre class="r"><code>calledMetrics &lt;- list.files(
  # path = here(&quot;08_callSnvs/1_called/log/&quot;),
  path = &quot;/hpcfs/users/a1647910/210216_sorl1_snv/08_callSnvs/1_called/log/&quot;,
  pattern = &quot;detail&quot;,
  full.names = TRUE
) %&gt;%
  lapply(read_tsv, comment = &quot;#&quot;) %&gt;%
  bind_rows() %&gt;%
  dplyr::select(-contains(c(&quot;DBSNP&quot;, &quot;DB_SNP&quot;, &quot;NOVEL&quot;))) %&gt;%
  as.data.frame()
extractedMetrics &lt;- list.files(
  # path = here(&quot;08_callSnvs/2_extracted/log/&quot;),
  path = &quot;/hpcfs/users/a1647910/210216_sorl1_snv/08_callSnvs/2_extracted/log/&quot;,
  pattern = &quot;detail&quot;,
  full.names = TRUE
) %&gt;%
  lapply(read_tsv, comment = &quot;#&quot;) %&gt;%
  bind_rows() %&gt;%
  dplyr::select(-contains(c(&quot;DBSNP&quot;, &quot;DB_SNP&quot;, &quot;NOVEL&quot;))) %&gt;%
  as.data.frame()
filteredMetrics &lt;- list.files(
  # path = here(&quot;08_callSnvs/3_filtered/log/&quot;),
  path = &quot;/hpcfs/users/a1647910/210216_sorl1_snv/08_callSnvs/3_filtered/log/&quot;,
  pattern = &quot;detail&quot;,
  full.names = TRUE
) %&gt;%
  lapply(read_tsv, comment = &quot;#&quot;) %&gt;%
  bind_rows() %&gt;%
  dplyr::select(-contains(c(&quot;DBSNP&quot;, &quot;DB_SNP&quot;, &quot;NOVEL&quot;))) %&gt;%
  as.data.frame()
selectedMetrics &lt;- list.files(
  # path = here(&quot;08_callSnvs/4_selected/log/&quot;),
  path = &quot;/hpcfs/users/a1647910/210216_sorl1_snv/08_callSnvs/4_selected/log/&quot;,
  pattern = &quot;detail&quot;,
  full.names = TRUE
) %&gt;%
  lapply(read_tsv, comment = &quot;#&quot;) %&gt;%
  bind_rows() %&gt;%
  dplyr::select(-contains(c(&quot;DBSNP&quot;, &quot;DB_SNP&quot;, &quot;NOVEL&quot;))) %&gt;%
  as.data.frame()</code></pre>
<pre class="r"><code>filtProgress &lt;- calledMetrics %&gt;%
  as_tibble() %&gt;%
  dplyr::select(sample = SAMPLE_ALIAS, initial = TOTAL_SNPS)
filtProgress &lt;- selectedMetrics %&gt;%
  as_tibble() %&gt;%
  dplyr::select(sample = SAMPLE_ALIAS, GATK = TOTAL_SNPS) %&gt;%
  left_join(filtProgress)</code></pre>
<p>After <code>GATK</code> recommended hard filters were applied thhe number of biallelic SNVs ready for ASE analysis ranged between <em>62,222</em> and <em>119,782</em>.</p>
</div>
</div>
<div id="allele-specific-expression" class="section level1">
<h1>Allele Specific Expression</h1>
<p>There are a number of important quality control measures that must be taken to ensure reliable data for allele specific expression (ASE) testing. The following procedures are based around the guidelines described in <a href="https://www.biorxiv.org/content/biorxiv/early/2015/03/05/016097.full.pdf">Castel et al. Tools and Best Practices for allelic expression analysis.</a></p>
<p>To gather SNV information as determined by the <code>GATK</code> workflow, <code>VCF</code> files were converted into <code>GDS</code> objects. <code>GDS</code> objects allow for easy access of the data in <code>R</code>, but are large so were kept on the HPC system.</p>
<pre class="r"><code>vcfPaths &lt;- list.files(
  &quot;/hpcfs/users/a1647910/210216_sorl1_snv/08_callSnvs/4_selected&quot;,
  pattern = &quot;.vcf.gz$&quot;,
  full.names = TRUE
)
lapply(vcfPaths, function(vcf){
  gdsPath &lt;- paste0(
    str_remove(dirname(vcf), &quot;4_selected&quot;),
    &quot;5_gds/&quot;,
    str_remove(basename(vcf), &quot;.vcf.gz&quot;),
    &quot;.gds&quot;
  )
  if (!file.exists(gdsPath)) {
    if (!dir.exists(dirname(gdsPath))) {
      dir.create(dirname(gdsPath), recursive = TRUE)
    }
    seqVCF2GDS(vcf, gdsPath)
  }
})</code></pre>
<pre class="r"><code>gdsPaths &lt;- list.files(
  &quot;/hpcfs/users/a1647910/210216_sorl1_snv/08_callSnvs/5_gds&quot;,
  pattern = &quot;.gds$&quot;,
  full.names = TRUE
)</code></pre>
<pre class="r"><code>snvList &lt;- lapply(seq_along(gdsPaths), function(x){
  gdsPath &lt;- gdsPaths[x]
  sample &lt;- basename(gdsPath) %&gt;%
    str_remove(&quot;.gds&quot;)
  gds &lt;- seqOpen(gdsPath, readonly = FALSE)
  snvs &lt;- tibble(
    variant.id = seqGetData(gds, &quot;variant.id&quot;),
    chromosome = seqGetData(gds, &quot;chromosome&quot;),
    position = seqGetData(gds, &quot;position&quot;),
    allele = seqGetData(gds, &quot;allele&quot;),
    filter = seqGetData(gds, &quot;annotation/filter&quot;)
  ) %&gt;%
    mutate(
      sample = sample,
      refAllele = str_split(allele, &quot;,&quot;, simplify = TRUE)[,1],
      altAllele = str_split(allele, &quot;,&quot;, simplify = TRUE)[,2]
    )
  seqClose(gds)
  return(snvs)
})</code></pre>
<div id="wasp" class="section level3">
<h3>WASP</h3>
<p>An important aspect to consider in ASE analysis is the potential for read mapping bias. For RNA-seq data mapped to a reference genome, reads that carry an alternate allele at positions of variation have at least one mismatch, and therefore a lower probability of aligning correctly than reads containing the reference allele.</p>
<p><code>WASP</code> provides a suite of tools for unbiased allele-specific read mapping. The <code>WASP</code> workflow for mappability filtering is defined by 5 steps:</p>
<ol style="list-style-type: decimal">
<li>Reads are mapped normally with the user’s mapper of choice (<code>STAR</code> was chosen in this case).</li>
<li>Reads that overlap identified SNVs and therefore may have mapping bias are determined using <code>find_intersecting_snps.py</code> script. Each overlapping read is output as a set of synthetic reads in <code>FASTQ</code> format with its allele swapped to to the other allele at the SNV site.</li>
<li>The set of allele-swapped reads are re-mapped using the same method and options used in step 1.</li>
<li>Reads where one or more allelic versions fail to map back to the same location are removed using <code>filter_remapped_reads.py</code> script.</li>
<li>Reads that did not overlap SNVs are merged with those that show unbiased mapping from step 4 using <code>samtools</code>, resulting in a complete set of mappability-filtered aligned reads in <code>BAM</code> format.</li>
</ol>
<p>To begin the <code>WASP</code> fltering procedure, input files containing genetic variants (SNVs) previously identified by the <code>GATK</code> workflow are needed. Since we do not have phasing information, it is recommended to use text-based format opposed to <code>HDF5</code> for the input files. The text-based input files require three space-delimited columns (position, ref_allele, alt_allele), and one input file per chromosome. The filenames must follow the convention <code>&lt;chr&gt;.snps.txt.gz</code> (e.g <code>1.snps.txt.gz</code> for chromosome 1).</p>
<pre class="r"><code>lapply(snvList, function(x){
  chrList &lt;- x %&gt;%
    dplyr::select(chromosome, position, refAllele, altAllele, sample) %&gt;%
    split(.[,&#39;chromosome&#39;])
  lapply(chrList, function(x){
    chr &lt;- unique(x$chromosome)
    sample &lt;- unique(x$sample)
    path &lt;- paste0(
      &quot;/hpcfs/users/a1647910/210216_sorl1_snv/09_wasp/1_snvs/&quot;,
      sample,
      &quot;/&quot;,
      chr,
      &quot;.snps.txt.gz&quot;
    )
    if (!file.exists(path)) {
      if (!dir.exists(dirname(path))) {
        dir.create(dirname(path), recursive = TRUE)
      }
      tibble(
        position = x$position,
        refAllele = x$refAllele,
        altAllele = x$altAllele
      ) %&gt;%
        write_delim(file = path, delim = &quot; &quot;, col_names = FALSE)
    }
  })
})</code></pre>
<p>Text-based input files were created for <code>WASP</code> filtering, which was subsequently performed on the HPC system. The resulting <code>BAM</code> files were then subject to allele specific expression read counting with <code>ASEReadCounter</code>.</p>
</div>
<div id="asereadcounter" class="section level3">
<h3>ASEReadCounter</h3>
<p><code>GATK</code> tool <code>ASEReadCounter</code> calculates allele counts at a set of positions after applying filters specifically tuned for ASE analysis of RNA-seq data. The filters operate on mapping quality, base quality, depth of coverage and overlapping paired reads. Each of these filters can be controlled by command-line arguments. The data in this analysis was generated with the following options:</p>
<ul>
<li><code>--min-mapping-quality 10</code></li>
<li><code>--min-base-quality 20</code></li>
<li><code>--count-overlap-reads-handling COUNT_FRAGMENTS_REQUIRE_SAME_BASE</code></li>
<li>Ths option counts all fragments where the base is consistent when mate pairs overlap.</li>
<li><code>--min-depth-of-non-filtered-base -1</code></li>
<li>This argument is disabled, because depth filtering was performed manually (see next section: Coverage).</li>
</ul>
<p><code>ASEReadCounter</code> was run on <code>BAM</code> alignment files that were subject to <code>WASP</code> filtering and also alignment files that had not been <code>WASP</code> filtered, to compare whether the filtering procedure improves any observable reference allele mapping bias. Heterozygous SNVs detected by <code>ASEReadCounter</code> were recorded in separate <code>tsv</code> files for each sample. These were loaded into R as a list of tibbles containing allele count data for ASE analysis.</p>
<pre class="r"><code>files_nowasp &lt;- list.files(
  &quot;/hpcfs/users/a1647910/210216_sorl1_snv/10_aseReadCounter/nowasp&quot;,
  full.names = TRUE
)
samples &lt;- basename(files_nowasp) %&gt;%
  str_remove(&quot;.tsv&quot;)
aseRC_nowasp &lt;- lapply(files_nowasp, function(file){
  sample &lt;- basename(file) %&gt;%
    str_remove(&quot;.tsv&quot;)
  read_tsv(
    file,
    col_types = &quot;cdcccdddddddd&quot;
  ) %&gt;%
    mutate(
      sample = sample,
      allele = paste0(refAllele, &quot;,&quot;, altAllele)
    ) %&gt;%
    left_join(metadata[,c(&quot;sample&quot;, &quot;genotype&quot;)]) %&gt;%
    dplyr::select(
      chromosome = contig, position, allele, refCount, altCount, totalCount,
      sample, genotype, lowMAPQDepth, lowBaseQDepth, rawDepth, otherBases,
      improperPairs, refAllele, altAllele
    )
}) %&gt;% 
  set_names(samples)</code></pre>
<pre class="r"><code>files_wasp &lt;- list.files(
  &quot;/hpcfs/users/a1647910/210216_sorl1_snv/10_aseReadCounter/wasp&quot;,
  full.names = TRUE
)
aseRC_wasp &lt;- lapply(files_wasp, function(file){
  sample &lt;- basename(file) %&gt;%
    str_remove(&quot;.tsv&quot;)
  read_tsv(
    file,
    col_types = &quot;cdcccdddddddd&quot;
  ) %&gt;%
    mutate(
      sample = sample,
      allele = paste0(refAllele, &quot;,&quot;, altAllele)
    ) %&gt;%
    left_join(metadata[,c(&quot;sample&quot;, &quot;genotype&quot;)]) %&gt;%
    dplyr::select(
      chromosome = contig, position, allele, refCount, altCount, totalCount,
      sample, genotype, lowMAPQDepth, lowBaseQDepth, rawDepth, otherBases,
      improperPairs, refAllele, altAllele
    )
}) %&gt;% 
  set_names(samples)</code></pre>
<pre class="r"><code>filtProgress &lt;- list(
  nowasp = aseRC_nowasp %&gt;%
    bind_rows() %&gt;%
    group_by(sample) %&gt;%
    summarise(
      aseRC = n()
    ) %&gt;%
    left_join(filtProgress),
  wasp = aseRC_wasp %&gt;%
    bind_rows() %&gt;%
    group_by(sample) %&gt;%
    summarise(
      software = n()
    ) %&gt;%
    left_join(filtProgress)
)</code></pre>
</div>
<div id="coverage" class="section level3">
<h3>Coverage</h3>
<p>SNV sites with low read coverage are uninformative, and therefore were subject to filtering. To determine the optimal read coverage value for filtering, the effects of potential cut-offs were inspected.</p>
<pre class="r"><code>covBins_nowasp &lt;- lapply(aseRC_nowasp, function(x){
  n0 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 0))
  n10 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 10))
  n20 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 20))
  n30 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 30))
  n40 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 40))
  n50 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 50))
  sample &lt;- unique(x$sample)
  tibble(
    bin = c(&quot;\u2265 0&quot;, &quot;\u2265 10&quot;, &quot;\u2265 20&quot;, &quot;\u2265 30&quot;, &quot;\u2265 40&quot;, &quot;\u2265 50&quot;),
    n = c(n0, n10, n20, n30, n40, n50),
    sample = sample,
    wasp = &quot;Without WASP filtering&quot;
  )
}) %&gt;%
  bind_rows()
covBins_wasp &lt;- lapply(aseRC_wasp, function(x){
  n0 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 0))
  n10 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 10))
  n20 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 20))
  n30 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 30))
  n40 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 40))
  n50 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 50))
  sample &lt;- unique(x$sample)
  tibble(
    bin = c(&quot;\u2265 0&quot;, &quot;\u2265 10&quot;, &quot;\u2265 20&quot;, &quot;\u2265 30&quot;, &quot;\u2265 40&quot;, &quot;\u2265 50&quot;),
    n = c(n0, n10, n20, n30, n40, n50),
    sample = sample,
    wasp = &quot;With WASP filtering&quot;
  )
}) %&gt;%
  bind_rows()</code></pre>
<pre class="r"><code>bind_rows(covBins_nowasp, covBins_wasp) %&gt;%
  mutate(wasp = factor(wasp, levels = c(
    &quot;Without WASP filtering&quot;, &quot;With WASP filtering&quot;
  ))) %&gt;%
  ggplot(aes(bin, n, fill = bin)) +
  geom_boxplot() +
  labs(x = &quot;Reads / SNV&quot;, y = &quot;Number of SNVs&quot;) +
  theme(legend.position = &quot;none&quot;) +
  facet_wrap(~wasp)</code></pre>
<div class="figure" style="text-align: center">
<img src="ase_qc_files/figure-html/unnamed-chunk-30-1.png" alt="*The number of SNVs per sample that satisfy read coverage cut-offs*" width="768" />
<p class="caption">
<em>The number of SNVs per sample that satisfy read coverage cut-offs</em>
</p>
</div>
<p>A cut-off of at least 10 counts per SNV position was applied, as it provides a reasonable estimate of ASE while keeping as many SNVs as possible.</p>
<pre class="r"><code>aseRC_wasp &lt;- lapply(aseRC_wasp, function(x){
  dplyr::filter(x, totalCount &gt;= 10)
})
aseRC_nowasp &lt;- lapply(aseRC_nowasp, function(x){
  dplyr::filter(x, totalCount &gt;= 10)
})</code></pre>
<pre class="r"><code>filtProgress &lt;- list(
  nowasp = aseRC_nowasp %&gt;%
    bind_rows() %&gt;%
    group_by(sample) %&gt;%
    summarise(
      coverage = n()
    ) %&gt;%
    left_join(filtProgress$nowasp),
  wasp = aseRC_wasp %&gt;%
    bind_rows() %&gt;%
    group_by(sample) %&gt;%
    summarise(
      coverage = n()
    ) %&gt;%
    left_join(filtProgress$wasp)
)</code></pre>
</div>
<div id="mono-allelic-expression" class="section level3">
<h3>Mono-allelic expression</h3>
<p>Producing evidence of mono-allelic expression from short read RNA-seq datasets without parental genotypes or imprinting information is a controversial issue. SNV sites were therefore filtered based on a depth criteria for each allele (<span class="math inline">\(\ge\)</span> 3 counts or <span class="math inline">\(&gt;\)</span> 1% of the total counts for that allele).</p>
<pre class="r"><code>aseRC_wasp &lt;- lapply(aseRC_wasp, function(x){
  dplyr::filter(
    x,
    refCount &gt;= 3,
    altCount &gt;= 3,
    refCount / totalCount &gt; 0.01,
    altCount / totalCount &gt; 0.01
  )
})
aseRC_nowasp &lt;- lapply(aseRC_nowasp, function(x){
  dplyr::filter(
    x,
    refCount &gt;= 3,
    altCount &gt;= 3,
    refCount / totalCount &gt; 0.01,
    altCount / totalCount &gt; 0.01
  )
})</code></pre>
<pre class="r"><code>filtProgress &lt;- list(
  nowasp = aseRC_nowasp %&gt;%
    bind_rows() %&gt;%
    group_by(sample) %&gt;%
    summarise(
      monoallelic = n()
    ) %&gt;%
    left_join(filtProgress$nowasp),
  wasp = aseRC_wasp %&gt;%
    bind_rows() %&gt;%
    group_by(sample) %&gt;%
    summarise(
      monoallelic = n()
    ) %&gt;%
    left_join(filtProgress$wasp)
)</code></pre>
</div>
</div>
<div id="summary" class="section level1">
<h1>Summary</h1>
<div id="reference-mapping-bias" class="section level3">
<h3>Reference mapping bias</h3>
<p><code>WASP</code> filtering is designed to remove the vast majority of reads with mapping bias. Reference allele ratios were calculated for all SNVs passing filtering procedures with and without <code>WASP</code> filtering, to determine the effectiveness of <code>WASP</code>. A residual mapping bias is indicated if reference allele ratios tend to deviate from 0.5.</p>
<pre class="r"><code>refRatios_nowasp &lt;- lapply(aseRC_nowasp, function(x){
  x %&gt;%
    dplyr::filter(totalCount != 0) %&gt;%
    mutate(
      refRatio = refCount / totalCount,
      altRatio = altCount / totalCount,
      WASP = FALSE
    ) %&gt;%
    dplyr::select(
      chromosome, position, allele, refRatio, altRatio, totalCount,
      sample, genotype, WASP
    )
}) %&gt;%
  bind_rows()</code></pre>
<pre class="r"><code>refRatios_wasp &lt;- lapply(aseRC_wasp, function(x){
  x %&gt;%
    dplyr::filter(totalCount != 0) %&gt;%
    mutate(
      refRatio = refCount / totalCount,
      altRatio = altCount / totalCount,
      WASP = TRUE
    ) %&gt;%
    dplyr::select(
      chromosome, position, allele, refRatio, altRatio, totalCount,
      sample, genotype, WASP
    )
}) %&gt;%
  bind_rows()</code></pre>
<p>Distributions of the reference allele ratio for all SNVs were visualised to determine if any mapping bias was present. Distributions showing balanced profile without 0 or 1 inflation are representative of no mapping bias.</p>
<pre class="r"><code>refRatios_nowasp %&gt;%
  left_join(metadata[,c(&quot;sample&quot;, &quot;genotype&quot;)]) %&gt;%
  dplyr::arrange(genotype, sample) %&gt;%
  ggplot(aes(refRatio, group = sample, colour = genotype)) +
  geom_density() +
  geom_vline(xintercept = 0.5, linetype = &quot;dashed&quot;) +
  labs(x = &quot;Reference allele ratio&quot;, y = &quot;Density&quot;, colour = &quot;Genotype&quot;) +
  theme(legend.position = &quot;bottom&quot;) +
  scale_color_manual(values = genoCols) +
  ggtitle(&quot;Reference allele ratio distributions without WASP filtering&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="ase_qc_files/figure-html/unnamed-chunk-37-1.png" alt="*Sample distributions of global reference allelic ratios for remaining SNVs when no WASP filtering was applied. Distributions are coloured by genotype. All samples tend to follow a similar distribution with inflations towards 1, indicating that a reference allele mapping bias is present.*" width="768" />
<p class="caption">
<em>Sample distributions of global reference allelic ratios for remaining SNVs when no WASP filtering was applied. Distributions are coloured by genotype. All samples tend to follow a similar distribution with inflations towards 1, indicating that a reference allele mapping bias is present.</em>
</p>
</div>
<pre class="r"><code>refRatios_wasp %&gt;%
  left_join(metadata[,c(&quot;sample&quot;, &quot;genotype&quot;)]) %&gt;%
  dplyr::arrange(genotype, sample) %&gt;%
  ggplot(aes(refRatio, group = sample, colour = genotype)) +
  geom_density() +
  geom_vline(xintercept = 0.5, linetype = &quot;dashed&quot;) +
  labs(x = &quot;Reference allele ratio&quot;, y = &quot;Density&quot;, colour = &quot;Genotype&quot;) +
  theme(legend.position = &quot;bottom&quot;) +
  scale_color_manual(values = genoCols) +
  ggtitle(&quot;Reference allele ratio distributions with WASP filtering&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="ase_qc_files/figure-html/unnamed-chunk-38-1.png" alt="*Sample distributions of global reference allelic ratios for remaining SNVs when WASP filtering was incorporated into the quality control procedure. Distributions are coloured by genotype. All samples show a reasonably balanced profile, indicating that WASP filtering successfully improved reference mapping bias.*" width="768" />
<p class="caption">
<em>Sample distributions of global reference allelic ratios for remaining SNVs when WASP filtering was incorporated into the quality control procedure. Distributions are coloured by genotype. All samples show a reasonably balanced profile, indicating that WASP filtering successfully improved reference mapping bias.</em>
</p>
</div>
<p>As an alternative viewpoint, boxplots of reference allele ratios were plotted for each sample. Median/mean reference allele ratios that deviate substantially from 0.5 indicate a mapping bias.</p>
<pre class="r"><code>refRatios_nowasp %&gt;%
  left_join(metadata[,c(&quot;sample&quot;, &quot;genotype&quot;)]) %&gt;%
  dplyr::arrange(genotype, sample) %&gt;%
  mutate(sample = factor(sample, levels = unique(sample))) %&gt;%
  ggplot(aes(sample, refRatio, fill = genotype)) +
  geom_boxplot(fatten = 3, outlier.shape = NA) +
  geom_hline(yintercept = 0.5, colour = &quot;black&quot;) +
  stat_summary(
    fun = mean, geom = &quot;point&quot;, colour = &quot;white&quot;, shape = &quot;\U2012&quot;, size = 6
  ) +
  labs(x = &quot;Sample&quot;, y = &quot;Reference allele ratio&quot;) +
  scale_fill_manual(name = &quot;Genotype&quot;, values = genoCols) +
  scale_y_continuous(breaks = seq(0, 1, 0.1)) +
  theme(
    axis.text.x = element_text(angle = -90, vjust = 0.5),
    legend.position = &quot;bottom&quot;
  ) +
  ggtitle(label = &quot;Reference allele ratios without WASP filtering&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="ase_qc_files/figure-html/unnamed-chunk-39-1.png" alt="*Boxplots showing reference allele ratios for remaining SNVs when no WASP filtering was applied. The mean reference ratio for each sample is indicated with a white dash, while the solid black line indicates a reference ratio of 0.5. Outliers are hidden for the ease of viewing. As indicated by mean and median reference ratios above 0.5, reference mapping bias is evident.*" width="768" />
<p class="caption">
<em>Boxplots showing reference allele ratios for remaining SNVs when no WASP filtering was applied. The mean reference ratio for each sample is indicated with a white dash, while the solid black line indicates a reference ratio of 0.5. Outliers are hidden for the ease of viewing. As indicated by mean and median reference ratios above 0.5, reference mapping bias is evident.</em>
</p>
</div>
<pre class="r"><code>refRatios_wasp %&gt;%
  dplyr::filter(totalCount &gt;= 10) %&gt;%
  left_join(metadata[,c(&quot;sample&quot;, &quot;genotype&quot;)]) %&gt;%
  dplyr::arrange(genotype, sample) %&gt;%
  mutate(sample = factor(sample, levels = unique(sample))) %&gt;%
  ggplot(aes(sample, refRatio, fill = genotype)) +
  geom_boxplot(fatten = 3, outlier.shape = NA) +
  geom_hline(yintercept = 0.5, colour = &quot;black&quot;) +
  stat_summary(
    fun = mean, geom = &quot;point&quot;, colour = &quot;white&quot;, shape = &quot;\U2012&quot;, size = 6
  ) +
  labs(x = &quot;Sample&quot;, y = &quot;Reference allele ratio&quot;) +
  scale_fill_manual(name = &quot;Genotype&quot;, values = genoCols) +
  scale_y_continuous(breaks = seq(0, 1, 0.1)) +
  guides(colour = &quot;none&quot;) +
  theme(
    axis.text.x = element_text(angle = -90, vjust = 0.5),
    legend.position = &quot;bottom&quot;
  ) +
  ggtitle(label = &quot;Reference allele ratios with WASP filtering&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="ase_qc_files/figure-html/unnamed-chunk-40-1.png" alt="*Boxplots showing reference allele ratios for remaining SNVs when WASP filtering was incorporated into the quality control procedure. The mean reference ratio for each sample is indicated with a white dash, while the solid black line indicates a reference ratio of 0.5. Outliers are hidden for the ease of viewing. Reference mapping bias was eliminated in WASP filtered data and presents a significant improvement over data that was not subject to WASP.*" width="768" />
<p class="caption">
<em>Boxplots showing reference allele ratios for remaining SNVs when WASP filtering was incorporated into the quality control procedure. The mean reference ratio for each sample is indicated with a white dash, while the solid black line indicates a reference ratio of 0.5. Outliers are hidden for the ease of viewing. Reference mapping bias was eliminated in WASP filtered data and presents a significant improvement over data that was not subject to WASP.</em>
</p>
</div>
<p><code>WASP</code> filtering significantly improved the reference allele mapping bias and therefore <code>WASP</code> filtered data was chosen for ASE testing.</p>
<pre class="r"><code>aseRC &lt;- aseRC_wasp
refRatios &lt;- refRatios_wasp
filtProgress &lt;- filtProgress$wasp</code></pre>
<pre class="r"><code>refBias &lt;- refRatios %&gt;%
  bind_rows() %&gt;%
  group_by(sample) %&gt;%
  summarise(mean = mean(refRatio), median = median(refRatio))
altBias &lt;- refRatios %&gt;%
  bind_rows() %&gt;%
  group_by(sample) %&gt;%
  summarise(mean = mean(altRatio), median = median(altRatio))</code></pre>
<p>After quality control procedures, mean reference ratios ranged between 0.501 and 0.504. Median reference ratios ranged between 0.5 and 0.5. The mean reference ratios for each sample will be used as the null in statistical testing for ASE, as this can improve results <a href="https://www.biorxiv.org/content/biorxiv/early/2015/03/05/016097.full.pdf">(Castel et al.)</a>.</p>
</div>
<div id="filtering" class="section level3">
<h3>Filtering</h3>
<pre class="r"><code>filtProgress %&gt;%
  dplyr::select(
    sample, `Within exons` = initial, GATK = GATK,
    `WASP/ASEReadCounter` = software,
    Coverage = coverage, `Mono-allelic expression` = monoallelic
  ) %&gt;%
  pivot_longer(cols = -sample, names_to = &quot;Filter&quot;, values_to = &quot;SNVs&quot;) %&gt;%
  mutate(Filter = factor(Filter, levels = unique(.$Filter))) %&gt;%
  ggplot(aes(Filter, SNVs, group = Filter, fill = Filter)) +
  geom_boxplot() +
  labs(y = &quot;Number of SNVs&quot;) +
  scale_y_continuous(labels = comma) +
  theme(
    legend.position = &quot;none&quot;,
    axis.title.x = element_blank(),
    panel.grid.major.x = element_blank()
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="ase_qc_files/figure-html/unnamed-chunk-43-1.png" alt="*Boxplots of the total number of SNVs remaining in each sample after various quality control filtering steps. The filters were applied sequentially from left to right across the x-axis.*" width="768" />
<p class="caption">
<em>Boxplots of the total number of SNVs remaining in each sample after various quality control filtering steps. The filters were applied sequentially from left to right across the x-axis.</em>
</p>
</div>
</div>
<div id="coverage-1" class="section level3">
<h3>Coverage</h3>
<pre class="r"><code>covPlotA &lt;- lapply(aseRC, function(x){
  n10 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 10))
  n20 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 20))
  n30 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 30))
  n40 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 40))
  n50 &lt;- nrow(dplyr::filter(x, totalCount &gt;= 50))
  sample &lt;- unique(x$sample)
  tibble(
    bin = c(&quot;\u2265 10&quot;, &quot;\u2265 20&quot;, &quot;\u2265 30&quot;, &quot;\u2265 40&quot;, &quot;\u2265 50&quot;),
    n = c(n10, n20, n30, n40, n50),
    sample = sample
  )
}) %&gt;%
  bind_rows() %&gt;%
  ggplot(aes(bin, n, fill = bin)) +
  geom_boxplot() +
  labs(x = &quot;Reads / SNV&quot;, y = &quot;SNV count&quot;) +
  scale_y_continuous(
    breaks = seq(0, 100000, 10000),
    labels = comma
  ) +
  theme(
    legend.position = &quot;none&quot;,
    panel.grid.major.x = element_blank()
  )</code></pre>
<pre class="r"><code>exonOverlaps &lt;- lapply(aseRC, function(x){
  indices &lt;- x %&gt;%
    dplyr::select(seqnames = chromosome, start = position) %&gt;%
    mutate(end = start) %&gt;%
    makeGRangesFromDataFrame() %&gt;%
    findOverlaps(exons) %&gt;%
    as.data.frame()
  tibble(
    snvInd = indices$queryHits,
    geneInd = indices$subjectHits
  )
})
aseRCByGene &lt;- lapply(seq_along(aseRC), function(x){
  aseRC[[x]][exonOverlaps[[x]]$snvInd,] %&gt;%
    cbind(gene = names(exons[exonOverlaps[[x]]$geneInd])) %&gt;%
    as_tibble()
})</code></pre>
<pre class="r"><code>covPlotB &lt;- lapply(aseRCByGene, function(x){
  n10 &lt;- length(unique(pull(dplyr::filter(x, totalCount &gt;= 10), gene)))
  n20 &lt;- length(unique(pull(dplyr::filter(x, totalCount &gt;= 20), gene)))
  n30 &lt;- length(unique(pull(dplyr::filter(x, totalCount &gt;= 30), gene)))
  n40 &lt;- length(unique(pull(dplyr::filter(x, totalCount &gt;= 40), gene)))
  n50 &lt;- length(unique(pull(dplyr::filter(x, totalCount &gt;= 50), gene)))
  sample &lt;- unique(x$sample)
  tibble(
    bin = c(&quot;\u2265 10&quot;, &quot;\u2265 20&quot;, &quot;\u2265 30&quot;, &quot;\u2265 40&quot;, &quot;\u2265 50&quot;),
    n = c(n10, n20, n30, n40, n50),
    sample = sample
  )
}) %&gt;%
  bind_rows() %&gt;%
  ggplot(aes(bin, n, fill = bin)) +
  geom_boxplot() +
  labs(x = &quot;Reads / SNV&quot;, y = &quot;Gene count&quot;) +
  scale_y_continuous(
    breaks = seq(0, 10000, 1000),
    labels = comma
  ) +
  theme(
    legend.position = &quot;none&quot;,
    panel.grid.major.x = element_blank()
  )</code></pre>
<pre class="r"><code>covPlotC &lt;- lapply(aseRCByGene, function(x){
  x %&gt;%
    dplyr::select(chromosome, position, totalCount, sample, gene) %&gt;%
    mutate(
      n10 = totalCount &gt;= 10,
      n20 = totalCount &gt;= 20,
      n30 = totalCount &gt;= 30,
      n40 = totalCount &gt;= 40,
      n50 = totalCount &gt;= 50
    ) %&gt;%
    group_by(gene) %&gt;%
    summarise(
      n10 = sum(n10),
      n20 = sum(n20),
      n30 = sum(n30),
      n40 = sum(n40), 
      n50 = sum(n50)
    ) %&gt;%
    ungroup() %&gt;%
    pivot_longer(
      cols = c(&quot;n10&quot;, &quot;n20&quot;, &quot;n30&quot;, &quot;n40&quot;, &quot;n50&quot;),
      names_to = &quot;bin&quot;,
      values_to = &quot;snvs&quot;
    ) %&gt;%
    group_by(bin, snvs) %&gt;%
    summarise(genes = n()) %&gt;%
    ungroup() %&gt;%
    dplyr::filter(snvs != 0)
}) %&gt;%
  bind_rows() %&gt;%
  group_by(bin, snvs) %&gt;%
  summarise(medianGenes = median(genes)) %&gt;%
  ggplot(aes(snvs, medianGenes, colour = bin)) +
  geom_point() +
  geom_line() +
  scale_colour_discrete(
    labels = c(&quot;\u2265 10&quot;, &quot;\u2265 20&quot;, &quot;\u2265 30&quot;, &quot;\u2265 40&quot;, &quot;\u2265 50&quot;)
  ) +
  scale_x_continuous(minor_breaks = seq(0, 30, 1)) +
  scale_y_continuous(
    breaks = seq(0, 5000, 100),
    labels = comma
  ) +
  coord_cartesian(xlim = c(0, 25)) +
  labs(x = &quot;SNVs / gene&quot;, y = &quot;Median gene count&quot;, colour = &quot;Coverage&quot;) +
  theme(legend.position= c(0.75, 0.75))</code></pre>
<pre class="r"><code>ggarrange(covPlotA, covPlotB, covPlotC, nrow = 1, labels = &quot;AUTO&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="ase_qc_files/figure-html/unnamed-chunk-48-1.png" alt="*Genomic coverage of ASE data. **A**, **B** The number of SNVs (**A**) and protein coding genes (**B**) per sample at different read coverage cut-offs. **C** The number of protein coding genes against the number of SNVs they contain at different read coverage cut-offs. Each line represents the median value for all samples.*" width="768" />
<p class="caption">
<em>Genomic coverage of ASE data. <strong>A</strong>, <strong>B</strong> The number of SNVs (<strong>A</strong>) and protein coding genes (<strong>B</strong>) per sample at different read coverage cut-offs. <strong>C</strong> The number of protein coding genes against the number of SNVs they contain at different read coverage cut-offs. Each line represents the median value for all samples.</em>
</p>
</div>
</div>
<div id="export" class="section level3">
<h3>Export</h3>
<p>For static ASE testing, which describes the difference between two variants of a heterozygous allele in a single sample in an unchanging condition, <code>geneiASE</code> software was chosen. <code>geneiASE</code> requires allele counts <code>tsv</code> format with four columns: featureID, snpID, alternative allele count, reference allele count. <code>tsv</code> files (one for each sample) were exported for ASE testing.</p>
<pre class="r"><code>staticCounts &lt;- lapply(aseRCByGene, function(x){
  x %&gt;%
    rownames_to_column(&quot;snvID&quot;) %&gt;%
    left_join(altBias[,c(&quot;sample&quot;, &quot;mean&quot;)]) %&gt;%
    dplyr::select(
      gene, snvID, altCount, refCount, betabinom.p = mean
    )
}) %&gt;%
  set_names(samples)
lapply(seq_along(staticCounts), function(x){
  path &lt;- paste0(
    &quot;/hpcfs/users/a1647910/210216_sorl1_snv/11_geneiase/1_counts/&quot;,
    names(staticCounts[x]),
    &quot;.static.tsv&quot;
  )
  if (!file.exists(path)) {
    if (!dir.exists(dirname(path))) {
      dir.create(dirname(path), recursive = TRUE)
    }
    write_tsv(staticCounts[[x]], path)
  }
})</code></pre>
</div>
</div>
<div id="session-information" class="section level1">
<h1>Session information</h1>
<pre class="r"><code>sessionInfo() %&gt;%
  pander()</code></pre>
<p><strong>R version 4.1.1 (2021-08-10)</strong></p>
<p><strong>Platform:</strong> x86_64-pc-linux-gnu (64-bit)</p>
<p><strong>locale:</strong> <em>LC_CTYPE=en_AU.UTF-8</em>, <em>LC_NUMERIC=C</em>, <em>LC_TIME=en_AU.UTF-8</em>, <em>LC_COLLATE=en_AU.UTF-8</em>, <em>LC_MONETARY=en_AU.UTF-8</em>, <em>LC_MESSAGES=en_AU.UTF-8</em>, <em>LC_PAPER=en_AU.UTF-8</em>, <em>LC_NAME=C</em>, <em>LC_ADDRESS=C</em>, <em>LC_TELEPHONE=C</em>, <em>LC_MEASUREMENT=en_AU.UTF-8</em> and <em>LC_IDENTIFICATION=C</em></p>
<p><strong>attached base packages:</strong> <em>stats4</em>, <em>parallel</em>, <em>stats</em>, <em>graphics</em>, <em>grDevices</em>, <em>utils</em>, <em>datasets</em>, <em>methods</em> and <em>base</em></p>
<p><strong>other attached packages:</strong> <em>ensembldb(v.2.16.2)</em>, <em>AnnotationFilter(v.1.16.0)</em>, <em>GenomicFeatures(v.1.44.0)</em>, <em>AnnotationDbi(v.1.54.1)</em>, <em>Biobase(v.2.52.0)</em>, <em>GenomicRanges(v.1.44.0)</em>, <em>GenomeInfoDb(v.1.28.1)</em>, <em>IRanges(v.2.26.0)</em>, <em>S4Vectors(v.0.30.0)</em>, <em>ngsReports(v.1.8.1)</em>, <em>SeqArray(v.1.32.0)</em>, <em>gdsfmt(v.1.28.0)</em>, <em>UpSetR(v.1.4.0)</em>, <em>rmarkdown(v.2.9)</em>, <em>pander(v.0.6.4)</em>, <em>ggpubr(v.0.4.0)</em>, <em>RColorBrewer(v.1.1-2)</em>, <em>ggrepel(v.0.9.1)</em>, <em>tictoc(v.1.0.1)</em>, <em>kableExtra(v.1.3.4)</em>, <em>scales(v.1.1.1)</em>, <em>AnnotationHub(v.3.0.1)</em>, <em>BiocFileCache(v.2.0.0)</em>, <em>dbplyr(v.2.1.1)</em>, <em>BiocGenerics(v.0.38.0)</em>, <em>here(v.1.0.1)</em>, <em>future.apply(v.1.7.0)</em>, <em>future(v.1.21.0)</em>, <em>magrittr(v.2.0.1)</em>, <em>forcats(v.0.5.1)</em>, <em>stringr(v.1.4.0)</em>, <em>dplyr(v.1.0.7)</em>, <em>purrr(v.0.3.4)</em>, <em>readr(v.1.4.0)</em>, <em>tidyr(v.1.1.3)</em>, <em>tibble(v.3.1.2)</em>, <em>ggplot2(v.3.3.5)</em> and <em>tidyverse(v.1.3.1)</em></p>
<p><strong>loaded via a namespace (and not attached):</strong> <em>readxl(v.1.3.1)</em>, <em>backports(v.1.2.1)</em>, <em>systemfonts(v.1.0.2)</em>, <em>plyr(v.1.8.6)</em>, <em>lazyeval(v.0.2.2)</em>, <em>crosstalk(v.1.1.1)</em>, <em>BiocParallel(v.1.26.1)</em>, <em>listenv(v.0.8.0)</em>, <em>digest(v.0.6.27)</em>, <em>htmltools(v.0.5.1.1)</em>, <em>fansi(v.0.5.0)</em>, <em>memoise(v.2.0.0)</em>, <em>openxlsx(v.4.2.4)</em>, <em>globals(v.0.14.0)</em>, <em>Biostrings(v.2.60.1)</em>, <em>modelr(v.0.1.8)</em>, <em>matrixStats(v.0.59.0)</em>, <em>svglite(v.2.0.0)</em>, <em>prettyunits(v.1.1.1)</em>, <em>colorspace(v.2.0-2)</em>, <em>blob(v.1.2.1)</em>, <em>rvest(v.1.0.0)</em>, <em>rappdirs(v.0.3.3)</em>, <em>haven(v.2.4.1)</em>, <em>xfun(v.0.24)</em>, <em>crayon(v.1.4.1)</em>, <em>RCurl(v.1.98-1.3)</em>, <em>jsonlite(v.1.7.2)</em>, <em>zoo(v.1.8-9)</em>, <em>glue(v.1.4.2)</em>, <em>gtable(v.0.3.0)</em>, <em>zlibbioc(v.1.38.0)</em>, <em>XVector(v.0.32.0)</em>, <em>webshot(v.0.5.2)</em>, <em>DelayedArray(v.0.18.0)</em>, <em>car(v.3.0-11)</em>, <em>abind(v.1.4-5)</em>, <em>DBI(v.1.1.1)</em>, <em>rstatix(v.0.7.0)</em>, <em>Rcpp(v.1.0.7)</em>, <em>progress(v.1.2.2)</em>, <em>viridisLite(v.0.4.0)</em>, <em>xtable(v.1.8-4)</em>, <em>foreign(v.0.8-81)</em>, <em>bit(v.4.0.4)</em>, <em>DT(v.0.18)</em>, <em>htmlwidgets(v.1.5.3)</em>, <em>httr(v.1.4.2)</em>, <em>ellipsis(v.0.3.2)</em>, <em>farver(v.2.1.0)</em>, <em>XML(v.3.99-0.6)</em>, <em>pkgconfig(v.2.0.3)</em>, <em>sass(v.0.4.0)</em>, <em>utf8(v.1.2.1)</em>, <em>labeling(v.0.4.2)</em>, <em>tidyselect(v.1.1.1)</em>, <em>rlang(v.0.4.11)</em>, <em>later(v.1.2.0)</em>, <em>munsell(v.0.5.0)</em>, <em>BiocVersion(v.3.13.1)</em>, <em>cellranger(v.1.1.0)</em>, <em>tools(v.4.1.1)</em>, <em>cachem(v.1.0.5)</em>, <em>cli(v.3.0.0)</em>, <em>generics(v.0.1.0)</em>, <em>RSQLite(v.2.2.7)</em>, <em>broom(v.0.7.8)</em>, <em>evaluate(v.0.14)</em>, <em>fastmap(v.1.1.0)</em>, <em>ggdendro(v.0.1.22)</em>, <em>yaml(v.2.2.1)</em>, <em>knitr(v.1.33)</em>, <em>bit64(v.4.0.5)</em>, <em>fs(v.1.5.0)</em>, <em>zip(v.2.2.0)</em>, <em>KEGGREST(v.1.32.0)</em>, <em>mime(v.0.11)</em>, <em>xml2(v.1.3.2)</em>, <em>biomaRt(v.2.48.2)</em>, <em>compiler(v.4.1.1)</em>, <em>rstudioapi(v.0.13)</em>, <em>plotly(v.4.9.4.1)</em>, <em>filelock(v.1.0.2)</em>, <em>curl(v.4.3.2)</em>, <em>png(v.0.1-7)</em>, <em>interactiveDisplayBase(v.1.30.0)</em>, <em>ggsignif(v.0.6.2)</em>, <em>reprex(v.2.0.0)</em>, <em>bslib(v.0.2.5.1)</em>, <em>stringi(v.1.6.2)</em>, <em>highr(v.0.9)</em>, <em>lattice(v.0.20-44)</em>, <em>ProtGenerics(v.1.24.0)</em>, <em>Matrix(v.1.3-4)</em>, <em>vctrs(v.0.3.8)</em>, <em>pillar(v.1.6.1)</em>, <em>lifecycle(v.1.0.0)</em>, <em>BiocManager(v.1.30.16)</em>, <em>jquerylib(v.0.1.4)</em>, <em>cowplot(v.1.1.1)</em>, <em>data.table(v.1.14.0)</em>, <em>bitops(v.1.0-7)</em>, <em>rtracklayer(v.1.52.0)</em>, <em>httpuv(v.1.6.1)</em>, <em>BiocIO(v.1.2.0)</em>, <em>R6(v.2.5.0)</em>, <em>promises(v.1.2.0.1)</em>, <em>gridExtra(v.2.3)</em>, <em>rio(v.0.5.27)</em>, <em>parallelly(v.1.26.1)</em>, <em>codetools(v.0.2-18)</em>, <em>MASS(v.7.3-54)</em>, <em>assertthat(v.0.2.1)</em>, <em>SummarizedExperiment(v.1.22.0)</em>, <em>rjson(v.0.2.20)</em>, <em>rprojroot(v.2.0.2)</em>, <em>withr(v.2.4.2)</em>, <em>GenomicAlignments(v.1.28.0)</em>, <em>Rsamtools(v.2.8.0)</em>, <em>GenomeInfoDbData(v.1.2.6)</em>, <em>hms(v.1.1.0)</em>, <em>grid(v.4.1.1)</em>, <em>MatrixGenerics(v.1.4.0)</em>, <em>carData(v.3.0-4)</em>, <em>shiny(v.1.6.0)</em>, <em>lubridate(v.1.7.10)</em> and <em>restfulr(v.0.0.13)</em></p>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
