---
title: "Template"
author: "Lachlan Baer"
date: "`r format(Sys.time(), '%d %B, %Y')`"
output:
  html_document:
    toc: true
    toc_float: true
    toc_depth: 3
    fig_width: 8
    fig_height: 6
    fig_align: "center"
    code_folding: hide
editor_options: 
  chunk_output_type: console
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(
  echo = TRUE,
  message = FALSE,
  warning = FALSE,
  error = FALSE, 
  fig.align = "center"
)
knitr::opts_knit$set(root.dir = rprojroot::find_rstudio_root_file())
```

# Setup 

```{r packages}
library(tidyverse)
library(magrittr)
library(here)
library(SeqArray)
```

```{r options}
if (interactive()) setwd(here::here())
theme_set(theme_bw())
cores <- detectCores() - 2
```

# GDS conversion

```{r}
vcfPath <- "/hpcfs/users/a1647910/210305_chromosomeProject/1_sorl1_4way/12_mergeVcfs/vcf/mergedVcf.vcf.gz"
gdsPath <- "/hpcfs/users/a1647910/210305_chromosomeProject/1_sorl1_4way/12_mergeVcfs/gds/mergedGds.gds"
makeGds <-!file.exists(gdsPath)
if (makeGds) {
  seqVCF2GDS(vcfPath, gdsPath)
}
```

```{r}
gds <- seqOpen(gdsPath, readonly = FALSE)
print(gds, all=TRUE, attribute=TRUE)
```

```{r}
meta <- read_csv(here("files/SraRunTable.txt")) %>%
  dplyr::select(Run, gender, Genotype) %>%
  mutate(
    bname = case_when(
      Genotype == "wild-type" ~ "WT",
      Genotype == "sorl1V1482Afs/+" ~ "V1482Afs",
      Genotype == "sorl1R122Pfs/+" ~ "R122Pfs",
      Genotype == "sorl1V1482Afs/R122Pfs" ~ "Trans"
    )
  ) %>%
  dplyr::arrange(Genotype, Run) %>%
  mutate(
    sample =  paste0(bname, "_", c(1:4, 1:6, 1:6, 1:8))
  ) %>%
  dplyr::select(-bname) %>%
  dplyr::arrange(Run)
```

```{r}
samples <-  seqGetData(gds, "sample.id")
chr <- paste(c(1:25))
sampleDP <- seqGetData(gds, "annotation/format/DP") %>% 
  t() %>% 
  set_colnames(meta$sample) %>% 
  as_tibble()
sampleAD <- 
```

```{r}
snps <- tibble(
  variant.id = seqGetData(gds, "variant.id"),
  chromosome = seqGetData(gds, "chromosome"),
  position = seqGetData(gds, "position"),
  allele = seqGetData(gds, "allele")
) %>%
  cbind(sampleDP) %>%
  dplyr::filter(chromosome %in% chr) %>%
  as_tibble()
```
```{r}
seqSetFilter(gds, 1)
```

