---
title: "Enrichment Analysis"
subtitle: "210216_sorl1_snv dataset"
author: "Lachlan Baer"
date: "`r format(Sys.time(), '%d %B, %Y')`"
output:
  html_document:
    toc: true
    toc_float: true
    toc_depth: 2
    fig_width: 8
    fig_height: 6
    fig_align: "center"
    code_folding: hide
editor_options:
  chunk_output_type: console
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(
  echo = TRUE,
  message = FALSE,
  warning = FALSE,
  error = FALSE,
  fig.align = "center"
)
knitr::opts_knit$set(root.dir = rprojroot::find_rstudio_root_file())
```

# Setup

```{r packages}
suppressPackageStartupMessages({
  ## Common
  library(tidyverse)
  library(magrittr)
  library(future.apply)
  library(here)
  library(AnnotationHub)
  library(purrr)
  library(scales)
  library(kableExtra)
  library(tictoc)
  library(ggrepel)
  library(RColorBrewer)
  library(ggpubr)
  library(pander)
  library(rmarkdown)
  ## Project specific
  library(edgeR)
  library(limma)
  library(pheatmap)
  library(cqn)
  library(DT)
  library(htmltools)
  library(msigdbr)
  library(fgsea)
})
```

```{r options}
if (interactive()) setwd(here::here())
theme_set(theme_bw())
cores <- availableCores() - 1
```

```{r}
source("~/bioinformatics/bioToolkit/lbFuncs.R")
```

## EnsDb

```{r ensParams}
ens_species <- "Danio rerio"
ens_release <- "101"
ens_assembly <- "GRCz11"
```

```{r ensDb}
ah <- AnnotationHub() %>%
  subset(species == ens_species) %>%
  subset(rdataclass == "EnsDb")
ahId <- ah$ah_id[str_detect(ah$title, ens_release)]
ensDb <- ah[[ahId]]
```

```{r chrInfo}
chrInfo <- getChromInfoFromEnsembl(ens_assembly, release = ens_release) %>%
  dplyr::filter(coord_system == "chromosome")
primary_chrs <- chrInfo$name
```

```{r transcripts}
transcripts <- transcripts(ensDb, filter = SeqNameFilter(primary_chrs))
txLen <- exonsBy(ensDb, "tx", filter = SeqNameFilter(primary_chrs)) %>%
  width() %>%
  vapply(sum, integer(1))
mcols(transcripts) <- mcols(transcripts)[
  c("tx_id", "gene_id", "gc_content")
] %>%
  as.data.frame() %>%
  mutate(length = txLen)
```

```{r geneGc}
geneGc <- transcripts %>%
  mcols() %>%
  as_tibble() %>%
  group_by(gene_id) %>%
  summarise(
    gc_content = sum(gc_content*length) / sum(length),
    length = ceiling(median(length))
  )
```

```{r genes}
genes <- genes(ensDb, filter = SeqNameFilter(primary_chrs)) %>%
  .[,c("gene_id", "gene_name", "gene_biotype", "entrezid")] %>%
  as_tibble() %>%
  left_join(geneGc) %>%
  GRanges()
```

An `EnsDb` object was obtained for Ensembl release 101 with the `AnnotationHub` package.
This contained the information required to set up gene and transcript annotations.
Gene-level estimates of GC content and length were also generated which may be required as covariates in suitable analyses.

```{r entrezGenes}
entrezGenes <- genes %>%
  as.data.frame() %>%
  dplyr::filter(!is.na(entrezid)) %>%
  mutate(entrezid = unAsIs(entrezid)) %>%
  unnest(entrezid) %>%
  dplyr::rename(entrez_gene = entrezid)
```

```{r geneChrs}
geneChrs <- as_tibble(genes) %>%
  dplyr::select(gene_id, chromosome = seqnames)
```

## Metadata

```{r metadata}
metadata <- read_csv(here("files/SraRunTable.txt")) %>%
  as.data.frame() %>%
  dplyr::arrange(Run) %>%
  mutate(
    genotype = case_when(
      Genotype == "wild-type" ~ "WT",
      Genotype == "sorl1V1482Afs/+" ~ "V1482Afs_het",
      Genotype == "sorl1R122Pfs/+" ~ "R122Pfs_het",
      Genotype == "sorl1V1482Afs/R122Pfs" ~ "Trans",
    ),
    alias = c(
      paste0(rep("WT", 8), seq(1, 8)),
      paste0(rep("V1482Afs", 6), seq(1, 6)),
      paste0(rep("R122Pfs", 4), seq(1, 4)),
      paste0(rep("Trans", 6), seq(1, 6))
    )
  ) %>%
  dplyr::select(
    sample = Run, genotype, genotype2 = Genotype, alias, gender, tank = Tank
  )
metadata$genotype <- fct_relevel(
  metadata$genotype,
  c("WT", "V1482Afs_het", "R122Pfs_het", "Trans")
)
```

```{r genoCols}
genoCols <- metadata$genotype %>%
  levels() %>%
  length() %>%
  brewer.pal("Set1") %>%
  setNames(levels(metadata$genotype))
```

```{r samples_byGroup}
samples_byGroup <- metadata %>%
  split(f = .$genotype) %>%
  sapply(function(x){
    pull(x, sample)
  }, simplify = FALSE)
```

## Differential expression data

```{r dgeList}
dgeList <- readRDS(here("files/dgeList.Rds"))
```

```{r design}
design <- model.matrix(~0 + genotype, data = dgeList$samples) %>%
  set_colnames(c("WT", "V1482Afs_het", "R122Pfs_het", "Trans"))
```

```{r contrasts}
contrasts <- makeContrasts(
  V1482Afs_het = V1482Afs_het - WT,
  R122Pfs_het = R122Pfs_het - WT,
  Trans = Trans - WT,
  levels = colnames(design)
)
```

```{r topTables}
topTables <- readRDS(here("files/topTables.Rds"))
```

```{r deGenes}
deGenes <- lapply(topTables, function(tt){
  dplyr::filter(tt, DE) %>%
    pull(gene_id)
})
```

# Fisher's exact test for clustering

## Chromosome clustering

```{r}
byChr_fisher <- sapply(topTables, function(tt){
  # Check for DE genes
  if (sum(tt$DE)) {
    sapply(chrInfo$name[chrInfo$name != "MT"], function(chr){
      table(
        tt$DE,
        tt$chromosome == chr
      ) %>%
        fisher.test()
    }, simplify = FALSE) %>%
      .[as.character(sort(as.numeric(names(.)), ))]
  }
}, simplify = FALSE)
```

```{r}
byChr_pvals <- sapply(byChr_fisher, function(geno){
  lapply(geno, function(fisherRes){
    fisherRes$p.value
  }) %>%
    unlist() %>%
    enframe(name = "chr", value = "p") %>%
    mutate(
      bonf = p.adjust(p, method = "bonf"),
      sig = bonf < 0.05,
      p = signif(p, digits = 3),
      bonf = signif(bonf, digits = 3),
    ) %>%
    dplyr::filter(sig)
}, simplify = FALSE)
```

```{r}
htmltools::tagList({
  lapply(seq_along(byChr_pvals), function(i, tbl, name){
    datatable(
      tbl[[i]],
      caption = htmltools::tags$caption(
        "Table ", i, ": ", htmltools::em(name[[i]]),
        style = "text-align: left;"
      ),
      options = list(
        dom = "ti",  # Available options: tflipr
        columnDefs = list(list(className = 'dt-left', targets = "_all")),  # Align all columns left
        pageLength = -1,  # Show all results
        scrollCollapse = TRUE,  # Removes whitespace when table is small
        scrollY = 350  # Sets max height of table before scrollbar is added
      ),
      rownames = FALSE,
      height = "100%",  # Seems to work in tandem with scrollCollapse
      width = "100%"  # Fills width when viewed in RStudio viewer
    )
  }, tbl = byChr_pvals, name = names(byChr_pvals))
})
```